<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-08-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>batch_effect/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220518code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220518)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220518code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220518)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong8b24ef2">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> 8b24ef2
</a>
</p>
</div>
<div id="strongRepositoryversionstrong8b24ef2"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version 8b24ef2.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Distributions.pdf
    Untracked:  PCA_all.pdf
    Untracked:  Proteins per samples.pdf
    Untracked:  analysis/Template_LFQ.Rmd
    Untracked:  analysis/Template_LFQ_qn.Rmd
    Untracked:  analysis/Template_LFQ_vsn.Rmd
    Untracked:  analysis/egf0_vs_h2o2.Rmd
    Untracked:  analysis/egf1_vs_H2O2.Rmd
    Untracked:  analysis/egf1_vs_egf0.Rmd
    Untracked:  analysis/egf2_vs_H2O2.Rmd
    Untracked:  analysis/egf2_vs_egf0.Rmd
    Untracked:  analysis/egf3_vs_H2O2.Rmd
    Untracked:  analysis/egf3_vs_egf0.Rmd
    Untracked:  code/custom_functions.R
    Untracked:  data/PG_Unique.txt
    Untracked:  data/expdesign.tsv
    Untracked:  data/expdesign2.tsv
    Untracked:  data/expdesign_exp2.txt
    Untracked:  data/expdesign_exp2_2.txt
    Untracked:  data/proteinGroups.txt
    Untracked:  data/proteinGroups_exp2.txt
    Untracked:  output/results_table_EGF0_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF1_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF1_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF2_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF2_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF3_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF3_vs_H2O2.xlsx
    Untracked:  output/results_table_EGFR_t0_t1_t2_t3.xlsx
    Untracked:  output/results_table_EGFR_unique_t0_t1_t2_t3.xlsx
    Untracked:  output/results_table_sample2_vs_sample1.xlsx
    Untracked:  output/results_table_sample3_vs_sample2.xlsx
    Untracked:  output/results_table_sample4_vs_sample2.xlsx
    Untracked:  output/results_table_sample5_vs_sample2.xlsx
    Untracked:  output/significant_common_list.csv
    Untracked:  output/unique_common_list.csv
    Untracked:  venn_significant.png
    Untracked:  venn_significant_table.csv

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/anova_no_h2o2.Rmd</code>) and
HTML (<code>docs/anova_no_h2o2.html</code>) files. If you’ve configured
a remote Git repository (see <code>?wflow_git_remote</code>), click on
the hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
8b24ef2
</td>
<td>
FabioBedin
</td>
<td>
2022-08-17
</td>
<td>
fist commit of the second experiment
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;limma&quot;)
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot;,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)</code></pre>
<pre class="r"><code># cond_1 &lt;- params$test_1
# 
# cond_2 &lt;- params$test_2
# 
# conditions&lt;-c(cond_1,cond_2)
# 
# test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

# expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
# if(max(expdesign$replicate)&lt;=3){
#        threshold&lt;-0
#      } else if(max(expdesign$replicate)&lt;6){
#        threshold&lt;-1
#      } else if (max(expdesign$replicate)&gt;=6){
#        threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
#      }

data_filt &lt;- filter_missval(data_se, thr = 1)</code></pre>
<div id="all-egfr" class="section level1">
<h1>All EGFR</h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the
<strong>proteinGroups</strong> obtained by <a
href="https://www.maxquant.org/">MaxQuant</a> software, I remove
proteins that are marked form the software as potential contaminant,
only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins
identified by less than 0 peptides.<br />
Now, we generate some quality control plots to investigate the structure
of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
<pre class="r"><code># batch &lt;- c(&quot;A&quot;, &quot;A&quot;, &quot;A&quot;, &quot;A&quot;, &quot;B&quot;, &quot;B&quot;, &quot;B&quot;, &quot;B&quot;, &quot;C&quot;, &quot;C&quot;, &quot;C&quot;, &quot;D&quot;, &quot;D&quot;, &quot;D&quot;, &quot;D&quot;)
# 
# assay(data_filt) &lt;- limma::removeBatchEffect(assay(data_filt), batch)</code></pre>
</div>
<div id="quality-control-plots"
class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>1. Quality control
plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3 class="tabset">Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples
<em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3 class="tabset">Distributions</h3>
<p>This plot is useful for checking the distributions of the samples
<em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3 class="tabset">Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and
before User define</em> filters. Notice that the percentages of
missingness are provided in the data. These are accurate to 1 decimal
place. Also the dataset is arranged by columns with most
missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3 class="tabset">Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the
dispersion of data points in a data series around the mean. The
coefficient of variation represents the ratio of the standard deviation
to the mean, and it is a useful statistic for comparing the degree of
variation from one data series to another, even if the means are
drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within
the 2 conditions or in common between all condition.</p>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 4

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        # sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;, &quot;red&quot;, &quot;yellow&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="imputation" class="section level2">
<h2><strong>3. Imputation</strong></h2>
<p>An idealized version of a label-free discovery mass spectrometry
proteomics experiment would provide absolute abundance measurements for
a whole proteome, across varying conditions. Unfortunately, this ideal
is not realized. Measurements are made on peptides requiring an
inferential step to obtain protein level estimates. The inference is
complicated by experimental factors that necessitate relative abundance
estimation and result in widespread non-ignorable missing data. Relative
abundance on the log scale takes the form of parameter contrasts. In a
complete-case analysis, contrast estimates may be biased by missing data
and a substantial amount of useful information will often go unused.</p>
<pre class="r"><code>data_imputed &lt;- DEP::impute(data_filt, fun = &quot;man&quot;)

plot_imputation(data_filt, data_imputed)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/imputation-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>data_filt &lt;- data_imputed</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics
are used for the differential enrichment analysis (or differential
expression analysis) and False discovery rate (FDR) is ajusted with
<em>Benjamini-Hochberg</em> procedure. Significant proteins are are
define with <strong>FDR = 0.05</strong> and <strong>Fold change =
1</strong></p>
<pre class="r"><code>data_diff &lt;- anova_test(data_filt)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>55</strong> significant proteins</p>
<p>The results from the previous analysis can be easily visualized by a
number of functions. These visualizations assist in the determination of
the optimal cutoffs to be used, highlight the most interesting samples
and contrasts, and pinpoint differentially enriched/expressed
proteins.</p>
<div id="visualization-of-the-results"
class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>4.1. Visualization of the
results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data.
This can be very useful to observe batch effects, such as clear
differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="heatmap-unsupervised" class="section level4">
<h4>Heatmap unsupervised</h4>
<pre class="r"><code>mat &lt;- as.matrix(assay(data_filt))
mat_z &lt;- scale(mat)
aaa &lt;- results %&gt;% filter(significant) %&gt;% pull(name)
mat_final &lt;- mat_z %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column(var = &quot;gene_names&quot;) %&gt;% 
  filter(gene_names %in% aaa) %&gt;% 
  column_to_rownames(&quot;gene_names&quot;) %&gt;% 
  as.matrix()


  

ComplexHeatmap::Heatmap(mat_final,
                        name = &quot;z-score&quot;,
                        # clustering_distance_rows = &quot;pearson&quot;,
                        # clustering_method_rows = &quot;average&quot;,
                        use_raster = FALSE,
                        show_row_names = FALSE)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/complex-heatmap-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="heatmap-time-course" class="section level4">
<h4>Heatmap time course</h4>
<pre class="r"><code>col_split &lt;- c(&quot;sample2&quot;, &quot;sample2&quot;, &quot;sample2&quot;, &quot;sample2&quot;, &quot;sample2&quot;,
               &quot;sample3&quot;, &quot;sample3&quot;, &quot;sample3&quot;, &quot;sample3&quot;, &quot;sample3&quot;,
               &quot;sample4&quot;, &quot;sample4&quot;, &quot;sample4&quot;, &quot;sample4&quot;, &quot;sample4&quot;,
               &quot;sample5&quot;, &quot;sample5&quot;, &quot;sample5&quot;, &quot;sample5&quot;, &quot;sample5&quot;)

ComplexHeatmap::Heatmap(mat_final,
                        name = &quot;heatmap&quot;,
                        use_raster = FALSE,
                        show_row_names = FALSE,
                        # clustering_distance_rows = &quot;single&quot;,
                        # cluster_rows = FALSE,
                        column_split = col_split,
                        cluster_column_slices = FALSE)</code></pre>
<p><img src="figure/anova_no_h2o2.Rmd/complex-heatmap-time-course-1.png" width="960" style="display: block; margin: auto;" /></p>
<pre class="r"><code>                        # row_km = 5)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] forcats_0.5.1               stringr_1.4.0              
 [3] dplyr_1.0.8                 purrr_0.3.4                
 [5] readr_2.1.2                 tidyr_1.2.0                
 [7] tibble_3.1.6                tidyverse_1.3.1            
 [9] openxlsx_4.2.5              here_1.0.1                 
[11] limma_3.46.0                RColorBrewer_1.1-3         
[13] RCy3_2.10.2                 STRINGdb_2.2.2             
[15] naniar_0.6.1                visdat_0.5.3               
[17] ggrepel_0.9.1               ggplot2_3.3.6              
[19] UpSetR_1.4.0                ggstatsplot_0.9.3          
[21] datapasta_3.1.0             patchwork_1.1.1            
[23] DT_0.23                     rrvgo_1.2.0                
[25] org.Hs.eg.db_3.12.0         AnnotationDbi_1.52.0       
[27] gprofiler2_0.2.1            enrichR_3.0                
[29] preprocessCore_1.52.1       SummarizedExperiment_1.20.0
[31] Biobase_2.50.0              GenomicRanges_1.42.0       
[33] GenomeInfoDb_1.26.7         IRanges_2.24.1             
[35] S4Vectors_0.28.1            BiocGenerics_0.36.1        
[37] MatrixGenerics_1.2.1        matrixStats_0.61.0         
[39] DEP_1.12.0                  MBQN_2.2.0                 
[41] workflowr_1.7.0            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3         R.methodsS3_1.8.1      bit64_4.0.5           
  [4] knitr_1.39             DelayedArray_0.16.2    R.utils_2.11.0        
  [7] data.table_1.14.2      RCurl_1.98-1.6         doParallel_1.0.17     
 [10] generics_0.1.2         callr_3.7.0            RSQLite_2.2.10        
 [13] correlation_0.8.1      chron_2.3-56           bit_4.0.4             
 [16] tzdb_0.3.0             lubridate_1.8.0        xml2_1.3.3            
 [19] httpuv_1.6.5           assertthat_0.2.1       xfun_0.30             
 [22] hms_1.1.1              jquerylib_0.1.4        evaluate_0.15         
 [25] promises_1.2.0.1       fansi_1.0.3            readxl_1.4.0          
 [28] caTools_1.18.2         dbplyr_2.1.1           igraph_1.3.0          
 [31] DBI_1.1.2              htmlwidgets_1.5.4      paletteer_1.4.0       
 [34] hash_2.2.6.2           ellipsis_0.3.2         backports_1.4.1       
 [37] insight_0.17.1         gridBase_0.4-7         vctrs_0.4.0           
 [40] Cairo_1.5-15           imputeLCMD_2.0         cachem_1.0.6          
 [43] withr_2.5.0            treemap_2.4-3          cluster_2.1.0         
 [46] lazyeval_0.2.2         crayon_1.5.1           labeling_0.4.2        
 [49] pkgconfig_2.0.3        slam_0.1-50            statsExpressions_1.3.2
 [52] ProtGenerics_1.22.0    wordcloud_2.6          rlang_1.0.2           
 [55] RJSONIO_1.3-1.6        lifecycle_1.0.1        sandwich_3.0-1        
 [58] affyio_1.60.0          BiocFileCache_1.14.0   modelr_0.1.8          
 [61] cellranger_1.1.0       rprojroot_2.0.3        graph_1.68.0          
 [64] datawizard_0.4.1       Matrix_1.2-18          zoo_1.8-10            
 [67] reprex_2.0.1           whisker_0.4            GlobalOptions_0.1.2   
 [70] processx_3.5.3         pheatmap_1.0.12        png_0.1-7             
 [73] viridisLite_0.4.0      rjson_0.2.21           mzR_2.24.1            
 [76] parameters_0.18.1      bitops_1.0-7           shinydashboard_0.7.2  
 [79] getPass_0.2-2          R.oo_1.24.0            KernSmooth_2.23-17    
 [82] blob_1.2.3             shape_1.4.6            tmvtnorm_1.5          
 [85] scales_1.2.0           memoise_2.0.1          magrittr_2.0.3        
 [88] plyr_1.8.7             gplots_3.1.3           zlibbioc_1.36.0       
 [91] compiler_4.0.3         plotrix_3.8-2          pcaMethods_1.82.0     
 [94] clue_0.3-60            cli_3.2.0              affy_1.68.0           
 [97] XVector_0.30.0         ps_1.6.0               MASS_7.3-53           
[100] tidyselect_1.1.2       vsn_3.58.0             stringi_1.7.6         
[103] highr_0.9              yaml_2.3.5             GOSemSim_2.16.1       
[106] norm_1.0-10.0          MALDIquant_1.21        grid_4.0.3            
[109] sass_0.4.1             tools_4.0.3            circlize_0.4.15       
[112] rstudioapi_0.13        foreach_1.5.2          git2r_0.30.1          
[115] gridExtra_2.3          farver_2.1.0           mzID_1.28.0           
[118] digest_0.6.29          BiocManager_1.30.18    shiny_1.7.1           
[121] proto_1.0.0            Rcpp_1.0.8.3           broom_0.8.0           
[124] performance_0.9.0      later_1.3.0            ncdf4_1.19            
[127] httr_1.4.3             MSnbase_2.15.7         ComplexHeatmap_2.13.1 
[130] colorspace_2.0-3       rvest_1.0.2            XML_3.99-0.9          
[133] fs_1.5.2               rematch2_2.1.2         gmm_1.6-6             
[136] plotly_4.10.0          xtable_1.8-4           jsonlite_1.8.0        
[139] zeallot_0.1.0          NLP_0.2-1              R6_2.5.1              
[142] tm_0.7-8               gsubfn_0.7             pillar_1.7.0          
[145] htmltools_0.5.2        mime_0.12              glue_1.6.2            
[148] fastmap_1.1.0          BiocParallel_1.24.1    codetools_0.2-16      
[151] mvtnorm_1.1-3          utf8_1.2.2             lattice_0.20-41       
[154] bslib_0.3.1            sqldf_0.4-11           curl_4.3.2            
[157] gtools_3.9.2           zip_2.2.0              GO.db_3.12.1          
[160] rmarkdown_2.14         munsell_0.5.0          GetoptLong_1.0.5      
[163] GenomeInfoDbData_1.2.4 iterators_1.0.14       impute_1.64.0         
[166] haven_2.5.0            gtable_0.3.0           bayestestR_0.12.1     </code></pre>
</div>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
