<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Fabio Bedin | MS-Unit at IEO" />


<title>LFQ_Analysis</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<script src="site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.23/datatables.js"></script>
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.11.3/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.11.3/js/jquery.dataTables.min.js"></script>
<link href="site_libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<script src="site_libs/plotly-binding-4.10.0/plotly.js"></script>
<script src="site_libs/typedarray-0.1/typedarray.min.js"></script>
<link href="site_libs/plotly-htmlwidgets-css-2.5.1/plotly-htmlwidgets.css" rel="stylesheet" />
<script src="site_libs/plotly-main-2.5.1/plotly-latest.min.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Template_LFQ</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LFQ_Analysis</h1>
<h4 class="author">Fabio Bedin | MS-Unit at IEO</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-08-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>batch_effect/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220518code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220518)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220518code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220518)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong8b24ef2">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong> 8b24ef2
</a>
</p>
</div>
<div id="strongRepositoryversionstrong8b24ef2"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version 8b24ef2.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Distributions.pdf
    Untracked:  PCA_all.pdf
    Untracked:  Proteins per samples.pdf
    Untracked:  analysis/Template_LFQ.Rmd
    Untracked:  analysis/Template_LFQ_qn.Rmd
    Untracked:  analysis/Template_LFQ_vsn.Rmd
    Untracked:  analysis/egf0_vs_h2o2.Rmd
    Untracked:  analysis/egf1_vs_H2O2.Rmd
    Untracked:  analysis/egf1_vs_egf0.Rmd
    Untracked:  analysis/egf2_vs_H2O2.Rmd
    Untracked:  analysis/egf2_vs_egf0.Rmd
    Untracked:  analysis/egf3_vs_H2O2.Rmd
    Untracked:  analysis/egf3_vs_egf0.Rmd
    Untracked:  code/custom_functions.R
    Untracked:  data/PG_Unique.txt
    Untracked:  data/expdesign.tsv
    Untracked:  data/expdesign2.tsv
    Untracked:  data/expdesign_exp2.txt
    Untracked:  data/expdesign_exp2_2.txt
    Untracked:  data/proteinGroups.txt
    Untracked:  data/proteinGroups_exp2.txt
    Untracked:  output/results_table_EGF0_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF1_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF1_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF2_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF2_vs_H2O2.xlsx
    Untracked:  output/results_table_EGF3_vs_EGF0.xlsx
    Untracked:  output/results_table_EGF3_vs_H2O2.xlsx
    Untracked:  output/results_table_EGFR_t0_t1_t2_t3.xlsx
    Untracked:  output/results_table_EGFR_unique_t0_t1_t2_t3.xlsx
    Untracked:  output/results_table_sample2_vs_sample1.xlsx
    Untracked:  output/results_table_sample3_vs_sample2.xlsx
    Untracked:  output/results_table_sample4_vs_sample2.xlsx
    Untracked:  output/results_table_sample5_vs_sample2.xlsx
    Untracked:  output/significant_common_list.csv
    Untracked:  output/unique_common_list.csv
    Untracked:  venn_significant.png
    Untracked:  venn_significant_table.csv

Unstaged changes:
    Modified:   analysis/_site.yml
    Deleted:    analysis/about.Rmd
    Deleted:    analysis/license.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/sample2_vs_sample1.Rmd</code>)
and HTML (<code>docs/sample2_vs_sample1.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
8b24ef2
</td>
<td>
FabioBedin
</td>
<td>
2022-08-17
</td>
<td>
fist commit of the second experiment
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
p{
  font-size: 16px;
}
</style>
<pre class="r"><code>## Proteomics
library(&quot;MBQN&quot;)
library(&quot;DEP&quot;)
library(&quot;SummarizedExperiment&quot;)
library(&quot;preprocessCore&quot;)
library(&quot;enrichR&quot;)
library(&quot;gprofiler2&quot;)
library(&quot;org.Hs.eg.db&quot;)
library(&quot;rrvgo&quot;)

## Plots
library(&quot;DT&quot;)
library(&quot;patchwork&quot;)
library(&quot;datapasta&quot;)
library(&quot;ggstatsplot&quot;)
library(&quot;UpSetR&quot;)
library(&quot;ggrepel&quot;)
library(&quot;visdat&quot;)
library(&quot;naniar&quot;)

## Networks
library(&quot;STRINGdb&quot;)
library(&quot;RCy3&quot;)
library(&quot;RColorBrewer&quot;)

## Other
library(&quot;limma&quot;)
library(&quot;here&quot;)
library(&quot;openxlsx&quot;)
library(&quot;tidyverse&quot;)</code></pre>
<pre class="r"><code>source(here::here(&quot;code/custom_functions.R&quot;))</code></pre>
<pre class="r"><code>data &lt;- read.csv(here::here(paste0(&quot;data/&quot;, params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = &quot;\t&quot;)
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data &lt;- data[data$Reverse != &quot;+&quot; &amp; data$Potential.contaminant != &quot;+&quot; &amp; data$Only.identified.by.site != &quot;+&quot;,]

data_unique &lt;- make_unique(data, &quot;Gene.names&quot;, &quot;Protein.IDs&quot;, delim = &quot;;&quot;)

data_unique &lt;- data_unique %&gt;% filter(!name == &quot;S100A7&quot;)</code></pre>
<pre class="r"><code>cond_1 &lt;- params$test_1

cond_2 &lt;- params$test_2

conditions&lt;-c(cond_1,cond_2)

test&lt;- paste(cond_1,cond_2,sep=&quot;_vs_&quot;)

expdesign &lt;- read.table(here::here(paste0(&quot;data/&quot;, params$expdesign)), header = T, stringsAsFactors = F)

expdesign &lt;- subset(expdesign, condition %in% conditions)

columns&lt;-match(paste(&quot;LFQ.intensity.&quot;,expdesign$label,sep=&quot;&quot;),colnames(data_unique))

data_se &lt;- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
# if(max(expdesign$replicate)&lt;=3){
#        threshold&lt;-0
#      } else if(max(expdesign$replicate)&lt;6){
#        threshold&lt;-1
#      } else if (max(expdesign$replicate)&gt;=6){
#        threshold&lt;-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
#      }

data_filt &lt;- filter_missval(data_se, thr = 1)</code></pre>
<div id="sample2-vs-sample1" class="section level1">
<h1><strong>sample2</strong> vs <strong>sample1</strong></h1>
<hr />
</div>
<div id="introduction" class="section level1">
<h1><strong>Introduction</strong></h1>
<p>First, we need to wrangling the original dataset. From the
<strong>proteinGroups</strong> obtained by <a
href="https://www.maxquant.org/">MaxQuant</a> software, I remove
proteins that are marked form the software as potential contaminant,
only identify by site and reverse.<br />
We che also filter the initial datasets by excluding all proteins
identified by less than 2 peptides.<br />
Now, we generate some quality control plots to investigate the structure
of the dataset and observe the effect of filters.</p>
<div id="normalization-strategy" class="section level4">
<h4><strong>Normalization strategy:</strong></h4>
<pre class="r"><code>if(params$Normalization == &quot;VSN&quot;){
  data_filt &lt;- normalize_vsn(data_filt)
  print(&quot;VSN normalization is used&quot;)
} else if(params$Normalization == &quot;MBQN&quot;){
  data_filt &lt;- normalize_MBQN(data_filt)
  print(&quot;MBQN normalization is used&quot;)
} else {
  print(&quot;Max-LFQ normalization is used&quot;)
}</code></pre>
<p>[1] “Max-LFQ normalization is used”</p>
<pre class="r"><code># 
# batch &lt;- c(&quot;A&quot;, &quot;A&quot;, &quot;A&quot;, &quot;A&quot;, &quot;B&quot;, &quot;B&quot;, &quot;B&quot;, &quot;B&quot;)
# 
# assay(data_filt) &lt;- limma::removeBatchEffect(assay(data_filt), batch)</code></pre>
</div>
<div id="quality-control-plots"
class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>1. Quality control
plots</strong></h2>
<div id="proteins-per-samples" class="section level3 tabset">
<h3 class="tabset">Proteins per samples</h3>
<p>This plot show the number of proteins identify in each samples
<em>after and before User define</em> filters:</p>
<div id="filtered" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_filt)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/proteins-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_numbers_lables(data_se)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/proteins-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="distributions" class="section level3 tabset">
<h3 class="tabset">Distributions</h3>
<p>This plot is useful for checking the distributions of the samples
<em>after and before User define</em> filters:</p>
<div id="filtered-1" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_normalization(data_filt)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/normalization-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-1" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_normalization(data_se)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/normalization-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="missing-data" class="section level3 tabset">
<h3 class="tabset">Missing data</h3>
<p>This plot we can explore the missing data pattern <em>after and
before User define</em> filters. Notice that the percentages of
missingness are provided in the data. These are accurate to 1 decimal
place. Also the dataset is arranged by columns with most
missingness.</p>
<div id="filtered-2" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/missing-data-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-2" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>assay(data_se) %&gt;% 
  as.data.frame() %&gt;% 
  vis_miss(., sort_miss = TRUE)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/missing-data-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample-cvs" class="section level3 tabset">
<h3 class="tabset">Sample CVs</h3>
<p>The coefficient of variation (CV) is a statistical measure of the
dispersion of data points in a data series around the mean. The
coefficient of variation represents the ratio of the standard deviation
to the mean, and it is a useful statistic for comparing the degree of
variation from one data series to another, even if the means are
drastically different from one another.</p>
<div id="filtered-3" class="section level4">
<h4>Filtered</h4>
<pre class="r"><code>plot_cvs(data_filt)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/CVs-filt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="not-filtered-3" class="section level4">
<h4>Not-Filtered</h4>
<pre class="r"><code>plot_cvs(data_se)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/CVs-NOfilt-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="upset-plot" class="section level2 tabset tabset-fade">
<h2 class="tabset tabset-fade"><strong>2. Upset plot</strong></h2>
<p>With this plot we can identify the numper of prteins in common within
the 2 conditions or in common between all condition.</p>
<div id="sample2" class="section level3">
<h3>sample2</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_1)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(cond_1)) %&gt;%
  rownames_to_column() %&gt;%
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;%
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;%
  pivot_wider(names_from = samples, values_from = intensity) %&gt;%
  as.data.frame() %&gt;%
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5,
        sets.bar.color = &quot;coral2&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/upset-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="sample1" class="section level3">
<h3>sample1</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;%  as.data.frame() %&gt;% select(starts_with(cond_2)) %&gt;% colnames()
n_sets &lt;- length(define_set)

assay(data_filt) %&gt;%
  as.data.frame() %&gt;%
  select(starts_with(cond_2)) %&gt;%
  rownames_to_column() %&gt;%
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;%
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;%
  pivot_wider(names_from = samples, values_from = intensity) %&gt;%
  as.data.frame() %&gt;%
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5,
        sets.bar.color = &quot;turquoise3&quot;,
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/upset-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
<div id="both-conditions" class="section level3">
<h3>Both conditions</h3>
<pre class="r"><code>define_set &lt;- assay(data_filt) %&gt;% colnames()
n_sets &lt;- length(define_set)
n_each &lt;- n_sets / 2

assay(data_filt) %&gt;% 
  as.data.frame() %&gt;% 
  rownames_to_column() %&gt;% 
  pivot_longer(!rowname, names_to = &quot;samples&quot;, values_to = &quot;intensity&quot;) %&gt;% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %&gt;% 
  pivot_wider(names_from = samples, values_from = intensity) %&gt;% 
  as.data.frame() %&gt;% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = &quot;freq&quot;,
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        sets.bar.color = rep(c(&quot;red3&quot;, &quot;royalblue1&quot;), each = n_each),
        main.bar.color  = &quot;gray44&quot;)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/upset-both-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="unique-proteins" class="section level2">
<h2><strong>3. Unique proteins</strong></h2>
<p>This table show unique protein for both conditions</p>
<pre class="r"><code>unique_pair(data_filt_unique, conditions = conditions)</code></pre>
<div id="htmlwidget-1bc5c7d9efa36a2864ae" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-1bc5c7d9efa36a2864ae">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879","880","881","882","883","884","885","886","887","888","889","890","891","892","893","894","895","896","897","898","899","900","901","902","903","904","905","906","907","908","909","910","911","912","913","914","915","916","917","918","919","920","921","922","923","924","925","926","927","928","929","930","931","932","933","934","935","936","937","938","939","940","941","942","943","944","945","946","947","948","949","950","951","952","953","954","955","956","957","958","959","960","961","962","963","964","965","966","967","968","969","970","971","972","973","974","975","976","977","978","979","980","981","982","983","984","985","986","987","988","989","990","991","992","993","994","995","996","997","998","999","1000","1001","1002","1003","1004","1005"],["A0A1B0GW03","A0A804HK76","AAAS","AACS","AAK1","AAR2","AASDHPPT","ABCC1","ABCD1","ABHD12","ABI1","ABI2","ABLIM1","ACAD9","ACBD3","ACIN1","ACOT13","ACOT2","ACOX1","ACSL4","ACTN1","ADAM15","ADCY9","ADD1","ADI1","AFG3L2","AFTPH","AGL","AHCTF1","AIFM2","AJUBA","AK6","AKAP1","AKAP13","AKAP17A","AKAP8L","AKAP9","ALDH3A1","ALDOC","ALG1","AMPD2","ANAPC2","ANKHD1","ANKHD1-EIF4EBP3","ANKLE2","ANKRD28","ANKRD50","ANKS1A","ANTXR1","ANXA4","AP3S1","APC","APEH","APEX2","APOBEC3C","ARFGAP1","ARFGAP3","ARFIP2","ARHGAP12","ARHGAP18","ARHGAP21","ARHGAP23","ARHGEF10","ARHGEF12","ARHGEF17","ARHGEF5","ARID2","ARL3","ARMC8","ARMCX3","ARPC5","ARRDC1","ASCC3","ASF1B","ASNA1","ASRGL1","ATAD1","ATAD2","ATG3","ATIC","ATP13A1","ATP1B1","ATP2B4","ATP7A","ATXN2L","AURKA","AURKB","AXL","B4GALT5","BAG2","BAG3","BAG5","BAG6","BASP1","BAZ1A","BBX","BCAR1","BCOR","BCR","BMP2K","BMPR2","BOLA1","BOLA2","BRAF","BRAT1","BRCC3","BST2","BUB1","BUB1B","BYSL","BZW1","C14orf166","C1orf122","C1orf226","C2CD5","C2orf44","C3orf17","C5orf51","CALU","CAMK2G","CAMSAP1","CAPG","CAPN2","CAPRIN1","CARHSP1","CASK","CASP2","CASP3","CASP8","CBL","CBR3","CBWD2","CBX5","CCBL2","CCDC22","CCDC6","CCM2","CCZ1B","CD109","CD151","CD2AP","CD33","CD46","CDC123","CDC16","CDC23","CDC34","CDC37","CDC37L1","CDC42","CDC42BPA","CDC42BPB","CDC42EP1","CDIPT","CDON","CEBPZ","CECR5","CEP170","CEP170B","CEP55","CHD4","CHERP","CHM","CHMP4B","CIAPIN1","CISD2","CLASP1","CLASP2","CLDN1","CLIC4","CLIP1","CLPB","CLTA","CMBL","CMIP","CNBP","CNDP2","CNN2","CNN3","CNOT1","CNOT2","CNOT6L","CNP","COBL","COBLL1","COG1","COIL","COMMD3-BMI1","COMT","COPG2","COPS4","COPZ1","CPNE1","CRK","CRKL","CROCC","CRTAP","CSK","CSNK1D","CSNK2A1","CSNK2A2","CSNK2B-LY6G5B-562","CSRP1","CTH","CTNNA1","CTNNA2","CTNNAL1","CTNNB1","CTPS2","CTTNBP2NL","CUL2","CUL3","CUX1","CXADR","CYP51A1","CYTH2","DAG1","DCAF16","DCK","DCP1A","DCPS","DCTN4","DCTPP1","DCUN1D5","DDB1","DDX20","DDX54","DDX56","DENND4C","DERA","DFFA","DHX38","DHX57","DIMT1","DIP2B","DLG1","DLG3","DLG5","DLGAP4","DMD","DNAJB14","DNAJC11","DNAJC5","DNM1","DNMBP","DNPEP","DNTTIP2","DOCK5","DPH1","DPP9","DRG2","DST","DSTN","DTL","DUSP12","DUSP3","DUSP9","DVL2","DVL3","DYNLT1","ECHDC1","ECT2","EDC3","EEF1B2","EEF1D","EEFSEC","EFHD2","EFNB1","EFTUD1","EHD2","EHD4","EIF2B2","EIF2B3","EIF2B4","EIF3G","EIF3H","EIF4G2","EIF4G3","EIF5","EIF6","ELAC2","ELFN2","ELMSAN1","ELP2","ELP3","ELP4","ELP6","Em:AP000351.3","EMC1","EML4","ENSA","EPB41L5","EPHB2","EPHB4","EPM2AIP1","EPS15","EPS15L1","ERBB2","ERC1","ERCC6L","ERMP1","ERO1L","ETNPPL","EWSR1","EXOC1","EXOC3","EXOC4","EXOC5","EXOC8","EXOSC3","EXOSC9","EXTL3","F11R","FAM114A1","FAM120B","FAM127C","FAM129A","FAM175B","FAM208A","FAM83D","FANCI","FAR1","FARP2.1","FAT1","FBXL18","FBXO22","FGD4","FHL2","FIGNL1","FKBP10","FKBP15","FKBP4","FLAD1","FLII","FLVCR1","FLYWCH2","FRG1","FTSJ1","FTSJ3","FUBP3","FUS","FXR2","G3BP2","GAGE13","GALK1","GCA","GEMIN4","GET4","GIGYF2","GIT2","GLI2","GLOD4","GMDS","GMPPB","GNA13","GNAS","GNE","GNL1","GNL3L","GOLGA2","GPAA1","GRAMD3","GRB2","GRK6","GSK3B","GSPT2","GSTM5","GSTO1","GTF3C2","GTF3C4","GTF3C5","GTPBP1","HAUS6","hCG_2004507","HDGF","HEATR3","HEATR5A","HECTD1","HERC4","HEXIM1","HINT2","HIP1R","HLTF","HMMR","HNRNPA3","HNRNPF","HNRNPLL","HPRT1","HPS3","HRAS","HSDL1","HSPA14","HSPA2","HSPBP1","IAH1","ID1","IDH3G","IFIT3","IFIT5","IGBP1","IGF1R","IGFBP7","IL6ST","INSR","IPO11","IPO8","IQGAP3","IQSEC1","IRF2BP1","IRGQ","ISOC1","ISYNA1","ITCH","ITGA1","ITGA2","ITGA2.1","ITGA6","ITGB5","ITGB6","ITPA","ITPK1","ITSN2","JMJD6","KANK2","KDM2A","KDM3B","KEAP1","KIAA0020","KIAA1217","KIAA1429","KIAA1522","KIAA1598","KIAA1671","KIDINS220","KIF14","KIF20A","KIF21A","KIF23","KIF2C","KIF4A","KIFC1","KLC1","KLC2","KRTCAP2","KYNU","L1CAM","LANCL1","LARP4B","LASP1","LATS1","LCLAT1","LCMT1","LDLR","LENG9","LGALS3","LIMCH1","LLGL1","LMTK2","LPP","LRCH3","LRRC1","LRRC16A","LRRFIP1","LRSAM1","LSR","LYPLA2","LYST","MACF1","MACROD1","MAD2L1","MAGT1","MAP2K1","MAP2K3","MAP4K4","MAP7","MAPK1","MAPKAPK3","MAPRE1","MAPRE2","MARCKS","MARK2","MARK3","MAT2B","MAVS","MCTS1","MDC1","MDN1","METAP1","METAP2","MICU1","MICU2","MID1","MINK1","MLH1","MLLT4","MMS19","MOB1B","MOCOS","MOSPD2","MPDZ","MPP1","MPP6","MPRIP","MPST","MRPL21","MRPL23","MRPL39","MRPL45","MRPL48","MRPS17","MRPS27","MT-ND2","MT-ND4","MTCH1","MTCL1","MTHFD1L","MTMR1","MTMR3","MTSS1L","MVP","MYDGF","MYLK","MYO1B","MYO9B","NAA30","NAA40","NADK2","NAP1L4","NAPG","NASP","NBAS","NBN","NCAPG","NCAPH","NCDN","NCK1","NCKAP5L","NCOR1","NCSTN","NDC1","NDUFA4","NEDD4","NEDD4L","NEK9","NEO1","NES","NF2","NFAT5","NFATC3","NFKB1","NFKB2","NFKBIB","NHLRC2","NHSL1.1","NIFK","NKRF","NMD3","NME1","NMRAL1","NMT1","NOB1","NOC4L","NOL3","NOL6","NOL9","NOSIP","NOTCH3","NPLOC4","NR3C1","NRAS","NRBP1","NT5C3A","NT5DC1","NTMT1","NUB1","NUBP1","NUBP2","NUDCD1","NUDT5","NUMB","NUP153","NUP188","NUP62","NUP98","NUTF2","OGFR","ORC3","ORC5","OSBP","OSBPL10","OSBPL11","OSBPL3","OSBPL8","OSBPL9","OTUD7B","OTULIN","PACSIN2","PACSIN3","PAFAH1B3","PAIP1","PAK1","PAK2","PAK4","PALLD","PAPOLA","PARD3","PARN","PARP14","PARP4","PARVA","PCBP2","PCDH1","PCK2","PCM1","PCYT1A","PDAP1","PDCD10","PDCD5","PDE12","PDE4DIP","PDLIM3","PDLIM5","PDLIM7","PDPK1","PDS5B","PDXDC1","PDZD11","PEAK1","PEBP1","PEF1","PEG10","PES1","PFDN5","PFDN6","PFN2.1","PGM2","PGM3","PHACTR4","PHF3","PHKA1","PHLDA1","PHLDB2","PHOSPHO1","PHPT1","PIK3C2A","PIK3C2B","PIP4K2C","PITPNB","PKP4","PLCD3","PLCL2","PLEKHG3","PLS1","PLSCR1","PLXNB2","PMPCA","PNPLA6","PNPO","PODXL","POLA1","POLA2","POLD1","POLR1B","POLR2C","POLR2L","PON2","POP4","PPAT","PPCDC","PPL","PPP1CB","PPP1R11","PPP1R18","PPP1R3D","PPP1R7","PPP1R9A","PPP2R2A","PPP5C","PPP6R2","PPP6R3","PPT1","PRC1","PRDX3","PRIM1","PRKAA1","PRKAG1","PRKCA","PRKCD","PRKCI","PRKD3","PRMT3","PRMT5","PRMT7","PROCR","PROSER2","PRPF40A","PRPSAP1","PRPSAP2","PSMA4","PSMB1","PSMD10","PSMD4","PSMD9","PSME3","PTGS2","PTPN23","PTPRF","PTPRJ","PTPRM","PTRF","PUM1","PUM2","PURA","PUS1","PVR","PVRL1","PVRL2","PYGB","QRICH1","RAB35","RAB3GAP1","RAB3GAP2","RAB5B","RAB9A","RABGAP1","RABGGTB","RACGAP1","RAD50","RAI1","RALBP1","RANBP3","RANBP6","RAP1GDS1","RAPGEF6","RAVER1","RBM25","RBM27","RBM47","RCL1","RETSAT","RHOA","RHOF","RHPN2","RING1","RINT1","RIPK1","RNF214","RNMT","ROR2","RP2","RPA2","RPAP2","RPIA","RPRD1B","RPRD2","RPS6KA1","RPS6KB1","RRAGC","RRAS2","RRP1B","RRP7A","RRP8","RTN3.1","RUFY1","SAFB","SAFB2","SAMM50","SASS6","SCAF1","SCAF11","SCARB1","SCRIB","SCRN2","SCRN3","SCYL1","SCYL2","SDC1","SEC23IP","SEC24B","SEC24D","SEC61B","SELENBP1","SEPT6","SERPINB8","SETD2","SF1","SF3B3","SH2D4A","SH3BP4","SH3GL1","SH3GLB1","SH3GLB2","SHB","SHC1","SHKBP1","SHOC2","SIN3A","SIN3B","SLAIN2","SLC12A2","SLC12A4","SLC1A4","SLC20A1","SLC25A20","SLC25A22","SLC26A2","SLC27A2","SLC30A1","SLC35F6","SLC38A2","SLC39A10","SLC39A6","SLC4A1AP","SLC4A2","SLC4A7","SLC5A6","SLC7A2","SLC9A3R1","SLCO4A1","SLFN11","SMAD2","SMAP1","SMCHD1","SMPD4","SMS","SMYD5","SNAP23","SNAP29","SNTB2","SNX1","SNX17","SNX18","SNX24","SNX27","SNX3","SNX8","SNX9","SOWAHC","SPAG5","SPATA5","SPG21","SPTLC1","SPTLC2","SQSTM1","SRGAP1","SRGAP2","SRI","SRPK1","SRPK2","SSRP1","SSSCA1","SSX2IP","ST13","STAM2","STAT2","STAT5B","STAT6","STAU1","STIP1","STK10","STK24","STRN","STRN3","STRN4","STXBP1","SUGP2","SUGT1","SUN1","SUPT5H","SURF1","SURF2","SUZ12","SYNGR2","SYNPO","TACC1","TACC2","TACC3","TANC1","TANC2","TAOK1","TAOK3","TAP1","TBC1D2","TBC1D4","TBC1D5","TBCD","TBK1","TBL1XR1","TBL3","TCEB1","TCOF1","TCP1","TDRD3","TELO2","TFB1M","TFCP2","THOC3","THOP1","THRAP3","THUMPD3","TIMM50","TIPRL","TK1","TLDC1","TMEM189-UBE2V1","TMEM223","TMEM30A","TNPO2","TOMM22","TOR1B","TP53BP2","TPI1","TPM3","TPM4","TPX2","TRAF2","TRAF6","TRAFD1","TRIM23","TRIM26","TRIM33","TRIP12","TRMT1","TSNAX","TSSC1","TTC28","TTC37","TTK","TTPAL","TUBGCP2","TUBGCP3","TWISTNB","TXNRD1","TYK2","U2AF1","U2SURP","UACA","UBA5","UBAP2","UBAP2L","UBASH3B","UBE2E2","UBE2H","UBE2O","UBE4B","UCKL1","UFL1","UFM1","UGGT1","UIMC1","UPP1","UQCRQ","URB1","USP15","USP19","USP33","USP54","USP8","UTP18","UTP20","UTRN","VANGL1","VASP","VCL","VPS33B","VPS37C","VPS45","VPS4A","VPS51","VRK2","VTA1","WAC","WAPAL","WASF2","WASL","WBP2","WDHD1","WDR26","WDR6","WEE1","WFS1","WHSC1","WIPF2","WIZ","WNK1","WWC1","XAB2","XIAP","XPNPEP1","XRN1","YAP1","YES1","YKT6","YTHDC2","ZC3HAV1L","ZFR","ZFYVE16","ZNF185","ZNF207","ZNF318","ZNF326","ZNF598","ZPR1","ZW10","ZYX"],[true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,false,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true,true],[false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,true,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false,false]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>unique_sample2<\/th>\n      <th>unique_sample1<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[2]; $(this.api().cell(row, 2).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\nvar value=data[3]; $(this.api().cell(row, 3).node()).css({'color':value == true ? \"green\" : value == false ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
<pre class="r"><code>library(devtools)
# install_github(&quot;jokergoo/ComplexHeatmap&quot;)
# mat &lt;- as.matrix(assay(data_filt))
# mat_z &lt;- scale(mat)
# 
# col_split &lt;- c(&quot;T0&quot;, &quot;T0&quot;, &quot;T0&quot;, &quot;T0&quot;, &quot;T1&quot;, &quot;T1&quot;, &quot;T1&quot;, &quot;T1&quot;, &quot;T2&quot;, &quot;T2&quot;, &quot;T2&quot;, &quot;T3&quot;, &quot;T3&quot;, &quot;T3&quot;, &quot;T3&quot;)
# 
# ComplexHeatmap::Heatmap(mat_z,
#                         name = &quot;heatmap&quot;,
#                         show_row_names = FALSE,
#                         column_split = col_split,
#                         cluster_column_slices = FALSE,
#                         cluster_row_slices = FALSE,
#                         use_raster = FALSE)</code></pre>
</div>
<div id="differential-enrichment-analysis" class="section level2">
<h2><strong>4. Differential enrichment analysis</strong></h2>
<p>Protein-wise linear models combined with empirical Bayes statistics
are used for the differential enrichment analysis (or differential
expression analysis) and False discovery rate (FDR) is ajusted with
<em>Benjamini-Hochberg</em> procedure. Significant proteins are are
define with <strong>FDR = 0.05</strong> and <strong>Fold change =
1</strong></p>
<pre class="r"><code>data_diff &lt;- test_diff_BH(data_filt, type = &quot;manual&quot;, test = test)

dep &lt;- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results&lt;-get_results(dep)</code></pre>
<p>There are <strong>369</strong> significant proteins in
<strong>sample2</strong> vs <strong>sample1</strong> comparison.</p>
<p>The results from the previous analysis can be easily visualized by a
number of functions. These visualizations assist in the determination of
the optimal cutoffs to be used, highlight the most interesting samples
and contrasts, and pinpoint differentially enriched/expressed
proteins.</p>
<div id="visualization-of-the-results"
class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>4.1. Visualization of the
results</strong></h3>
<div id="pca" class="section level4">
<h4>PCA</h4>
<p>The PCA plot can be used to get a high-level overview of the data.
This can be very useful to observe batch effects, such as clear
differences between replicates.</p>
<pre class="r"><code>if(params$Imputation){
  if(nrow(dep) &gt;= 500){
    n_value &lt;- 500
  }else {
    n_value &lt;- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
} else {
  var &lt;- apply(assay(dep), 1, sd)
  df &lt;- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value &lt;- df %&gt;% as.data.frame() %&gt;% drop_na() %&gt;% nrow()
  if(n_value &gt;= 500){
    n_value &lt;- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = &quot;condition&quot;, label = T, label_size = 3)
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/PCA-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="volcano" class="section level4">
<h4>Volcano</h4>
<p>Volcano plots allows to inspect the enrichment of proteins between
the two samples (x axis) and their corresponding adjusted p value (y
axis).</p>
<pre class="r"><code>plot_volcano(dep, contrast=test, add_names=T, label_size=5, adjusted = F)</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/volcano-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="section" class="section level4">
<h4></h4>
</div>
<div id="section-1" class="section level4">
<h4></h4>
</div>
<div id="section-2" class="section level4">
<h4></h4>
</div>
</div>
<div id="result-table" class="section level3">
<h3><strong>4.2. Result table</strong></h3>
<p>In this table are summarized the results of DEP analysis, sorted by
p.value. Proteins that are UP regulated are colored with a red box in
Fold change column, in the orhter hand proteins that are DOWN regulated
are colored with blue.</p>
<pre class="r"><code>results %&gt;%
  select(starts_with(&quot;name&quot;) | starts_with(test)) %&gt;% 
  filter(across(ends_with(&quot;significant&quot;))) %&gt;% 
  arrange(across(ends_with(&quot;p.adj&quot;))) %&gt;%
  mutate(across(2:3, format, scientific = T, digits = 2)) %&gt;%
  dplyr::rename_with(~ tolower(gsub(paste0(test,&quot;_&quot;), &quot;&quot;, .x)), .cols = starts_with(test)) %&gt;% 
  mutate(significant = str_to_title(significant)) %&gt;%
  dplyr::rename(Fold.Change = ratio, Gene.name = name) %&gt;% 
  DT::datatable(options = list(
  columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:5)),
  pageLength = 10)) %&gt;%
  formatStyle(&#39;Fold.Change&#39;, backgroundColor = styleInterval(c(-1, 1), c(&#39;lightblue&#39;, &#39;white&#39;, &#39;tomato&#39;))) %&gt;% 
  formatStyle(&#39;significant&#39;, color = styleEqual(c(&quot;True&quot;, &quot;False&quot;), c(&#39;green&#39;, &#39;red&#39;)))</code></pre>
<div id="htmlwidget-a6aed18fefac875dc9ab" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-a6aed18fefac875dc9ab">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369"],["ACTN4","EGFR","PLS3","ANXA11","TAGLN2","AHNAK","CTTN","MISP","SPTBN1","EZR","NRD1","ACTN1.1","SEC16A","BAIAP2","ITGB1","FLNC","FLNB","RRM1","MYO1E","DIAPH1","EPB41L1","MSN","TNKS1BP1","USP10","MKI67","PCCA","IMPDH2","CTNND1","EDC4","GSPT1","TJP2","FAM120A","PLIN3","ATP1A1","TRIM25","EIF4G1","CSDE1","GART","PC","ANKS1B","ITGB4","HYOU1","ROCK2","DBNL.1","DDX3X","GEMIN5","TMEM263","PCBP1","AARS","FLNA.1","ZC3HAV1","LMNA","FBXO4","PKN2","ANLN","TSN","HNRNPH1.1","HLA-A","SMC2","PKP3","CALM2","EIF5A","MCCC1","CSTB","RTCB","FAM129B","PRKAB1","USP5","EPS8L2","LIMA1","YWHAQ","HCFC1","INF2","MYOF","NME1-NME2","NUMA1","CAPZB","GCLM","GLRX3","EIF4B","ERBB2IP","IQGAP1","RANBP2","UPF1","GPRC5A","HGSNAT","HUWE1","RAPH1","STAT5A","ATXN10","C3P1","PRDX1","FLOT1","HLA-B","SEC24C","ADAR","ANXA2","FAU","PKP2","CBS","FERMT2","PPP1CA","TLN1","CSPG4","NCL","STAT3","AFAP1","AGO2","RALB","DSP","CORO1B","ACACA","USP9X","ACLY","MAGEB2","PTBP1","TRIM29","STXBP3","CAPZA1","ACTA1","SERPINB1","FHOD1","DDX1","OLA1","VAV2","UGDH","FARSA","SLC3A2","H1F0","CLUH","CORO1C","EHD1","ILF2","PABPC4","S100A8","XIRP2","EIF3E","SRPR","DSG2","AP2B1","RECQL","PSMD7","ALDOA","EIF5B","EPHA2","DLGAP5","DUT","FARP1","GLYR1","JUP","ALPP","CHORDC1","TJP1","VARS","PDCD6","MYH14","PRDX6","TWF1","LGALS1","SLC25A12","A0A3B3IT21","CDK5RAP2","LARP4","TARBP1","SLK","RBBP7","GAPVD1","AIP","ARF6","MCMBP","UBE2N","B2M","S100A9","PDCD6IP","UBA6","DOCK7","IQGAP2","PCBP2.1","PABPC1","PRDX2","SPTAN1","ATP2B1","CAPZA2","KATNAL2","RAB5C","PTPN1","TRIP13","HDLBP","EPB41L1.1","CLNS1A","TFG","EXOC2","MYO1C","PTPN11","SAR1A","HNRNPK","KHSRP","PUS7","DNMT1","SPTBN5","LARS","GNAI2","HK1","CAND1","KRT18","LRRC40","GSTM3","SDHA","NUP155","HSPA8","SEPT9","SRRM2","ABCF2","RANBP9","PRDX4","STK38","ETF1","MYL6","SLC16A3","PPP2R1A","RBM39","CFL1","ACTR3","DIAPH3","SRSF3","RPL23A","SNRNP200","FXR1","ANXA7","GFPT1","COG7","SNRPD3","CKB","HSPA1B","NDUFA9","POLR2B","PPA1","RAC1","GDI2","PSME2","RTN4","EHMT1","GBE1","KPNB1","SYNRG","EIF2S1","ERH","GCN1L1","PRRC2C","ACACB","PRDX5","RPS10","SEPT2","OGT","ABCD3","VBP1","ZDHHC5","WDR44","UBE2Z","SF3B1","TNRC6B","CSRP2","EIF2AK2","GMPS","UHRF1","BAIAP2L1","MPZL1","VCP","LTA4H","PAPSS1","NPEPPS","SCAMP3","PSMC2","ANXA1","PFN1","ESYT2","CARM1","PPP1CC","RARS","TAF15","PSMD12","UBC","CAP1","EFTUD2","RAB6A","NOL10","CTSD","THOC2","HSPA4","SPCS2","STRAP","CPNE8","ABCF1","ANO10","SVIL","RBMX","COPE","MAP4","PLEC","RPS6KA3","ANKFY1","RAP1B","PRKDC","THOC6","MYBBP1A","MYH10","HARS","PARP1","RPS19","EIF2S3","NSUN2","ALG8","ACAT2","PARK7","QARS","RAP2C","ANXA8L2","PPP1R12A","TMPO.1","LARP1","SRSF7","AGPS","MATR3","ARHGAP35","MCM7","HNRNPC","SERPINB6","UBA1","LYN","MYH9","ACTR2","RAB7A","MET","RAB14","STAT1","FEN1","ARMC6","ANXA6.1","G6PD","STAM","CACYBP","CIT","SMARCA5","RPLP1","SYNCRIP","LRRC59","MEPCE","MTHFD1","NAMPT","TM9SF3","CAV1","HSPA4L","NPM1","BLMH","EPB41L2","KDM1A","SRPRB","MRPL4","SH3PXD2B","DPYSL2","NOL11","PPIA","NUP205","AP3D1","ATP11A","TOMM40","MGST3","VIM","KIF11"],["1.6e-12","6.9e-11","6.2e-10","2.4e-09","3.0e-09","3.7e-09","8.6e-09","9.0e-09","7.1e-09","1.1e-08","1.1e-08","1.6e-08","2.0e-08","3.1e-08","3.8e-08","4.3e-08","5.1e-08","5.2e-08","8.0e-08","9.4e-08","1.0e-07","9.8e-08","9.0e-08","1.2e-07","1.3e-07","1.3e-07","1.5e-07","2.9e-07","3.0e-07","3.3e-07","4.3e-07","4.9e-07","4.9e-07","5.7e-07","6.3e-07","6.7e-07","8.5e-07","9.5e-07","9.8e-07","1.0e-06","1.3e-06","1.4e-06","1.4e-06","1.4e-06","1.6e-06","1.5e-06","1.5e-06","2.0e-06","2.1e-06","2.1e-06","2.1e-06","2.5e-06","2.9e-06","2.8e-06","3.1e-06","3.1e-06","3.7e-06","4.3e-06","4.3e-06","4.4e-06","4.6e-06","5.4e-06","5.9e-06","6.6e-06","6.6e-06","7.1e-06","7.3e-06","7.5e-06","8.3e-06","9.1e-06","1.1e-05","1.3e-05","1.3e-05","1.3e-05","1.4e-05","1.3e-05","1.7e-05","1.7e-05","1.6e-05","1.7e-05","1.8e-05","1.8e-05","1.8e-05","1.8e-05","1.9e-05","2.0e-05","1.9e-05","2.0e-05","1.9e-05","2.2e-05","2.3e-05","2.6e-05","2.7e-05","2.7e-05","2.8e-05","3.0e-05","3.1e-05","3.3e-05","3.4e-05","3.5e-05","3.6e-05","3.7e-05","4.2e-05","4.4e-05","4.5e-05","4.8e-05","5.1e-05","5.4e-05","5.4e-05","5.8e-05","6.1e-05","6.5e-05","6.8e-05","7.4e-05","7.3e-05","7.2e-05","7.3e-05","8.3e-05","8.4e-05","8.7e-05","9.3e-05","9.8e-05","1.1e-04","1.1e-04","1.2e-04","1.2e-04","1.3e-04","1.3e-04","1.3e-04","1.4e-04","1.5e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.6e-04","1.7e-04","1.8e-04","1.8e-04","1.9e-04","2.0e-04","2.0e-04","2.0e-04","2.1e-04","2.1e-04","2.1e-04","2.1e-04","2.2e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.4e-04","2.4e-04","2.5e-04","2.5e-04","2.7e-04","2.7e-04","2.8e-04","2.9e-04","2.9e-04","3.0e-04","3.0e-04","3.0e-04","3.1e-04","3.2e-04","3.3e-04","3.4e-04","3.4e-04","3.6e-04","3.7e-04","3.8e-04","4.2e-04","4.3e-04","4.3e-04","4.5e-04","4.5e-04","4.5e-04","4.9e-04","4.8e-04","4.8e-04","4.9e-04","5.1e-04","5.6e-04","6.5e-04","6.7e-04","6.9e-04","6.9e-04","7.0e-04","7.2e-04","7.2e-04","7.4e-04","7.9e-04","7.9e-04","7.8e-04","8.1e-04","8.1e-04","8.5e-04","8.7e-04","9.0e-04","9.1e-04","9.3e-04","9.6e-04","9.9e-04","9.9e-04","1.0e-03","1.0e-03","1.1e-03","1.1e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.2e-03","1.2e-03","1.3e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.1e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.4e-03","2.4e-03","2.5e-03","2.7e-03","2.7e-03","2.6e-03","2.8e-03","2.8e-03","2.8e-03","2.9e-03","3.0e-03","3.0e-03","3.1e-03","3.1e-03","3.2e-03","3.2e-03","3.4e-03","3.3e-03","3.4e-03","3.5e-03","3.5e-03","3.5e-03","3.6e-03","3.6e-03","3.6e-03","3.7e-03","3.7e-03","3.8e-03","4.0e-03","4.1e-03","4.2e-03","4.2e-03","4.2e-03","4.3e-03","4.3e-03","4.3e-03","4.4e-03","4.4e-03","4.5e-03","4.5e-03","4.6e-03","4.7e-03","4.7e-03","4.7e-03","4.8e-03","4.9e-03","4.8e-03","4.9e-03","4.9e-03","5.0e-03","5.1e-03","5.2e-03","5.4e-03","5.6e-03","5.7e-03","5.8e-03","5.8e-03","6.2e-03","6.3e-03","6.5e-03","6.6e-03","6.7e-03","7.0e-03","7.0e-03","7.1e-03","7.2e-03","7.1e-03","7.2e-03","7.3e-03","7.6e-03","7.5e-03","7.7e-03","7.9e-03","7.9e-03","8.0e-03","8.4e-03","8.5e-03","8.4e-03","8.7e-03","8.9e-03","9.0e-03","9.1e-03","9.3e-03","9.5e-03","9.8e-03","9.9e-03","9.9e-03","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02"],["2.0e-09","4.3e-08","2.6e-07","7.4e-07","7.4e-07","7.8e-07","1.2e-06","1.2e-06","1.2e-06","1.3e-06","1.3e-06","1.7e-06","1.9e-06","2.7e-06","3.1e-06","3.3e-06","3.7e-06","3.7e-06","5.3e-06","5.6e-06","5.6e-06","5.6e-06","5.6e-06","6.0e-06","6.3e-06","6.3e-06","7.0e-06","1.3e-05","1.3e-05","1.4e-05","1.7e-05","1.8e-05","1.8e-05","2.1e-05","2.3e-05","2.3e-05","2.9e-05","3.1e-05","3.1e-05","3.3e-05","4.0e-05","4.0e-05","4.0e-05","4.0e-05","4.1e-05","4.1e-05","4.1e-05","5.2e-05","5.2e-05","5.2e-05","5.2e-05","6.1e-05","6.7e-05","6.7e-05","7.0e-05","7.0e-05","8.1e-05","9.2e-05","9.2e-05","9.3e-05","9.5e-05","1.1e-04","1.2e-04","1.3e-04","1.3e-04","1.4e-04","1.4e-04","1.4e-04","1.5e-04","1.6e-04","1.9e-04","2.2e-04","2.2e-04","2.2e-04","2.3e-04","2.3e-04","2.7e-04","2.7e-04","2.7e-04","2.7e-04","2.7e-04","2.7e-04","2.7e-04","2.7e-04","2.8e-04","2.8e-04","2.8e-04","2.8e-04","2.8e-04","3.1e-04","3.2e-04","3.5e-04","3.6e-04","3.7e-04","3.8e-04","4.0e-04","4.1e-04","4.2e-04","4.3e-04","4.4e-04","4.4e-04","4.6e-04","5.1e-04","5.4e-04","5.4e-04","5.7e-04","5.9e-04","6.2e-04","6.2e-04","6.6e-04","6.9e-04","7.3e-04","7.5e-04","7.9e-04","7.9e-04","7.9e-04","7.9e-04","8.9e-04","8.9e-04","9.1e-04","9.6e-04","1.0e-03","1.1e-03","1.1e-03","1.2e-03","1.2e-03","1.3e-03","1.3e-03","1.3e-03","1.4e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.5e-03","1.6e-03","1.6e-03","1.7e-03","1.7e-03","1.7e-03","1.7e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.8e-03","1.9e-03","1.9e-03","1.9e-03","2.0e-03","2.0e-03","2.1e-03","2.1e-03","2.2e-03","2.2e-03","2.2e-03","2.3e-03","2.3e-03","2.3e-03","2.4e-03","2.4e-03","2.5e-03","2.5e-03","2.5e-03","2.6e-03","2.7e-03","2.7e-03","3.0e-03","3.1e-03","3.1e-03","3.1e-03","3.1e-03","3.1e-03","3.3e-03","3.3e-03","3.3e-03","3.3e-03","3.5e-03","3.7e-03","4.3e-03","4.4e-03","4.6e-03","4.6e-03","4.6e-03","4.7e-03","4.7e-03","4.8e-03","5.0e-03","5.0e-03","5.0e-03","5.1e-03","5.1e-03","5.3e-03","5.4e-03","5.6e-03","5.6e-03","5.7e-03","5.8e-03","6.0e-03","6.0e-03","6.1e-03","6.2e-03","6.3e-03","6.4e-03","6.5e-03","6.6e-03","6.7e-03","6.8e-03","6.9e-03","7.1e-03","7.2e-03","8.2e-03","8.3e-03","8.3e-03","8.5e-03","8.9e-03","8.9e-03","9.2e-03","9.3e-03","9.5e-03","9.6e-03","9.7e-03","9.7e-03","9.8e-03","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.0e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.1e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.2e-02","1.3e-02","1.3e-02","1.3e-02","1.3e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.4e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.5e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.6e-02","1.7e-02","1.7e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.8e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","1.9e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.0e-02","2.1e-02","2.1e-02","2.1e-02","2.2e-02","2.2e-02","2.3e-02","2.3e-02","2.3e-02","2.4e-02","2.5e-02","2.5e-02","2.6e-02","2.6e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.7e-02","2.8e-02","2.8e-02","2.8e-02","2.8e-02","2.9e-02","2.9e-02","2.9e-02","3.1e-02","3.1e-02","3.1e-02","3.2e-02","3.2e-02","3.2e-02","3.3e-02","3.3e-02","3.4e-02","3.5e-02","3.5e-02","3.5e-02","3.5e-02","3.6e-02","3.8e-02","3.8e-02","3.8e-02","3.8e-02","3.9e-02","3.9e-02","3.9e-02","4.0e-02","4.0e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.1e-02","4.2e-02","4.2e-02","4.2e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.5e-02","4.5e-02","4.6e-02","4.6e-02","4.6e-02","4.7e-02","4.7e-02","4.7e-02","4.9e-02"],["True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True","True"],[9.73,5.66,7.68,6.06,5.92,5.89,10,5.46,5.35,6.37,4.08,7.83,4.56,4.99,7.32,6.29,5.91,3.09,3.55,2.83,4.33,3.74,3.67,3.52,5.53,-2.6,3.8,6.24,3.67,3.86,5.02,2.5,3.49,3.98,3.5,3.68,2.44,3.4,-2.36,-4.66,4.88,-2.74,4.47,3.18,2.24,3.47,-1.83,3.21,2.55,6.98,3.85,3.68,-3.6,3.48,3.96,4.1,4.27,3.6,2.48,4.84,3.5,3.16,-3.13,4.34,2.34,3.99,-2.1,3.03,2.37,4.36,3.05,2.5,2.7,2.93,2.23,3.83,3.12,2.77,2.77,2.07,4.21,3.65,2.92,2.36,5.45,-3.4,3.77,-1.79,2.71,2.09,-2.52,2.78,2.16,2.31,1.78,2.97,4.18,-2.65,3.5,2.56,3.51,3.05,2.61,1.8,-2.55,4.18,3.32,-2.76,2.71,4.82,3.35,-1.88,2.38,3.39,3.27,2.28,2.77,3.02,1.29,2.66,1.59,-1.26,3.09,1.82,2.15,2.2,2.36,2.25,-2.9,1.68,2.31,1.74,1.76,2.86,-4.63,-2.18,1.48,2.35,-2.27,1.42,1.71,-3.02,4.51,2.36,3.34,-1.88,-2.54,2.94,-1.71,3.97,3.59,1.57,2.34,1.35,1.31,-2.17,1.28,1.51,2.05,-2.08,-1.31,2.65,2.3,2.65,-1.29,2.35,2.53,2.71,1.77,2.11,-2.47,4.07,-5,2.87,1.89,1.58,1.89,1.11,1.87,2.62,4.99,2.82,1.13,1.87,1.19,1.51,1.65,1.28,2.79,1.78,2.23,-2.13,1.3,1.73,1.24,1.95,1.75,1.25,1.15,-2.39,1.36,1.44,1.05,1.61,3.1,1.88,1.13,-2.17,1.88,3.67,1.28,2.07,2.69,-2.04,1.81,-2.29,1.17,2.07,2.28,1.99,-1.13,2.91,1.69,1.79,-1.53,-1.4,1.94,1.45,2.63,1.16,-1.46,-1.21,2.42,2.32,1.29,1.12,1.29,1.85,2.63,1.7,-2.15,-2.64,1.76,2.87,-1.79,1.07,-2.02,2.38,1.62,-3.2,1.06,-1.22,1.3,1.12,1.14,-2.26,1.41,2.31,1.04,1.26,-1.86,-2.13,1.58,1.57,1.18,1.41,1.54,2.38,1.54,1.15,1.07,1.44,1.55,-1.55,1.44,1.38,-1.78,1.51,1.02,2.23,1.25,1.31,1.21,1.41,1.02,-1.33,-1.09,1.57,2.01,1.36,1.2,-1.57,1.41,2.92,-1.57,-1.71,1.61,2.82,2,1.49,-1.44,1.13,1.57,-1.54,1.22,1.4,1.33,2.03,-2.57,1.09,1.7,-1.58,2.55,1.16,-3.24,1.37,3.18,1.69,1.39,1.68,-2.14,-1.51,1.06,1.67,1.15,-1.35,1.37,1.21,-3.5,1.33,1.04,1.14,1.6,1.36,1.29,-2.48,-1.4,1.98,1.19,-1.85,1.29,-1.55,1.03,-1.36,1.03,-1.68,1.74,1.97,1.87,-1.08,2.22,1.29,1.06,-1.37,1.36,-1.04,1.32,-1.31,-1.19,1.48,-2.33,1.56,1.22,1.09,1.34,-1.28,1.2,2,1.12]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Gene.name<\/th>\n      <th>p.val<\/th>\n      <th>p.adj<\/th>\n      <th>significant<\/th>\n      <th>Fold.Change<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[5]; $(this.api().cell(row, 5).node()).css({'background-color':isNaN(parseFloat(value)) ? '' : value <= -1 ? \"lightblue\" : value <= 1 ? \"white\" : \"tomato\"});\nvar value=data[4]; $(this.api().cell(row, 4).node()).css({'color':value == \"True\" ? \"green\" : value == \"False\" ? \"red\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
</div>
<div id="gene-ontology" class="section level2">
<h2><strong>5. Gene Ontology</strong></h2>
<p>Now I perfom a gene onology analysis (GO or by branch <a href="GO:MF"
class="uri">GO:MF</a>, <a href="GO:BP" class="uri">GO:BP</a>, <a
href="GO:CC" class="uri">GO:CC</a>) and a KEGG ontology.<br />
The enrichment results are visualized with an interactive
<strong>Manhattan-like-plot</strong> named “Gostplot”.</p>
<div id="sample2-1" class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>5.1. sample2</strong></h3>
<div id="gostplot" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in sample2 were used to generate these gene
ontologies. In this case we also use the specific unique genes
identify.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &gt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_1)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-674493e088ef4386f8d8" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-674493e088ef4386f8d8">{"x":{"data":[{"x":[86.5836916874251,86.6343397100053,135.977201708256,105.10917994654,105.136451958698,124.682692672889,152.180672932159,124.698276679837,97.2002964205664,88.9563567452171,172.034697783567,86.7512197621132,111.062270600573,108.86882162268,111.070062604046,85.9174753904097,136.37848988716,85.9291633956205,135.98499371173,101.579402372879,137.543394406503,136.172001795103,133.039616398609,160.288252546716,104.72737177632,124.406076549567,133.024032391662,147.727542946845,123.13208398159,134.901905228863,115.971232789108,86.6460277152161,124.41386855304,137.099250208493,117.814041610677,84.7252988589083,136.382385888897,124.651524658993,75.66709482054,114.093359951906,149.340487665935,91.6251179350159,108.876613626153,76.4034391488203,89.4628369710183,86.3733075936308,189.500473570236,97.5782085890489,177.126772053738,137.574562420399,105.113075948277,84.5811467946418,176.604707820989,97.7106726481046,110.586958388667,89.4667329727552,136.546017961849,97.8665127175819,110.890846524148,107.392236964382,147.723646945108,87.1408199358065,134.913593234074,93.8419429233305,136.343425871528,113.368703628837,175.377467273856,163.946598177695,134.216208923163,113.563503715683,133.043512400346,126.482645475352,124.6943806781,75.6086547944861,87.4992520956043,141.89133234492,137.601834432557,133.238312487193,112.289511147706,131.473423700362,100.42228985701,136.152521786418,137.792738517667,122.765859818318,104.762435791953,137.539498404766,134.266856945743,137.660274458611,110.840198501567,86.4434356248956,107.376652957434,91.8277100253364,132.887672330869,84.916202944018,109.507765907536,105.000091897905,172.073657800937,122.35677963594,136.386281890634,96.0392879029605,84.565562787694,108.872717624416,84.7369868641191,172.042489787041,107.532493026912,93.873110937226,85.9837074199375,136.148625784681,107.36886095396,86.6538197186899,87.5421081147106,119.847754517356,163.541413997054,151.896264805363,90.8731895997879,156.458482839311,149.679439817048,76.4112311522942,106.796148698631,91.621221933279,93.5770148052191,100.211905763215,84.569458789431,123.642460209128,107.205228881009,134.262960944006,93.7133748660118,87.1057559201741,97.1769204101448,126.580045518775,123.646356210865,133.518824612252,91.6329099384898,110.590854390404,151.884576800152,86.7551157638502,165.559542896786,133.768168723416,105.268916017754,136.140833781208,137.800530521141,110.953182551938,93.8458389250675,124.429452559988,84.5304987720617,134.224000926637,86.7862837777456,86.3304515745245,179.822805255696,134.898009227126,135.244753381713,147.719750943371,120.665914882111,87.109651921911,110.957078553675,95.1393115017291,89.077132799062,165.937455065268,114.171279986645,76.422919157505,135.673313572776,177.97610043239,144.408149466979,146.363942338919,105.003987899642,163.950494179432,93.5692228017453,128.368310316027,113.559607713946,124.70606868331,160.483052633562,101.213178209607,133.027928393398,135.981097709993,136.425241908004,84.6746508363282,107.400028967856,91.9874460965506,136.448617918425,114.194655997066,130.939671462403,107.540285030386,137.107042211967,122.450283677626,87.5031480973412,164.612814474711,84.5266027703247,134.048680848475,142.939356812155,104.598803719001,104.813083814533,87.9706683057731,189.629041627555,105.646828186236,142.658844687096,174.037242676351,124.425556558251,150.559936209595,108.88050962789,149.765151855261,107.446780988699,122.442491674153,88.9719407521648,122.461971682837,110.933702543254,120.673706885585,144.427629475663,107.40782097133,94.9289274079347,176.608603822726,122.208731569937,123.37363608928,86.7434277586394,104.801395809322,75.3320386711638,132.092887976535,176.409907734143,140.617339776943,120.054242609413,123.369740087543,171.306145458761,163.942702175958,90.7251415337845,105.323460042071,124.40218054783,76.3917511436095,122.438595672416,133.643496667834,113.37649563231,120.482802800476,136.320049861106,137.10314621023,111.829782942748,119.851650519092,92.7705424456741,122.465867684574,105.284500024702,127.62806998601,135.778505619673,188.635561184637,87.1486119392804,77.7865197654314,176.612499824463,137.590146427346,161.344069017424,138.478434823367,146.500302399711,109.593477945749,136.67069001743,93.8029829059612,135.755129609251,163.931014170748,138.90699501443,142.436772588091,137.742090495087,98.8677851639736,105.097491941329,86.2720115484705,109.5038699058,122.742483807896,125.769677157493,135.751233607514,125.777469160967,136.678482020904,107.536389028649,132.330544082488,86.2798035519444,134.909697232337,87.1641959462281,106.511740571835,98.1587128478518,100.430081860484,75.6787828257508,107.454572992173,86.7395317569025,89.6927010734973,75.6748868240139,172.471049978104,91.0524056796868,136.471993928847,77.6423677011649,128.36441431429,133.522720613989,93.5731188034822,77.7943117689052,186.905736413439,109.519453912747,163.927118169011,87.7719722171896,160.923300829836,137.644690451664,185.725247887149,105.116971950013,107.719501110284,109.227253782477,85.7889073330909,124.674900669415,151.241736513558,157.37404324749,172.077553802674,88.9135007261108,107.883133183236,105.218267995174,84.744778867593,141.564068199018,172.467153976367,106.394860519727,114.206344002277,189.504369571973,105.148139963909,105.650724187973,86.7590117655871,174.029450672877,122.469763686311,87.1291319305957,101.941730534413,150.969016391973,123.732068249077,144.412045468715,172.038593785304,105.167619972594,108.888301631364,113.372599630574,124.65542066073,122.302235611623,78.2189759582309,89.4706289744922,157.603907349969,189.636833631029,123.52558015702,124.982684806633,110.614230400825,107.976637224922,97.5821045907858,114.654384202024,145.230205833471,93.814670911172,161.453157066059,91.3134377960613,136.557705967059,102.58067481927,84.8928269335964,113.918039873744,106.507844570098,136.51874594969,157.611699353443,142.199116482138,86.2291555293643,105.12086795175,154.572817998636,137.410930347448,136.717442038274,130.99421548672,134.621393103804,134.617497102067,102.140426622997,123.502204146598,136.456409921899,97.7145686498415,77.170951490996,92.7237904248309,133.440904577513,119.871130527777,133.004552382977,148.865175454029,86.3226595710507,136.133041777734,123.178836002433,105.01957190659,136.436929913214,177.983892435864,173.573618469656,88.0291083318271,107.372756955697,136.351217875002,123.08143595901,173.168434289015,126.809909621254,114.007647913694,131.726663813263,91.6017419245943,76.5047351939806,136.144729782944,185.717455883675,150.805384319022,106.398756521464,98.9301211917645,106.983156782004,165.941351067005,173.164538287278,150.930056374604,98.5755850337036,110.547998371297,109.235045785951,86.3265555727876,147.045742642882,93.8770069389629,86.2875955554183,175.381363275593,104.797499807585,86.2759075502075,120.307482722314,158.262331643511,87.1564039427542,91.8355020288103,144.930213699727,75.4878787406411,91.0757816901084,136.713546036537,107.715605108547,120.669810883848,93.7289588729595,112.616775293609,163.436221950157,122.843779853057,159.727228296597,97.2158804275142,124.709964685047,131.126679545775,126.533293497932,91.3056457925874,122.734691804423,173.561930464445,77.8449597914854,134.547369070802,181.770806124162,181.763014120688,136.362905880213,89.1160928164313,123.120395976379,172.330793915574,105.144243962172,122.617811752315,134.878529218441,133.24220848893,91.8316060270733,176.028099563923,147.606766893,131.153951557934,105.023467908327,77.112511464942,136.335633868054,172.326897913837,110.551894373034,81.9435536187384,97.2197764292511,84.5343947737986,134.995409270549,89.1044048112205,178.170900519236,93.8809029406999,88.0018363196686,164.30892633923,95.4003436181036,86.2525315397859,154.565025995162,175.385259277329,97.8236566984756,110.544102369561,91.9952381000245,136.553809965322,87.3823720434963,135.685001577986,127.643653992958,128.087798190968,128.726742475825,124.963204797948,121.507451257289,93.8185669129089,134.9058012306,112.612879291872,86.2836995536813,89.8134771273423,105.152035965646,88.0407963370379,93.3198786905816,130.573447299131,83.7123384073058,137.095354206756,90.230349313194,164.305030337493,133.047408402083,95.0458074600427,141.817308311918,156.532506872313,151.245632515295,89.2290768668024,158.071427558401,146.332774325023,127.854038086752,136.705754033063,173.19960230291,137.578458422136,75.6904708309616,90.7017655233629,84.7019228484867,106.406548524938,96.9392643041919,154.346849897894,115.301120490355,90.7757895563646,86.9031638298536,121.304859166968,189.293985478179,89.7199730856559,136.308361855896,125.952789239129,108.732461561887,177.247548107583,106.987052783741,144.567885538193,157.381835250964,107.010428794163,107.396132966119,193.634131413122,164.262174318387,164.266070320124,164.269966321861,164.25827831665,126.303429395453,111.073958605783,136.581081977481,172.478841981578,109.383093851955,166.856911475184,122.656771769684,126.299533393716,124.733340695469,91.617325931542,144.443213482611,132.112367985219,149.605415784047,106.476676556203,86.6655077239007,125.009956818791,106.402652523201,97.7808006793694,93.3822147183725,81.3942173738309,111.334990722158,137.570666418662,109.348029836322,128.341038303869,135.887593668307,133.246104490667,132.143535999115,132.147432000852,119.808794499986,136.522641951427,159.703852286176,123.498308144861,100.28203379448,163.938806174222,146.862630561246,87.1797799531758,177.138460058949,130.086447082014,119.832170510408,172.474945979841,78.5267600954486,173.577514471393,75.2463266329513,135.891489670044,78.5189680919747,183.808415032578,89.8212691308161,115.305016492092,138.190130694834,157.525987315231,108.440261431617,91.8705660444427,154.701386055954,76.9878394093602,112.873911408246,156.53640287405,106.998740788952,97.6210646081551,165.917975056583,81.3085053356184,180.052669358175,91.6290139367528,106.994844787215,90.4524214121992,154.697490054217,112.908975423879,75.3437266763746,136.565497970533,126.829389629939,89.8680211516593,156.859771018215,157.545467323915,89.8758131551332,93.7172708677487,193.645819418332,76.4423991661896,76.4307111609788,86.6148597013206,118.846482070964,77.7904157671683,137.722610486402,87.5070440990782,86.2564275415228,171.056801347597,89.7082850804451,94.3211511369732,176.035891567397,83.4435142874575,84.740882865856,127.850142085015,105.272812019491,106.768876686473,113.547919708736,109.390885855428,93.8263589163828,137.114834215441,94.4691992029767,106.655892636102,167.565983791306,86.5213556596342,78.0553438852797,84.8811389283856,86.8992678281167,105.942924318243,153.965041727674,90.234245314931,161.531077100797,89.7238690873928,124.916452777105,150.688504266914,123.186628005907,107.033804804584,161.492117083428,86.5875876891621,110.73500645467,78.4878000780792,141.229012049642,144.232829388817,150.758632298179,128.091694192705,98.5872730389144,124.624252646835,136.452513920162,109.316861822427,111.42070276037,136.561601968796,177.239756104109,98.9418091969753,95.0068474426734,158.297395659143,162.808965670511,150.201504049797,102.300162694211,111.323302716947,119.836066512145,133.760376719942,83.7162344090428,134.052576850212,113.571295719157,111.07785460752,105.950716321717,125.808637174862,84.0785625705775,189.632937629292,191.491330457809,156.863667019952,97.975600766216,106.414340528412,89.8524371447116,93.5497427930606,137.54729040824,136.136937779471,144.345813439188,86.5369396665819,89.7433490960775,145.588637993269,144.848397663252,105.187099981278,93.6666228451686,124.417764554777,93.409486730531,178.17869252271,159.731124298334,150.692400268651,108.736357563624,114.066087939748,167.57377579478,141.232908051378,77.3228955587364,158.963611956159,186.913528416913,136.043433737784,84.5188107668509,84.5772507929048,120.050346607676,108.054557259661,86.8603078107473,154.35074589963,89.4745249762291,136.413553902793,132.260416051223,123.630772203917,136.36680188195,112.877807409983,141.677052249389,114.069983941484,84.686338841539,123.537268162231,176.982619989472,94.3250471387102,188.573225156846,123.124291978116,135.669417571039,177.979996434127,93.5458467913237,149.554767761466,157.023403091166,75.499566745852,105.65462018971,89.9459411863979,173.967114645086,107.403924969593,162.875197700039,123.295716054541,115.153072424352,144.225037385343,85.2512590933942,75.3671026867962,156.964963065112,109.336341831111,95.9574718664849,88.7771406653182,91.6056379263312,149.609311785783,112.803783376981,106.418236530149,173.565826466182,107.380548959171,97.2938004622528,150.178128039376,86.9070598315905,150.174232037639,140.383579672727,105.939028316506,136.41744990453,108.884405629627,136.514849947953,135.872009661359,192.134170744403,157.054571105062,158.620763803309,78.7800002083492,92.0069261052353,107.388340962645,159.816836336547,113.594671729579,149.165167587773,87.4486040730242,107.840277164129,120.062034612887,89.8797091568701,115.285536483407,170.787977227749,108.444157433354,123.954140348082,117.221849346663,131.227975590936,136.312257857633,124.928140782316,83.7668824316229,91.8549820374949,123.404804103175,111.085646610994,122.559371726261,164.979038637983,76.3761671366618,181.338349931362,181.330557927889,193.50945935754,163.405053936262,84.3863467077952,193.704259444386,88.3524764759925,128.025462163177,95.0497034617796,131.648743778524,93.8068789076981,116.173824879428,88.3914364933619,193.638027414859,102.806642920012,125.96447724434,113.70765577995,170.674993177378,91.6134299298051,142.093924435241,149.745671846576,87.1525079410173,92.0186141104461,144.630221565984,89.2173888615916,94.9484074166194,150.762528299916,76.0021509699163,150.836552332917,148.113247118801],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.9475218188085,15.7783967073921,15.6789447391496,15.6297398211967,15.6153134891971,15.5390437108363,15.3263397746936,15.0922818848249,14.9439190875542,14.9428051062969,14.8229246193952,14.8220118304258,14.7398557699069,14.7226386262663,14.7217733198388,14.6757383644678,14.4543191294978,14.4139138608968,14.3702868781458,14.3436793146494,14.2739480974318,14.2196028110833,14.1547958819331,14.1104797715849,14.062954088279,14.0286371653255,13.9849296562928,13.9849296562928,13.9179652671023,13.8846941246659,13.8622007229981,13.8311191002183,13.7268833984704,13.6589258843916,13.5769644957726,13.47916261078,13.4468700844655,13.4263778032931,13.3631593596244,13.3582337903217,13.3280779814495,13.1244218424558,13.1155684024673,13.0071554193329,12.9876242867451,12.9536514886656,12.6563573148575,12.5948973653924,12.4933027168617,12.4066369252734,12.1121472578015,11.9581330296562,11.8872545872175,11.8523215507847,11.798602063289,11.7621505963698,11.7365582783022,11.7049431975829,11.4832931665447,11.3881064309889,11.3871805222818,11.3306941252622,11.3145067470092,11.3017719027532,11.2918175843958,11.2686531545631,11.0300311313103,11.0239940862256,11.0066821499122,10.9690983214307,10.954279507292,10.8986517931237,10.8720902163109,10.8534109249442,10.8425692837063,10.7224159294511,10.7192628770324,10.6863861274778,10.6574488357774,10.6349128432505,10.5890857096822,10.5478519411934,10.5039640361518,10.4915032822044,10.4685816363863,10.4637962751562,10.4156819744214,10.4130372994735,10.3886468409117,10.3882748398913,10.366287629633,10.2672186427324,10.2501825056896,10.2089680614028,10.208240588217,10.1680657353021,10.1329228331846,10.1090903214911,10.1087809741196,10.1013192831679,10.0594436337925,9.95537485926915,9.89317717578536,9.80587544227663,9.77031031537739,9.73981654982781,9.73233347710244,9.73233347710244,9.62145135374163,9.55669339105346,9.47815677813536,9.47168543241771,9.46678783416493,9.46097491194403,9.44535715625539,9.43150057719495,9.41691267983661,9.38261299595317,9.32705933198043,9.31658279173575,9.28095784990446,9.22012781555476,9.19810340888498,9.19364213428509,9.17579493088669,9.0451550414749,9.02914175087995,9.02900572993511,8.95144903496606,8.88680342495951,8.86853056367502,8.83997375276968,8.8271514313214,8.82282150406644,8.82035554022461,8.80113249305931,8.78384545645901,8.77931256239877,8.76399686110297,8.75212290643585,8.74092494645784,8.71133028799684,8.45104808635377,8.42309732313101,8.37595884226694,8.31682626414799,8.28922962749215,8.26458376045364,8.23925619649972,8.16087902889135,8.13373903269112,8.12178967607947,8.11411584893963,8.09423761604612,8.04582606347779,8.02810707954808,8.01055373984215,8.00273747127698,7.99546074931565,7.94841692422819,7.92934079990366,7.92477234612648,7.8318765255546,7.81061115545321,7.79432472299892,7.79172675877155,7.75235945686681,7.70543796572067,7.70049202227302,7.69578329004085,7.66658044063909,7.64738527979429,7.61030672152005,7.60914368380914,7.58871763449947,7.56877187588193,7.43296322233769,7.43294922222722,7.41663296939328,7.41418215125528,7.4038201862313,7.37616440667643,7.37287668718504,7.34282778678806,7.27728953369401,7.26258253995416,7.24736643346489,7.19960965723121,7.18122725199434,7.15527839805546,7.08726581801554,7.07799950260723,7.03072758231432,6.97342148671273,6.92691815752622,6.86112453774011,6.80504461498427,6.80108565551369,6.78226293150149,6.75281317414546,6.73548110442483,6.72976005964132,6.68957934298071,6.68344998177636,6.6713952567646,6.64296160604563,6.63665317574682,6.52922451196104,6.52559673449037,6.49320076628921,6.45944823165368,6.45807109693801,6.45244307639996,6.44038038378237,6.42562012276543,6.4061611769272,6.39378716772666,6.3627337318886,6.31806582227818,6.31333536645316,6.29485096570594,6.26968893360937,6.25274316557333,6.23296520066635,6.23019098734409,6.22009975187851,6.20396906877073,6.17913170818015,6.16820890583458,6.09635947574161,6.09304026139584,6.09014980376722,6.08660688308419,6.08068322335489,6.07908983537642,6.07368059447481,6.06548193239936,6.05416468333446,6.04200401213977,6.03267763939406,6.01168369330233,6.00170326720939,5.99605299105875,5.97898305082342,5.97668378696003,5.968331240676,5.95680991637375,5.89688396566869,5.88029014602056,5.84448705746758,5.83299249882525,5.79094178974558,5.7690372921723,5.75612427012699,5.75113828432537,5.75035810279852,5.72522666276418,5.71944105623145,5.71672218031421,5.70642062112945,5.65197926021048,5.64744541536172,5.64096670233366,5.6216416793819,5.60969831533582,5.58205056775276,5.57247238657599,5.50662944785091,5.50476033577423,5.50436141725502,5.49679340049875,5.48122783237211,5.43553144692462,5.40368414277136,5.39890605712201,5.39890605712201,5.37684553256969,5.35592531722065,5.33749591886841,5.32738189489288,5.32548041862859,5.28941304727882,5.27527549572748,5.26983903713839,5.26849663100648,5.24638894452049,5.22144288036276,5.20964763141659,5.19708842301426,5.18774854600032,5.18089083878156,5.17154761599988,5.15607537266441,5.14590414229086,5.14590414229086,5.11157672771018,5.08915834006545,5.08542729432943,5.047878299676,5.04721051292247,5.0459531503148,5.03016231149477,5.02703094762974,5.02563756427376,5.01597653343076,5.01414836329154,5.01414836329154,5.01015088338433,5.00261994770286,4.99317200042849,4.98619388315484,4.98511956955502,4.98365816342191,4.97480595872244,4.97344793745117,4.97299787522553,4.96813652680901,4.93619196885611,4.90992123549225,4.90903139159429,4.89813324221742,4.85066185770116,4.84262091008387,4.83428420998799,4.79273865879896,4.79159169859485,4.78735966970055,4.7742270009115,4.72376034669725,4.72083090755625,4.70687166593377,4.70221685279848,4.68465586048157,4.67460302482064,4.61767850139287,4.61767850139287,4.58742456207092,4.58149952222224,4.56119084716264,4.54369910951301,4.54218985017697,4.52204169444755,4.51067246999524,4.49841955567016,4.49030083395266,4.48063778260193,4.47238942135928,4.4401919540879,4.42308839982427,4.4162394202989,4.40359878134885,4.39492653323524,4.3828346029139,4.38071201274036,4.38046526479165,4.37525107357042,4.36640735436496,4.34436081159501,4.32878620802762,4.32595346541051,4.30945881219955,4.30194406229883,4.28132399454267,4.27546571357014,4.27522679939878,4.27183486427919,4.245808462209,4.24481574677634,4.22690078880448,4.20570796973513,4.17931097594361,4.17792666035086,4.17792666035086,4.15474574301615,4.13509659838901,4.12901201059938,4.10369354108922,4.07460639983293,4.04740750098712,4.04684036169466,4.03604571749061,4.00600375755195,3.99344166920828,3.95877861587162,3.95467223351879,3.91684960621714,3.90712511625115,3.89610279684101,3.89237118018985,3.87763759178544,3.8651304197947,3.85214439740818,3.83246308762883,3.80241729820932,3.79634165019049,3.79510441057406,3.78730774236994,3.77496002914308,3.75341713874313,3.75172904311691,3.74102374287325,3.7353395780933,3.72949967279225,3.71972851015521,3.71893622798761,3.68843485870564,3.67301971411472,3.65120527728455,3.64524452929238,3.63316617757685,3.62815489681806,3.60619344924598,3.60616395610464,3.60445011696757,3.59754477117269,3.57840077373173,3.57644500382323,3.56801302447616,3.56038005584375,3.52682011176475,3.51499879316056,3.49822523256244,3.48722408791602,3.48383676463543,3.48194293195002,3.4764327911695,3.47436981952443,3.47004590336031,3.46822925081249,3.45906436251804,3.45356475004917,3.43438922275978,3.38790891349439,3.38586336539467,3.37875030000231,3.36387746601184,3.36154890388285,3.34749060274363,3.33719648866205,3.33520012773496,3.33180246931044,3.32899866341425,3.31876350893667,3.30723838515344,3.29866826521515,3.29281840571962,3.29280407485276,3.29099683789457,3.28132952287642,3.27541143876838,3.2419931687505,3.22682062495398,3.22446930252263,3.2210417696304,3.21361001397397,3.2011385893605,3.2011385893605,3.2011385893605,3.2011385893605,3.19635624911493,3.18440179328393,3.15420502293645,3.15074455776183,3.14878243942381,3.14384670583683,3.14200661688148,3.1371782262304,3.13503523448486,3.12452469396426,3.10694316053477,3.10186013134915,3.09492517481704,3.07250083511216,3.07071803091161,3.0682918406924,3.05818844713353,3.04656668242119,3.01208498149199,3.01004824111147,2.99401921769256,2.95332931673533,2.94349248755002,2.94331235048162,2.94111742003991,2.93198525666249,2.92484872587638,2.92484872587638,2.91703661256839,2.91703661256839,2.91039665721083,2.91036004437845,2.90228324485645,2.90021353570147,2.89234052848257,2.88476546563612,2.87000863395735,2.86853395534251,2.8580293060441,2.85710927240613,2.85245577293018,2.82142549941692,2.8143355483515,2.78336259981862,2.75381328261204,2.75239106083733,2.74582476768074,2.72912550120905,2.72567355026179,2.72567355026179,2.72349840352009,2.7209935073136,2.71371165381992,2.69065270100582,2.68847228829155,2.68679315378898,2.6833508057942,2.68288510063435,2.68090145772369,2.66775965140857,2.66736652457199,2.66575733579855,2.65723899232324,2.64995085804607,2.64152351156975,2.62089314166663,2.61678009436634,2.60981192218479,2.60719502859095,2.59953560810979,2.59927384686861,2.59876690494599,2.59731445580915,2.59587052839482,2.59237223569238,2.58068305367085,2.58068305367085,2.5785578006416,2.57792935670413,2.57166957198386,2.56616118779982,2.53771707348477,2.53303784295783,2.52893575623595,2.51824522766897,2.50220368026976,2.49227291132559,2.48872972372946,2.48098585911426,2.43882679782109,2.43280061196331,2.42911060618754,2.41149328446306,2.40793298758013,2.39723241488435,2.39159200861419,2.39145317315258,2.39003396252864,2.38619709082779,2.38567172991978,2.37797555081052,2.36875807441429,2.35097191136419,2.3475802769241,2.3381657461607,2.32960000921648,2.31095087661982,2.30955699552812,2.30781826196997,2.3072797172009,2.30352879474659,2.29457157867309,2.28942661842462,2.28793894425906,2.28575236483363,2.28249927720254,2.27275344938263,2.27275344938263,2.26813036580239,2.26182847497994,2.25370261287274,2.25087486010919,2.21660131752964,2.2134414727676,2.20179803743209,2.20179803743209,2.19698737370469,2.16748516873632,2.1644599654588,2.16406683087878,2.16147314243094,2.16147314243094,2.16061432445459,2.14127344192035,2.12897954383884,2.12036647097993,2.10957125643914,2.10690740128817,2.087936709455,2.08173109878419,2.07514101264006,2.07429270684418,2.06678608088196,2.06638167818992,2.06137889786249,2.05568527328624,2.04228883609915,2.03302044016564,2.02393700000592,2.01729320350223,2.01716888431543,2.01653153039199,2.00405581945043,2.00187870099542,1.99662308234754,1.98881917466138,1.97975651347786,1.9778115522201,1.96963277406529,1.96955682849295,1.96872062727386,1.94499821472198,1.93606391064806,1.93065222865936,1.93055757724831,1.92986746523542,1.89369792746205,1.89369792746205,1.89125596809307,1.88173242750423,1.88007879519542,1.87245865484112,1.87173526613803,1.8700200457451,1.86953730131774,1.85540449066205,1.85073852777193,1.85068332700409,1.84916731019328,1.82417504077159,1.8166340543213,1.815543601095,1.81529156383258,1.80676291959955,1.80644316539867,1.80264994581593,1.80147025497747,1.80103329393928,1.80062303256824,1.79815433845926,1.79617303379316,1.79302926142653,1.78818109868907,1.78677806407345,1.77720622487192,1.77283850429468,1.76907327843323,1.76830686658332,1.7658930706178,1.7658276832599,1.76308326326572,1.73976083689841,1.72827550850162,1.72503989038986,1.72078240561095,1.7205149790318,1.71661947543172,1.71606023618643,1.7129135511489,1.71236489473721,1.70822952234247,1.70163983515246,1.70106692895032,1.69032374547043,1.68343065716044,1.67964425208049,1.67240307687271,1.67176720400783,1.66975588098952,1.66745145627542,1.66745145627542,1.66745145627542,1.66366344831827,1.64589666063607,1.64392254258079,1.63494046613794,1.63451002004918,1.61559932668096,1.6144367954407,1.59632921978536,1.5956815577451,1.59378415988379,1.5854459085436,1.5831274750607,1.57084953846249,1.57005076910604,1.56567999839944,1.56392414432524,1.55108973945592,1.5460838649622,1.53363659724204,1.51190334079254,1.51159773673189,1.50356417290654,1.50046774626143,1.49732862282127,1.49587975260808,1.49391988211006,1.49045976543213,1.49008980988756,1.48777282001668,1.48744223804152,1.4806249048701,1.47723364339082,1.46491975018716,1.46491975018716,1.46491741522774,1.46491741522774,1.46491741522774,1.46491741522774,1.46237323669686,1.46161735352711,1.44079262256549,1.43871680987546,1.43682399000859,1.43526166912189,1.43493440994263,1.4324640690851,1.41830716625395,1.41078888460066,1.39999746677419,1.39914400728689,1.39682921049771,1.39514180448458,1.38488320303295,1.37986247045975,1.37762525691508,1.37618321360789,1.37089292103484,1.36612518506471,1.35581054799707,1.35206403082361,1.33341265634285,1.32618565976831,1.31352577660809,1.30321917996944],"text":["GO:0006996 (3874) <br> organelle organization <br> 1.824e-57","GO:0007010 (1453) <br> cytoskeleton organization <br> 7.944e-51","GO:0051128 (2276) <br> regulation of cellular component organization <br> 5.680e-47","GO:0030029 (804) <br> actin filament-based process <br> 7.359e-40","GO:0030036 (715) <br> actin cytoskeleton organization <br> 9.403e-39","GO:0044260 (5766) <br> cellular macromolecule metabolic process <br> 1.647e-35","GO:0071840 (8070) <br> cellular component organization or biogenesis <br> 2.271e-34","GO:0044267 (4882) <br> cellular protein metabolic process <br> 8.200e-33","GO:0016043 (7868) <br> cellular component organization <br> 1.515e-31","GO:0008104 (2665) <br> protein localization <br> 1.003e-30","GO:1901564 (6446) <br> organonitrogen compound metabolic process <br> 2.616e-30","GO:0007049 (1774) <br> cell cycle <br> 3.564e-30","GO:0033036 (3125) <br> macromolecule localization <br> 5.075e-30","GO:0032268 (2370) <br> regulation of cellular protein metabolic process <br> 8.923e-30","GO:0033043 (1137) <br> regulation of organelle organization <br> 1.830e-29","GO:0006793 (2803) <br> phosphorus metabolic process <br> 3.076e-29","GO:0051246 (2536) <br> regulation of protein metabolic process <br> 6.163e-29","GO:0006796 (2781) <br> phosphate-containing compound metabolic process <br> 6.889e-29","GO:0051130 (1046) <br> positive regulation of cellular component organization <br> 1.229e-28","GO:0019538 (5487) <br> protein metabolic process <br> 1.099e-27","GO:0051641 (3031) <br> cellular localization <br> 1.150e-27","GO:0051179 (6436) <br> localization <br> 1.221e-26","GO:0048522 (5687) <br> positive regulation of cellular process <br> 1.638e-26","GO:0097435 (778) <br> supramolecular fiber organization <br> 2.383e-26","GO:0022402 (1206) <br> cell cycle process <br> 1.292e-25","GO:0044087 (946) <br> regulation of cellular component biogenesis <br> 3.218e-25","GO:0048518 (6224) <br> positive regulation of biological process <br> 1.261e-24","GO:0065009 (3050) <br> regulation of molecular function <br> 1.569e-24","GO:0043412 (3936) <br> macromolecule modification <br> 1.255e-23","GO:0050790 (2345) <br> regulation of catalytic activity <br> 6.206e-23","GO:0035556 (2629) <br> intracellular signal transduction <br> 7.707e-23","GO:0007015 (448) <br> actin filament organization <br> 1.518e-22","GO:0044089 (497) <br> positive regulation of cellular component biogenesis <br> 2.887e-21","GO:0051493 (529) <br> regulation of cytoskeleton organization <br> 3.304e-21","GO:0036211 (3719) <br> protein modification process <br> 3.490e-21","GO:0006464 (3719) <br> cellular protein modification process <br> 3.490e-21","GO:0051247 (1493) <br> positive regulation of protein metabolic process <br> 1.352e-20","GO:0044248 (2199) <br> cellular catabolic process <br> 2.306e-20","GO:0000278 (906) <br> mitotic cell cycle <br> 3.289e-20","GO:0034613 (1913) <br> cellular protein localization <br> 3.467e-20","GO:0070727 (1922) <br> cellular macromolecule localization <br> 6.381e-20","GO:0009894 (1009) <br> regulation of catabolic process <br> 2.067e-19","GO:0032270 (1406) <br> positive regulation of cellular protein metabolic process <br> 2.432e-19","GO:0000902 (1012) <br> cell morphogenesis <br> 2.679e-19","GO:0009056 (2559) <br> catabolic process <br> 2.942e-19","GO:0006928 (2085) <br> movement of cell or subcellular component <br> 3.809e-19","GO:2000112 (483) <br> regulation of cellular macromolecule biosynthetic process <br> 1.108e-18","GO:0016192 (1604) <br> vesicle-mediated transport <br> 1.151e-18","GO:1903047 (755) <br> mitotic cell cycle process <br> 2.106e-18","GO:0051649 (2196) <br> establishment of localization in cell <br> 5.146e-18","GO:0030030 (1525) <br> cell projection organization <br> 9.147e-18","GO:0006417 (393) <br> regulation of translation <br> 1.146e-17","GO:1902903 (374) <br> regulation of supramolecular fiber organization <br> 1.929e-17","GO:0016310 (1882) <br> phosphorylation <br> 1.978e-17","GO:0032879 (2725) <br> regulation of localization <br> 2.361e-17","GO:0009057 (1359) <br> macromolecule catabolic process <br> 2.450e-17","GO:0051301 (646) <br> cell division <br> 3.471e-17","GO:0016477 (1461) <br> cell migration <br> 3.604e-17","GO:0032970 (392) <br> regulation of actin filament-based process <br> 4.001e-17","GO:0031329 (871) <br> regulation of cellular catabolic process <br> 8.023e-17","GO:0065008 (3768) <br> regulation of biological quality <br> 1.128e-16","GO:0007163 (226) <br> establishment or maintenance of cell polarity <br> 1.666e-16","GO:0050794 (11193) <br> regulation of cellular process <br> 2.094e-16","GO:0010638 (509) <br> positive regulation of organelle organization <br> 2.346e-16","GO:0051234 (4903) <br> establishment of localization <br> 2.425e-16","GO:0034248 (449) <br> regulation of cellular amide metabolic process <br> 2.890e-16","GO:1902531 (1691) <br> regulation of intracellular signal transduction <br> 4.717e-16","GO:0120036 (1485) <br> plasma membrane bounded cell projection organization <br> 8.086e-16","GO:0048856 (5754) <br> anatomical structure development <br> 1.138e-15","GO:0034330 (691) <br> cell junction organization <br> 1.141e-15","GO:0048523 (4856) <br> negative regulation of cellular process <br> 1.503e-15","GO:0045184 (1889) <br> establishment of protein localization <br> 1.507e-15","GO:0044265 (1152) <br> cellular macromolecule catabolic process <br> 1.820e-15","GO:0000226 (630) <br> microtubule cytoskeleton organization <br> 1.894e-15","GO:0007264 (510) <br> small GTPase mediated signal transduction <br> 1.898e-15","GO:0060341 (741) <br> regulation of cellular localization <br> 2.110e-15","GO:0051656 (410) <br> establishment of organelle localization <br> 3.513e-15","GO:0048583 (3895) <br> regulation of response to stimulus <br> 3.856e-15","GO:0033554 (1875) <br> cellular response to stress <br> 4.263e-15","GO:0046907 (1766) <br> intracellular transport <br> 4.532e-15","GO:0019220 (1386) <br> regulation of phosphate metabolic process <br> 5.322e-15","GO:0051174 (1388) <br> regulation of phosphorus metabolic process <br> 6.031e-15","GO:0051716 (7399) <br> cellular response to stimulus <br> 7.002e-15","GO:0043254 (402) <br> regulation of protein-containing complex assembly <br> 7.754e-15","GO:0022411 (471) <br> cellular component disassembly <br> 8.651e-15","GO:0051640 (542) <br> organelle localization <br> 9.362e-15","GO:0048870 (1655) <br> cell motility <br> 1.035e-14","GO:0051674 (1655) <br> localization of cell <br> 1.035e-14","GO:0032956 (352) <br> regulation of actin cytoskeleton organization <br> 1.208e-14","GO:0006950 (3851) <br> response to stress <br> 1.304e-14","GO:0031325 (3271) <br> positive regulation of cellular metabolic process <br> 1.373e-14","GO:0009966 (2947) <br> regulation of signal transduction <br> 1.475e-14","GO:0048468 (2094) <br> cell development <br> 1.875e-14","GO:0006518 (894) <br> peptide metabolic process <br> 2.193e-14","GO:0032502 (6317) <br> developmental process <br> 2.649e-14","GO:0023051 (3351) <br> regulation of signaling <br> 3.318e-14","GO:1901575 (2103) <br> organic substance catabolic process <br> 3.574e-14","GO:0043043 (736) <br> peptide biosynthetic process <br> 3.746e-14","GO:0051248 (1034) <br> negative regulation of protein metabolic process <br> 4.334e-14","GO:0015031 (1778) <br> protein transport <br> 4.383e-14","GO:0006412 (707) <br> translation <br> 4.698e-14","GO:0032269 (970) <br> negative regulation of cellular protein metabolic process <br> 7.509e-14","GO:0006468 (1587) <br> protein phosphorylation <br> 7.664e-14","GO:1901566 (1708) <br> organonitrogen compound biosynthetic process <br> 9.837e-14","GO:0031399 (1527) <br> regulation of protein modification process <br> 1.029e-13","GO:0010646 (3336) <br> regulation of cell communication <br> 1.113e-13","GO:0006810 (4742) <br> transport <br> 2.206e-13","GO:0051173 (3083) <br> positive regulation of nitrogen compound metabolic process <br> 2.542e-13","GO:0031323 (6004) <br> regulation of cellular metabolic process <br> 3.211e-13","GO:0007017 (904) <br> microtubule-based process <br> 3.921e-13","GO:0007275 (4768) <br> multicellular organism development <br> 7.724e-13","GO:0040011 (1833) <br> locomotion <br> 1.101e-12","GO:0110053 (269) <br> regulation of actin filament organization <br> 1.296e-12","GO:0071705 (2233) <br> nitrogen compound transport <br> 1.405e-12","GO:0009653 (2702) <br> anatomical structure morphogenesis <br> 1.590e-12","GO:0080090 (5812) <br> regulation of primary metabolic process <br> 1.729e-12","GO:0070887 (3019) <br> cellular response to chemical stimulus <br> 1.834e-12","GO:0000904 (722) <br> cell morphogenesis involved in differentiation <br> 1.973e-12","GO:0031032 (196) <br> actomyosin structure organization <br> 3.286e-12","GO:0009893 (3776) <br> positive regulation of metabolic process <br> 4.092e-12","GO:0010564 (695) <br> regulation of cell cycle process <br> 4.100e-12","GO:0019058 (313) <br> viral life cycle <br> 4.670e-12","GO:0006413 (129) <br> translational initiation <br> 4.847e-12","GO:0043603 (1167) <br> cellular amide metabolic process <br> 4.991e-12","GO:0031175 (957) <br> neuron projection development <br> 5.107e-12","GO:0048869 (4184) <br> cellular developmental process <br> 5.387e-12","GO:0010604 (3465) <br> positive regulation of macromolecule metabolic process <br> 9.332e-12","GO:0007154 (6508) <br> cell communication <br> 9.463e-12","GO:0016032 (419) <br> viral process <br> 9.847e-12","GO:0045216 (203) <br> cell-cell junction organization <br> 1.074e-11","GO:0043604 (867) <br> amide biosynthetic process <br> 1.111e-11","GO:0048666 (1108) <br> neuron development <br> 1.263e-11","GO:0009896 (519) <br> positive regulation of catabolic process <br> 1.342e-11","GO:0032880 (845) <br> regulation of protein localization <br> 1.401e-11","GO:0071702 (2654) <br> organic substance transport <br> 1.437e-11","GO:0007051 (192) <br> spindle organization <br> 1.895e-11","GO:0140694 (379) <br> non-membrane-bounded organelle assembly <br> 1.909e-11","GO:0048731 (4313) <br> system development <br> 2.059e-11","GO:0030154 (4160) <br> cell differentiation <br> 2.201e-11","GO:0051171 (5654) <br> regulation of nitrogen compound metabolic process <br> 2.318e-11","GO:0051726 (1089) <br> regulation of cell cycle <br> 2.576e-11","GO:0032989 (757) <br> cellular component morphogenesis <br> 2.832e-11","GO:0010639 (352) <br> negative regulation of organelle organization <br> 3.134e-11","GO:0044093 (1556) <br> positive regulation of molecular function <br> 3.225e-11","GO:0006402 (267) <br> mRNA catabolic process <br> 3.400e-11","GO:0048858 (642) <br> cell projection morphogenesis <br> 3.437e-11","GO:0007059 (344) <br> chromosome segregation <br> 3.840e-11","GO:0006915 (1865) <br> apoptotic process <br> 3.863e-11","GO:1903829 (457) <br> positive regulation of protein localization <br> 4.087e-11","GO:0050789 (12276) <br> regulation of biological process <br> 4.090e-11","GO:0050896 (8885) <br> response to stimulus <br> 4.302e-11","GO:0065007 (13001) <br> biological regulation <br> 5.405e-11","GO:0042325 (1242) <br> regulation of phosphorylation <br> 5.621e-11","GO:0007155 (1489) <br> cell adhesion <br> 6.181e-11","GO:0032990 (661) <br> cell part morphogenesis <br> 6.191e-11","GO:0012501 (1918) <br> programmed cell death <br> 6.791e-11","GO:0008219 (2074) <br> cell death <br> 7.363e-11","GO:0150115 (99) <br> cell-substrate junction organization <br> 7.779e-11","GO:0034645 (1171) <br> cellular macromolecule biosynthetic process <br> 7.784e-11","GO:0000910 (179) <br> cytokinesis <br> 7.919e-11","GO:0051017 (158) <br> actin filament bundle assembly <br> 8.721e-11","GO:1903311 (326) <br> regulation of mRNA metabolic process <br> 1.108e-10","GO:0061013 (203) <br> regulation of mRNA catabolic process <br> 1.279e-10","GO:0061572 (161) <br> actin filament bundle organization <br> 1.564e-10","GO:0023052 (6451) <br> signaling <br> 1.697e-10","GO:0120039 (638) <br> plasma membrane bounded cell projection morphogenesis <br> 1.820e-10","GO:0010562 (895) <br> positive regulation of phosphorus metabolic process <br> 1.852e-10","GO:0045937 (895) <br> positive regulation of phosphate metabolic process <br> 1.852e-10","GO:0034329 (424) <br> cell junction assembly <br> 2.391e-10","GO:0044270 (487) <br> cellular nitrogen compound catabolic process <br> 2.775e-10","GO:0097581 (91) <br> lamellipodium organization <br> 3.325e-10","GO:0019439 (502) <br> aromatic compound catabolic process <br> 3.375e-10","GO:0048519 (5876) <br> negative regulation of biological process <br> 3.414e-10","GO:0051129 (678) <br> negative regulation of cellular component organization <br> 3.460e-10","GO:0051258 (277) <br> protein polymerization <br> 3.586e-10","GO:0006446 (82) <br> regulation of translational initiation <br> 3.703e-10","GO:0031331 (451) <br> positive regulation of cellular catabolic process <br> 3.829e-10","GO:0010033 (3010) <br> response to organic substance <br> 4.144e-10","GO:0051270 (1028) <br> regulation of cellular component movement <br> 4.709e-10","GO:0034655 (442) <br> nucleobase-containing compound catabolic process <br> 4.824e-10","GO:0046700 (481) <br> heterocycle catabolic process <br> 5.237e-10","GO:0031401 (999) <br> positive regulation of protein modification process <br> 6.024e-10","GO:0051495 (196) <br> positive regulation of cytoskeleton organization <br> 6.337e-10","GO:0043069 (887) <br> negative regulation of programmed cell death <br> 6.403e-10","GO:0007265 (348) <br> Ras protein signal transduction <br> 6.671e-10","GO:0140014 (301) <br> mitotic nuclear division <br> 9.012e-10","GO:0006401 (315) <br> RNA catabolic process <br> 9.351e-10","GO:0048812 (623) <br> neuron projection morphogenesis <br> 9.354e-10","GO:0060627 (531) <br> regulation of vesicle-mediated transport <br> 1.118e-09","GO:0022008 (1642) <br> neurogenesis <br> 1.298e-09","GO:0022607 (4587) <br> cellular component assembly <br> 1.354e-09","GO:0007399 (2430) <br> nervous system development <br> 1.446e-09","GO:2000145 (952) <br> regulation of cell motility <br> 1.489e-09","GO:0030334 (893) <br> regulation of cell migration <br> 1.504e-09","GO:0060548 (983) <br> negative regulation of cell death <br> 1.512e-09","GO:1902117 (80) <br> positive regulation of organelle assembly <br> 1.581e-09","GO:0044092 (1112) <br> negative regulation of molecular function <br> 1.645e-09","GO:0071310 (2391) <br> cellular response to organic substance <br> 1.662e-09","GO:0032271 (204) <br> regulation of protein polymerization <br> 1.722e-09","GO:0070925 (900) <br> organelle assembly <br> 1.770e-09","GO:0031344 (635) <br> regulation of cell projection organization <br> 1.816e-09","GO:0043067 (1448) <br> regulation of programmed cell death <br> 1.944e-09","GO:0008154 (199) <br> actin polymerization or depolymerization <br> 3.540e-09","GO:0043085 (1170) <br> positive regulation of catalytic activity <br> 3.775e-09","GO:0032984 (245) <br> protein-containing complex disassembly <br> 4.208e-09","GO:0042327 (816) <br> positive regulation of phosphorylation <br> 4.821e-09","GO:0061024 (829) <br> membrane organization <br> 5.138e-09","GO:0031334 (194) <br> positive regulation of protein-containing complex assembly <br> 5.438e-09","GO:0010941 (1592) <br> regulation of cell death <br> 5.764e-09","GO:1902904 (167) <br> negative regulation of supramolecular fiber organization <br> 6.904e-09","GO:0042981 (1419) <br> regulation of apoptotic process <br> 7.350e-09","GO:0043488 (188) <br> regulation of mRNA stability <br> 7.555e-09","GO:0007044 (93) <br> cell-substrate junction assembly <br> 7.689e-09","GO:0022604 (307) <br> regulation of cell morphogenesis <br> 8.049e-09","GO:0000070 (173) <br> mitotic sister chromatid segregation <br> 8.999e-09","GO:0048193 (310) <br> Golgi vesicle transport <br> 9.373e-09","GO:1902850 (157) <br> microtubule cytoskeleton organization involved in mitosis <br> 9.760e-09","GO:0055086 (562) <br> nucleobase-containing small molecule metabolic process <br> 9.937e-09","GO:0042060 (429) <br> wound healing <br> 1.011e-08","GO:0043487 (201) <br> regulation of RNA stability <br> 1.126e-08","GO:1901361 (530) <br> organic cyclic compound catabolic process <br> 1.177e-08","GO:0120035 (619) <br> regulation of plasma membrane bounded cell projection organization <br> 1.189e-08","GO:0009611 (558) <br> response to wounding <br> 1.473e-08","GO:0030182 (1366) <br> neuron differentiation <br> 1.547e-08","GO:0044085 (4834) <br> cellular component biogenesis <br> 1.606e-08","GO:0000819 (202) <br> sister chromatid segregation <br> 1.615e-08","GO:0043066 (869) <br> negative regulation of apoptotic process <br> 1.769e-08","GO:0048699 (1436) <br> generation of neurons <br> 1.970e-08","GO:0034250 (167) <br> positive regulation of cellular amide metabolic process <br> 1.993e-08","GO:0042221 (4336) <br> response to chemical <br> 2.015e-08","GO:0051225 (122) <br> spindle assembly <br> 2.155e-08","GO:0051494 (165) <br> negative regulation of cytoskeleton organization <br> 2.252e-08","GO:0033365 (1109) <br> protein localization to organelle <br> 2.453e-08","GO:0040012 (997) <br> regulation of locomotion <br> 2.460e-08","GO:0010256 (550) <br> endomembrane system organization <br> 2.578e-08","GO:0043086 (755) <br> negative regulation of catalytic activity <br> 2.699e-08","GO:0030163 (996) <br> protein catabolic process <br> 3.690e-08","GO:0045727 (142) <br> positive regulation of translation <br> 3.690e-08","GO:0051056 (307) <br> regulation of small GTPase mediated signal transduction <br> 3.831e-08","GO:1990778 (336) <br> protein localization to cell periphery <br> 3.853e-08","GO:0007165 (5957) <br> signal transduction <br> 3.946e-08","GO:0001932 (1097) <br> regulation of protein phosphorylation <br> 4.206e-08","GO:1902905 (177) <br> positive regulation of supramolecular fiber organization <br> 4.238e-08","GO:0051653 (59) <br> spindle localization <br> 4.541e-08","GO:0098813 (284) <br> nuclear chromosome segregation <br> 5.281e-08","GO:0051983 (96) <br> regulation of chromosome segregation <br> 5.463e-08","GO:0061640 (106) <br> cytoskeleton-dependent cytokinesis <br> 5.658e-08","GO:0032535 (357) <br> regulation of cellular component size <br> 6.315e-08","GO:0051336 (992) <br> regulation of hydrolase activity <br> 6.588e-08","GO:0010628 (1135) <br> positive regulation of gene expression <br> 6.994e-08","GO:0051050 (896) <br> positive regulation of transport <br> 8.180e-08","GO:0120032 (194) <br> regulation of plasma membrane bounded cell projection assembly <br> 8.356e-08","GO:0052126 (179) <br> movement in host environment <br> 9.317e-08","GO:0060491 (196) <br> regulation of cell projection assembly <br> 1.063e-07","GO:0051701 (194) <br> biological process involved in interaction with host <br> 1.183e-07","GO:0018193 (1285) <br> peptidyl-amino acid modification <br> 1.377e-07","GO:0030010 (147) <br> establishment of cell polarity <br> 1.567e-07","GO:0006897 (641) <br> endocytosis <br> 1.581e-07","GO:0032501 (7354) <br> multicellular organismal process <br> 1.651e-07","GO:0043244 (130) <br> regulation of protein-containing complex disassembly <br> 1.767e-07","GO:0044770 (538) <br> cell cycle phase transition <br> 1.839e-07","GO:0051049 (1724) <br> regulation of transport <br> 1.863e-07","GO:0044772 (440) <br> mitotic cell cycle phase transition <br> 2.044e-07","GO:0051338 (893) <br> regulation of transferase activity <br> 2.073e-07","GO:0031400 (500) <br> negative regulation of protein modification process <br> 2.131e-07","GO:0048285 (500) <br> organelle fission <br> 2.275e-07","GO:0006900 (62) <br> vesicle budding from membrane <br> 2.309e-07","GO:0050793 (2425) <br> regulation of developmental process <br> 2.956e-07","GO:0007169 (613) <br> transmembrane receptor protein tyrosine kinase signaling pathway <br> 2.981e-07","GO:0030866 (41) <br> cortical actin cytoskeleton organization <br> 3.212e-07","GO:0017148 (174) <br> negative regulation of translation <br> 3.472e-07","GO:0019222 (7313) <br> regulation of metabolic process <br> 3.483e-07","GO:0000281 (77) <br> mitotic cytokinesis <br> 3.528e-07","GO:0031346 (340) <br> positive regulation of cell projection organization <br> 3.628e-07","GO:0007043 (148) <br> cell-cell junction assembly <br> 3.753e-07","GO:0009117 (485) <br> nucleotide metabolic process <br> 3.925e-07","GO:0000280 (454) <br> nuclear division <br> 4.038e-07","GO:1901699 (683) <br> cellular response to nitrogen compound <br> 4.338e-07","GO:0009719 (1557) <br> response to endogenous stimulus <br> 4.808e-07","GO:0051276 (1059) <br> chromosome organization <br> 4.860e-07","GO:0001885 (67) <br> endothelial cell development <br> 5.072e-07","GO:0045936 (417) <br> negative regulation of phosphate metabolic process <br> 5.374e-07","GO:0048667 (564) <br> cell morphogenesis involved in neuron differentiation <br> 5.588e-07","GO:0010563 (418) <br> negative regulation of phosphorus metabolic process <br> 5.848e-07","GO:0001934 (735) <br> positive regulation of protein phosphorylation <br> 5.886e-07","GO:1905818 (77) <br> regulation of chromosome separation <br> 6.024e-07","GO:0032506 (41) <br> cytokinetic process <br> 6.252e-07","GO:0120031 (553) <br> plasma membrane bounded cell projection assembly <br> 6.620e-07","GO:0007346 (475) <br> regulation of mitotic cell cycle <br> 6.789e-07","GO:0098609 (913) <br> cell-cell adhesion <br> 8.010e-07","GO:0051668 (877) <br> localization within membrane <br> 8.072e-07","GO:1905477 (110) <br> positive regulation of protein localization to membrane <br> 8.126e-07","GO:0030031 (567) <br> cell projection assembly <br> 8.192e-07","GO:0031532 (109) <br> actin cytoskeleton reorganization <br> 8.305e-07","GO:0032386 (339) <br> regulation of intracellular transport <br> 8.335e-07","GO:0006753 (495) <br> nucleoside phosphate metabolic process <br> 8.440e-07","GO:0044257 (833) <br> cellular protein catabolic process <br> 8.600e-07","GO:0071495 (1316) <br> cellular response to endogenous stimulus <br> 8.827e-07","GO:0090066 (491) <br> regulation of anatomical structure size <br> 9.078e-07","GO:1901576 (5774) <br> organic substance biosynthetic process <br> 9.275e-07","GO:0008064 (160) <br> regulation of actin polymerization or depolymerization <br> 9.735e-07","GO:0031589 (362) <br> cell-substrate adhesion <br> 9.961e-07","GO:0030100 (203) <br> regulation of endocytosis <br> 1.009e-06","GO:0006470 (296) <br> protein dephosphorylation <br> 1.050e-06","GO:0060255 (6787) <br> regulation of macromolecule metabolic process <br> 1.055e-06","GO:1901698 (1100) <br> response to nitrogen compound <br> 1.076e-06","GO:0030832 (161) <br> regulation of actin filament length <br> 1.105e-06","GO:0034660 (568) <br> ncRNA metabolic process <br> 1.268e-06","GO:2000113 (196) <br> negative regulation of cellular macromolecule biosynthetic process <br> 1.317e-06","GO:0030041 (169) <br> actin filament polymerization <br> 1.431e-06","GO:0030335 (523) <br> positive regulation of cell migration <br> 1.469e-06","GO:0007052 (127) <br> mitotic spindle organization <br> 1.618e-06","GO:1902115 (203) <br> regulation of organelle assembly <br> 1.702e-06","GO:0043087 (367) <br> regulation of GTPase activity <br> 1.753e-06","GO:0007160 (234) <br> cell-matrix adhesion <br> 1.774e-06","GO:0019637 (943) <br> organophosphate metabolic process <br> 1.777e-06","GO:0071417 (626) <br> cellular response to organonitrogen compound <br> 1.883e-06","GO:0043632 (697) <br> modification-dependent macromolecule catabolic process <br> 1.908e-06","GO:0061014 (94) <br> positive regulation of mRNA catabolic process <br> 1.920e-06","GO:1901565 (1350) <br> organonitrogen compound catabolic process <br> 1.966e-06","GO:0030048 (132) <br> actin filament-based movement <br> 2.229e-06","GO:0032273 (93) <br> positive regulation of protein polymerization <br> 2.252e-06","GO:0034249 (196) <br> negative regulation of cellular amide metabolic process <br> 2.286e-06","GO:0044249 (5700) <br> cellular biosynthetic process <br> 2.390e-06","GO:0043009 (653) <br> chordate embryonic development <br> 2.456e-06","GO:0002064 (206) <br> epithelial cell development <br> 2.618e-06","GO:0009058 (5861) <br> biosynthetic process <br> 2.676e-06","GO:0090130 (302) <br> tissue migration <br> 3.114e-06","GO:2000147 (547) <br> positive regulation of cell motility <br> 3.128e-06","GO:0043555 (22) <br> regulation of translation in response to stress <br> 3.131e-06","GO:0044409 (154) <br> entry into host <br> 3.186e-06","GO:0032886 (256) <br> regulation of microtubule-based process <br> 3.302e-06","GO:0031647 (301) <br> regulation of protein stability <br> 3.668e-06","GO:0016197 (232) <br> endosomal transport <br> 3.947e-06","GO:0035088 (51) <br> establishment or maintenance of apical/basal cell polarity <br> 3.991e-06","GO:0061245 (51) <br> establishment or maintenance of bipolar cell polarity <br> 3.991e-06","GO:0010631 (293) <br> epithelial cell migration <br> 4.199e-06","GO:0098876 (141) <br> vesicle-mediated transport to the plasma membrane <br> 4.406e-06","GO:0009792 (674) <br> embryo development ending in birth or egg hatching <br> 4.597e-06","GO:0051304 (100) <br> chromosome separation <br> 4.706e-06","GO:0019941 (686) <br> modification-dependent protein catabolic process <br> 4.726e-06","GO:0006511 (675) <br> ubiquitin-dependent protein catabolic process <br> 5.136e-06","GO:0034470 (429) <br> ncRNA processing <br> 5.305e-06","GO:0030865 (63) <br> cortical cytoskeleton organization <br> 5.372e-06","GO:0051293 (54) <br> establishment of spindle localization <br> 5.389e-06","GO:0090132 (296) <br> epithelium migration <br> 5.670e-06","GO:0060429 (1180) <br> epithelium development <br> 6.006e-06","GO:0006886 (1084) <br> intracellular protein transport <br> 6.171e-06","GO:0030032 (73) <br> lamellipodium assembly <br> 6.352e-06","GO:0072659 (284) <br> protein localization to plasma membrane <br> 6.490e-06","GO:0051603 (778) <br> proteolysis involved in cellular protein catabolic process <br> 6.593e-06","GO:0051348 (265) <br> negative regulation of transferase activity <br> 6.737e-06","GO:0046718 (147) <br> viral entry into host cell <br> 6.981e-06","GO:0050658 (163) <br> RNA transport <br> 7.147e-06","GO:0050657 (163) <br> nucleic acid transport <br> 7.147e-06","GO:0019693 (393) <br> ribose phosphate metabolic process <br> 7.734e-06","GO:0043549 (770) <br> regulation of kinase activity <br> 8.144e-06","GO:0051272 (561) <br> positive regulation of cellular component movement <br> 8.214e-06","GO:0016311 (434) <br> dephosphorylation <br> 8.956e-06","GO:0001701 (394) <br> in utero embryonic development <br> 8.970e-06","GO:0010243 (1010) <br> response to organonitrogen compound <br> 8.996e-06","GO:0048646 (1134) <br> anatomical structure formation involved in morphogenesis <br> 9.329e-06","GO:0040017 (563) <br> positive regulation of locomotion <br> 9.397e-06","GO:0048513 (3513) <br> animal organ development <br> 9.427e-06","GO:0070507 (155) <br> regulation of microtubule cytoskeleton organization <br> 9.639e-06","GO:0006913 (313) <br> nucleocytoplasmic transport <br> 9.679e-06","GO:0051169 (313) <br> nuclear transport <br> 9.679e-06","GO:0043434 (398) <br> response to peptide hormone <br> 9.769e-06","GO:0023056 (1715) <br> positive regulation of signaling <br> 9.940e-06","GO:0051261 (122) <br> protein depolymerization <br> 1.016e-05","GO:1903313 (123) <br> positive regulation of mRNA metabolic process <br> 1.032e-05","GO:1901990 (320) <br> regulation of mitotic cell cycle phase transition <br> 1.035e-05","GO:0007417 (1037) <br> central nervous system development <br> 1.038e-05","GO:0031324 (2506) <br> negative regulation of cellular metabolic process <br> 1.060e-05","GO:0051236 (166) <br> establishment of RNA localization <br> 1.063e-05","GO:0043393 (198) <br> regulation of protein binding <br> 1.064e-05","GO:1901880 (77) <br> negative regulation of protein depolymerization <br> 1.076e-05","GO:0045446 (109) <br> endothelial cell differentiation <br> 1.158e-05","GO:0034504 (301) <br> protein localization to nucleus <br> 1.230e-05","GO:0048041 (83) <br> focal adhesion assembly <br> 1.233e-05","GO:0009888 (1940) <br> tissue development <br> 1.264e-05","GO:0000956 (125) <br> nuclear-transcribed mRNA catabolic process <br> 1.410e-05","GO:0051172 (2371) <br> negative regulation of nitrogen compound metabolic process <br> 1.437e-05","GO:1905475 (181) <br> regulation of protein localization to membrane <br> 1.465e-05","GO:0071375 (288) <br> cellular response to peptide hormone stimulus <br> 1.612e-05","GO:0030833 (141) <br> regulation of actin filament polymerization <br> 1.616e-05","GO:0018209 (336) <br> peptidyl-serine modification <br> 1.632e-05","GO:0031109 (123) <br> microtubule polymerization or depolymerization <br> 1.682e-05","GO:0150116 (68) <br> regulation of cell-substrate junction organization <br> 1.889e-05","GO:1901879 (93) <br> regulation of protein depolymerization <br> 1.902e-05","GO:0071407 (569) <br> cellular response to organic cyclic compound <br> 1.964e-05","GO:0018105 (313) <br> peptidyl-serine phosphorylation <br> 1.985e-05","GO:0032869 (199) <br> cellular response to insulin stimulus <br> 2.067e-05","GO:0032388 (204) <br> positive regulation of intracellular transport <br> 2.115e-05","GO:0006914 (541) <br> autophagy <br> 2.412e-05","GO:0061919 (541) <br> process utilizing autophagic mechanism <br> 2.412e-05","GO:0010647 (1711) <br> positive regulation of cell communication <br> 2.586e-05","GO:0006903 (44) <br> vesicle targeting <br> 2.621e-05","GO:1902532 (508) <br> negative regulation of intracellular signal transduction <br> 2.747e-05","GO:0022603 (930) <br> regulation of anatomical structure morphogenesis <br> 2.860e-05","GO:0006898 (243) <br> receptor-mediated endocytosis <br> 2.870e-05","GO:0042176 (404) <br> regulation of protein catabolic process <br> 3.006e-05","GO:0090307 (71) <br> mitotic spindle assembly <br> 3.086e-05","GO:0007167 (951) <br> enzyme-linked receptor protein signaling pathway <br> 3.174e-05","GO:0009968 (1243) <br> negative regulation of signal transduction <br> 3.234e-05","GO:0061157 (90) <br> mRNA destabilization <br> 3.306e-05","GO:0000165 (788) <br> MAPK cascade <br> 3.370e-05","GO:0009725 (839) <br> response to hormone <br> 3.629e-05","GO:0051347 (575) <br> positive regulation of transferase activity <br> 3.775e-05","GO:0031529 (56) <br> ruffle organization <br> 3.835e-05","GO:0042326 (362) <br> negative regulation of phosphorylation <br> 3.948e-05","GO:0010608 (1104) <br> post-transcriptional regulation of gene expression <br> 4.028e-05","GO:0033674 (489) <br> positive regulation of kinase activity <br> 4.142e-05","GO:0110020 (98) <br> regulation of actomyosin structure organization <br> 4.162e-05","GO:0043297 (74) <br> apical junction assembly <br> 4.164e-05","GO:0097190 (592) <br> apoptotic signaling pathway <br> 4.215e-05","GO:0016049 (490) <br> cell growth <br> 4.301e-05","GO:0044271 (4765) <br> cellular nitrogen compound biosynthetic process <br> 4.525e-05","GO:0046777 (227) <br> protein autophosphorylation <br> 4.690e-05","GO:0045197 (45) <br> establishment or maintenance of epithelial cell apical/basal polarity <br> 4.721e-05","GO:0009790 (1110) <br> embryo development <br> 4.904e-05","GO:0043242 (87) <br> negative regulation of protein-containing complex disassembly <br> 4.989e-05","GO:1901987 (414) <br> regulation of cell cycle phase transition <br> 5.232e-05","GO:0001952 (119) <br> regulation of cell-matrix adhesion <br> 5.303e-05","GO:0050000 (88) <br> chromosome localization <br> 5.306e-05","GO:1904377 (67) <br> positive regulation of protein localization to cell periphery <br> 5.348e-05","GO:1904375 (125) <br> regulation of protein localization to cell periphery <br> 5.678e-05","GO:0051239 (2666) <br> regulation of multicellular organismal process <br> 5.691e-05","GO:0008286 (114) <br> insulin receptor signaling pathway <br> 5.931e-05","GO:0043408 (664) <br> regulation of MAPK cascade <br> 6.227e-05","GO:1901653 (349) <br> cellular response to peptide <br> 6.617e-05","GO:0030038 (105) <br> contractile actin filament bundle assembly <br> 6.639e-05","GO:0043149 (105) <br> stress fiber assembly <br> 6.639e-05","GO:0050779 (94) <br> RNA destabilization <br> 7.003e-05","GO:0048584 (2169) <br> positive regulation of response to stimulus <br> 7.327e-05","GO:0009967 (1513) <br> positive regulation of signal transduction <br> 7.430e-05","GO:1902743 (53) <br> regulation of lamellipodium organization <br> 7.876e-05","GO:0062197 (323) <br> cellular response to chemical stress <br> 8.422e-05","GO:0046785 (83) <br> microtubule polymerization <br> 8.966e-05","GO:0023057 (1356) <br> negative regulation of signaling <br> 8.978e-05","GO:0001667 (418) <br> ameboidal-type cell migration <br> 9.204e-05","GO:0051231 (10) <br> spindle elongation <br> 9.863e-05","GO:1901652 (477) <br> response to peptide <br> 1.015e-04","GO:0032870 (582) <br> cellular response to hormone stimulus <br> 1.100e-04","GO:0003158 (126) <br> endothelium development <br> 1.110e-04","GO:0016050 (304) <br> vesicle organization <br> 1.211e-04","GO:0006403 (199) <br> RNA localization <br> 1.238e-04","GO:0050821 (191) <br> protein stabilization <br> 1.270e-04","GO:0008283 (1958) <br> cell population proliferation <br> 1.281e-04","GO:1903362 (264) <br> regulation of cellular protein catabolic process <br> 1.325e-04","GO:0010648 (1351) <br> negative regulation of cell communication <br> 1.364e-04","GO:0007409 (438) <br> axonogenesis <br> 1.406e-04","GO:0120193 (78) <br> tight junction organization <br> 1.471e-04","GO:0014070 (901) <br> response to organic cyclic compound <br> 1.576e-04","GO:0006892 (109) <br> post-Golgi vesicle-mediated transport <br> 1.598e-04","GO:0072657 (799) <br> protein localization to membrane <br> 1.603e-04","GO:1902533 (975) <br> positive regulation of intracellular signal transduction <br> 1.632e-04","GO:0016358 (234) <br> dendrite development <br> 1.679e-04","GO:0032868 (253) <br> response to insulin <br> 1.764e-04","GO:0010035 (538) <br> response to inorganic substance <br> 1.771e-04","GO:0051303 (86) <br> establishment of chromosome localization <br> 1.815e-04","GO:0007229 (112) <br> integrin-mediated signaling pathway <br> 1.839e-04","GO:0051028 (130) <br> mRNA transport <br> 1.864e-04","GO:0045732 (238) <br> positive regulation of protein catabolic process <br> 1.907e-04","GO:0045859 (653) <br> regulation of protein kinase activity <br> 1.910e-04","GO:0046037 (27) <br> GMP metabolic process <br> 2.049e-04","GO:0044403 (289) <br> biological process involved in symbiotic interaction <br> 2.123e-04","GO:0042692 (362) <br> muscle cell differentiation <br> 2.233e-04","GO:0010632 (225) <br> regulation of epithelial cell migration <br> 2.263e-04","GO:0050792 (161) <br> regulation of viral process <br> 2.327e-04","GO:0033673 (226) <br> negative regulation of kinase activity <br> 2.354e-04","GO:0006901 (19) <br> vesicle coating <br> 2.476e-04","GO:0009150 (364) <br> purine ribonucleotide metabolic process <br> 2.476e-04","GO:0030042 (62) <br> actin filament depolymerization <br> 2.486e-04","GO:0007420 (762) <br> brain development <br> 2.526e-04","GO:0010468 (5452) <br> regulation of gene expression <br> 2.640e-04","GO:0046578 (191) <br> regulation of Ras protein signal transduction <br> 2.652e-04","GO:0006163 (391) <br> purine nucleotide metabolic process <br> 2.704e-04","GO:0051492 (90) <br> regulation of stress fiber assembly <br> 2.752e-04","GO:0009259 (383) <br> ribonucleotide metabolic process <br> 2.973e-04","GO:0120192 (72) <br> tight junction assembly <br> 3.055e-04","GO:0048524 (62) <br> positive regulation of viral process <br> 3.175e-04","GO:0010975 (431) <br> regulation of neuron projection development <br> 3.257e-04","GO:0060322 (809) <br> head development <br> 3.282e-04","GO:0080134 (1312) <br> regulation of response to stress <br> 3.297e-04","GO:0071496 (317) <br> cellular response to external stimulus <br> 3.339e-04","GO:0008360 (153) <br> regulation of cell shape <br> 3.355e-04","GO:0090257 (227) <br> regulation of muscle system process <br> 3.388e-04","GO:0061564 (484) <br> axon development <br> 3.402e-04","GO:0045787 (347) <br> positive regulation of cell cycle <br> 3.475e-04","GO:0051345 (573) <br> positive regulation of hydrolase activity <br> 3.519e-04","GO:1901888 (200) <br> regulation of cell junction assembly <br> 3.678e-04","GO:0051650 (173) <br> establishment of vesicle localization <br> 4.093e-04","GO:0000288 (63) <br> nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay <br> 4.113e-04","GO:0009605 (2788) <br> response to external stimulus <br> 4.181e-04","GO:0006457 (214) <br> protein folding <br> 4.326e-04","GO:0030835 (48) <br> negative regulation of actin filament depolymerization <br> 4.350e-04","GO:0015931 (226) <br> nucleobase-containing compound transport <br> 4.493e-04","GO:0072521 (413) <br> purine-containing compound metabolic process <br> 4.600e-04","GO:0035303 (129) <br> regulation of dephosphorylation <br> 4.622e-04","GO:0009628 (1130) <br> response to abiotic stimulus <br> 4.658e-04","GO:0007093 (134) <br> mitotic cell cycle checkpoint signaling <br> 4.688e-04","GO:0042592 (1658) <br> homeostatic process <br> 4.800e-04","GO:2000058 (171) <br> regulation of ubiquitin-dependent protein catabolic process <br> 4.929e-04","GO:0009126 (44) <br> purine nucleoside monophosphate metabolic process <br> 5.027e-04","GO:0051222 (300) <br> positive regulation of protein transport <br> 5.095e-04","GO:0044837 (7) <br> actomyosin contractile ring organization <br> 5.096e-04","GO:0032231 (103) <br> regulation of actin filament bundle assembly <br> 5.117e-04","GO:1903078 (60) <br> positive regulation of protein localization to plasma membrane <br> 5.232e-04","GO:0031110 (89) <br> regulation of microtubule polymerization or depolymerization <br> 5.304e-04","GO:0061061 (621) <br> muscle structure development <br> 5.728e-04","GO:0090068 (248) <br> positive regulation of cell cycle process <br> 5.932e-04","GO:0031116 (33) <br> positive regulation of microtubule polymerization <br> 5.964e-04","GO:0031330 (267) <br> negative regulation of cellular catabolic process <br> 6.011e-04","GO:2001233 (358) <br> regulation of apoptotic signaling pathway <br> 6.115e-04","GO:0120181 (6) <br> focal adhesion disassembly <br> 6.293e-04","GO:0120182 (6) <br> regulation of focal adhesion disassembly <br> 6.293e-04","GO:0120183 (6) <br> positive regulation of focal adhesion disassembly <br> 6.293e-04","GO:0120180 (6) <br> cell-substrate junction disassembly <br> 6.293e-04","GO:0045070 (32) <br> positive regulation of viral genome replication <br> 6.363e-04","GO:0033044 (214) <br> regulation of chromosome organization <br> 6.540e-04","GO:0051310 (69) <br> metaphase plate congression <br> 7.011e-04","GO:1901701 (1166) <br> cellular response to oxygen-containing compound <br> 7.067e-04","GO:0032465 (91) <br> regulation of cytokinesis <br> 7.099e-04","GO:1900180 (137) <br> regulation of protein localization to nucleus <br> 7.180e-04","GO:0043161 (444) <br> proteasome-mediated ubiquitin-dependent protein catabolic process <br> 7.211e-04","GO:0045069 (86) <br> regulation of viral genome replication <br> 7.292e-04","GO:0044281 (1796) <br> small molecule metabolic process <br> 7.328e-04","GO:0009892 (3570) <br> negative regulation of metabolic process <br> 7.507e-04","GO:0061028 (49) <br> establishment of endothelial barrier <br> 7.817e-04","GO:0048199 (22) <br> vesicle targeting, to, from or within Golgi <br> 7.909e-04","GO:0070848 (685) <br> response to growth factor <br> 8.037e-04","GO:0030855 (670) <br> epithelial cell differentiation <br> 8.463e-04","GO:0007020 (37) <br> microtubule nucleation <br> 8.497e-04","GO:0044419 (1659) <br> biological process involved in interspecies interaction between organisms <br> 8.545e-04","GO:0030834 (57) <br> regulation of actin filament depolymerization <br> 8.746e-04","GO:0016344 (3) <br> meiotic chromosome movement towards spindle pole <br> 8.983e-04","GO:0010498 (509) <br> proteasomal protein catabolic process <br> 9.726e-04","GO:0003012 (421) <br> muscle system process <br> 9.771e-04","GO:0033146 (36) <br> regulation of intracellular estrogen receptor signaling pathway <br> 1.014e-03","GO:0051648 (190) <br> vesicle localization <br> 1.113e-03","GO:0032456 (73) <br> endocytic recycling <br> 1.139e-03","GO:0045930 (240) <br> negative regulation of mitotic cell cycle <br> 1.139e-03","GO:0051098 (366) <br> regulation of binding <br> 1.145e-03","GO:0048585 (1599) <br> negative regulation of response to stimulus <br> 1.170e-03","GO:0048207 (15) <br> vesicle targeting, rough ER to cis-Golgi <br> 1.189e-03","GO:0048208 (15) <br> COPII vesicle coating <br> 1.189e-03","GO:0040001 (39) <br> establishment of mitotic spindle localization <br> 1.210e-03","GO:0051294 (39) <br> establishment of spindle orientation <br> 1.210e-03","GO:0097178 (44) <br> ruffle assembly <br> 1.229e-03","GO:0043547 (276) <br> positive regulation of GTPase activity <br> 1.229e-03","GO:0019079 (130) <br> viral genome replication <br> 1.252e-03","GO:0120034 (107) <br> positive regulation of plasma membrane bounded cell projection assembly <br> 1.258e-03","GO:0061842 (34) <br> microtubule organizing center localization <br> 1.281e-03","GO:0007173 (104) <br> epidermal growth factor receptor signaling pathway <br> 1.304e-03","GO:1903050 (229) <br> regulation of proteolysis involved in cellular protein catabolic process <br> 1.349e-03","GO:0046390 (189) <br> ribose phosphate biosynthetic process <br> 1.354e-03","GO:0040007 (959) <br> growth <br> 1.387e-03","GO:1901700 (1624) <br> response to oxygen-containing compound <br> 1.390e-03","GO:0002183 (37) <br> cytoplasmic translational initiation <br> 1.405e-03","GO:1901991 (191) <br> negative regulation of mitotic cell cycle phase transition <br> 1.509e-03","GO:0000022 (9) <br> mitotic spindle elongation <br> 1.533e-03","GO:0051099 (174) <br> positive regulation of binding <br> 1.647e-03","GO:0002181 (157) <br> cytoplasmic translation <br> 1.763e-03","GO:1904951 (316) <br> positive regulation of establishment of protein localization <br> 1.769e-03","GO:0009152 (168) <br> purine ribonucleotide biosynthetic process <br> 1.795e-03","GO:0035304 (88) <br> regulation of protein dephosphorylation <br> 1.866e-03","GO:0051893 (63) <br> regulation of focal adhesion assembly <br> 1.881e-03","GO:0090109 (63) <br> regulation of cell-substrate junction assembly <br> 1.881e-03","GO:0032091 (94) <br> negative regulation of protein binding <br> 1.890e-03","GO:0009991 (490) <br> response to extracellular stimulus <br> 1.901e-03","GO:0072698 (54) <br> protein localization to microtubule cytoskeleton <br> 1.933e-03","GO:0001558 (425) <br> regulation of cell growth <br> 2.039e-03","GO:0034097 (921) <br> response to cytokine <br> 2.049e-03","GO:0080135 (657) <br> regulation of cellular response to stress <br> 2.057e-03","GO:0031113 (56) <br> regulation of microtubule polymerization <br> 2.073e-03","GO:0016236 (308) <br> macroautophagy <br> 2.075e-03","GO:0150105 (21) <br> protein localization to cell-cell junction <br> 2.085e-03","GO:0002934 (10) <br> desmosome organization <br> 2.149e-03","GO:1903900 (139) <br> regulation of viral life cycle <br> 2.151e-03","GO:0009895 (325) <br> negative regulation of catabolic process <br> 2.159e-03","GO:0031112 (37) <br> positive regulation of microtubule polymerization or depolymerization <br> 2.202e-03","GO:0009408 (107) <br> response to heat <br> 2.239e-03","GO:0072697 (11) <br> protein localization to cell cortex <br> 2.283e-03","GO:0034109 (94) <br> homotypic cell-cell adhesion <br> 2.394e-03","GO:0000075 (175) <br> cell cycle checkpoint signaling <br> 2.417e-03","GO:0051306 (67) <br> mitotic sister chromatid separation <br> 2.456e-03","GO:0045454 (38) <br> cell redox homeostasis <br> 2.471e-03","GO:0009165 (253) <br> nucleotide biosynthetic process <br> 2.515e-03","GO:0086001 (73) <br> cardiac muscle cell action potential <br> 2.516e-03","GO:0090114 (31) <br> COPII-coated vesicle budding <br> 2.519e-03","GO:0009167 (42) <br> purine ribonucleoside monophosphate metabolic process <br> 2.527e-03","GO:0010605 (3361) <br> negative regulation of macromolecule metabolic process <br> 2.536e-03","GO:2001236 (153) <br> regulation of extrinsic apoptotic signaling pathway <br> 2.556e-03","GO:0000915 (6) <br> actomyosin contractile ring assembly <br> 2.626e-03","GO:0000912 (6) <br> assembly of actomyosin apparatus involved in cytokinesis <br> 2.626e-03","GO:0007005 (528) <br> mitochondrion organization <br> 2.639e-03","GO:0038127 (119) <br> ERBB signaling pathway <br> 2.643e-03","GO:0001933 (321) <br> negative regulation of protein phosphorylation <br> 2.681e-03","GO:0051693 (44) <br> actin filament capping <br> 2.715e-03","GO:0007266 (137) <br> Rho protein signal transduction <br> 2.899e-03","GO:0006893 (64) <br> Golgi to plasma membrane transport <br> 2.931e-03","GO:1901293 (256) <br> nucleoside phosphate biosynthetic process <br> 2.958e-03","GO:0009123 (79) <br> nucleoside monophosphate metabolic process <br> 3.032e-03","GO:0010769 (97) <br> regulation of cell morphogenesis involved in differentiation <br> 3.146e-03","GO:1902745 (37) <br> positive regulation of lamellipodium organization <br> 3.219e-03","GO:0006082 (988) <br> organic acid metabolic process <br> 3.245e-03","GO:0006469 (203) <br> negative regulation of protein kinase activity <br> 3.304e-03","GO:0045786 (388) <br> negative regulation of cell cycle <br> 3.641e-03","GO:0030155 (740) <br> regulation of cell adhesion <br> 3.691e-03","GO:0031023 (145) <br> microtubule organizing center organization <br> 3.723e-03","GO:0034314 (39) <br> Arp2/3 complex-mediated actin nucleation <br> 3.877e-03","GO:0032467 (41) <br> positive regulation of cytokinesis <br> 3.909e-03","GO:0010634 (147) <br> positive regulation of epithelial cell migration <br> 4.007e-03","GO:0051497 (28) <br> negative regulation of stress fiber assembly <br> 4.059e-03","GO:0010810 (212) <br> regulation of cell-substrate adhesion <br> 4.060e-03","GO:0030953 (11) <br> astral microtubule organization <br> 4.073e-03","GO:1900368 (41) <br> regulation of post-transcriptional gene silencing by RNA <br> 4.110e-03","GO:0006974 (826) <br> cellular response to DNA damage stimulus <br> 4.115e-03","GO:0002011 (58) <br> morphogenesis of an epithelial sheet <br> 4.188e-03","GO:0006508 (1787) <br> proteolysis <br> 4.278e-03","GO:0007091 (62) <br> metaphase/anaphase transition of mitotic cell cycle <br> 4.457e-03","GO:0030520 (55) <br> intracellular estrogen receptor signaling pathway <br> 4.492e-03","GO:0072359 (1099) <br> circulatory system development <br> 4.590e-03","GO:0009260 (182) <br> ribonucleotide biosynthetic process <br> 4.682e-03","GO:0098911 (12) <br> regulation of ventricular cardiac muscle cell action potential <br> 4.887e-03","GO:0009127 (22) <br> purine nucleoside monophosphate biosynthetic process <br> 4.903e-03","GO:0044380 (58) <br> protein localization to cytoskeleton <br> 4.922e-03","GO:0071345 (831) <br> cellular response to cytokine stimulus <br> 4.929e-03","GO:0043436 (966) <br> oxoacid metabolic process <br> 4.971e-03","GO:0031122 (58) <br> cytoplasmic microtubule organization <br> 5.075e-03","GO:0098901 (27) <br> regulation of cardiac muscle cell action potential <br> 5.135e-03","GO:0006997 (142) <br> nucleus organization <br> 5.153e-03","GO:0032922 (69) <br> circadian regulation of gene expression <br> 5.179e-03","GO:0002159 (4) <br> desmosome assembly <br> 5.218e-03","GO:0060147 (42) <br> regulation of post-transcriptional gene silencing <br> 5.336e-03","GO:0060966 (42) <br> regulation of gene silencing by RNA <br> 5.336e-03","GO:0071363 (655) <br> cellular response to growth factor stimulus <br> 5.393e-03","GO:0045860 (409) <br> positive regulation of protein kinase activity <br> 5.472e-03","GO:0018108 (373) <br> peptidyl-tyrosine phosphorylation <br> 5.576e-03","GO:0044237 (12865) <br> cellular metabolic process <br> 5.612e-03","GO:0051271 (289) <br> negative regulation of cellular component movement <br> 6.073e-03","GO:0032434 (141) <br> regulation of proteasomal ubiquitin-dependent protein catabolic process <br> 6.117e-03","GO:0033206 (7) <br> meiotic cytokinesis <br> 6.284e-03","GO:0051305 (7) <br> chromosome movement towards spindle pole <br> 6.284e-03","GO:1903076 (104) <br> regulation of protein localization to plasma membrane <br> 6.353e-03","GO:0018212 (376) <br> peptidyl-tyrosine modification <br> 6.800e-03","GO:0010965 (64) <br> regulation of mitotic sister chromatid separation <br> 6.848e-03","GO:0090316 (161) <br> positive regulation of intracellular protein transport <br> 6.854e-03","GO:0104004 (328) <br> cellular response to environmental stimulus <br> 6.895e-03","GO:0071214 (328) <br> cellular response to abiotic stimulus <br> 6.895e-03","GO:0019752 (940) <br> carboxylic acid metabolic process <br> 6.909e-03","GO:0033143 (73) <br> regulation of intracellular steroid hormone receptor signaling pathway <br> 7.223e-03","GO:0040008 (665) <br> regulation of growth <br> 7.431e-03","GO:0048729 (640) <br> tissue morphogenesis <br> 7.579e-03","GO:0006164 (190) <br> purine nucleotide biosynthetic process <br> 7.770e-03","GO:0048813 (140) <br> dendrite morphogenesis <br> 7.818e-03","GO:0034332 (51) <br> adherens junction organization <br> 8.167e-03","GO:0033045 (73) <br> regulation of sister chromatid segregation <br> 8.285e-03","GO:0030522 (264) <br> intracellular receptor signaling pathway <br> 8.411e-03","GO:0044784 (65) <br> metaphase/anaphase transition of cell cycle <br> 8.428e-03","GO:0006259 (1023) <br> DNA metabolic process <br> 8.575e-03","GO:2000146 (281) <br> negative regulation of cell motility <br> 8.583e-03","GO:2000637 (22) <br> positive regulation of miRNA-mediated gene silencing <br> 8.682e-03","GO:0086002 (51) <br> cardiac muscle cell action potential involved in contraction <br> 8.797e-03","GO:0016567 (849) <br> protein ubiquitination <br> 9.072e-03","GO:0030837 (68) <br> negative regulation of actin filament polymerization <br> 9.268e-03","GO:0009161 (60) <br> ribonucleoside monophosphate metabolic process <br> 9.464e-03","GO:0010557 (1859) <br> positive regulation of macromolecule biosynthetic process <br> 9.610e-03","GO:0051642 (33) <br> centrosome localization <br> 9.612e-03","GO:0051170 (167) <br> import into nucleus <br> 9.627e-03","GO:0060996 (93) <br> dendritic spine development <br> 9.907e-03","GO:0006979 (420) <br> response to oxidative stress <br> 9.957e-03","GO:0009132 (131) <br> nucleoside diphosphate metabolic process <br> 1.008e-02","GO:0061337 (96) <br> cardiac conduction <br> 1.026e-02","GO:0061136 (195) <br> regulation of proteasomal protein catabolic process <br> 1.048e-02","GO:0030071 (60) <br> regulation of mitotic metaphase/anaphase transition <br> 1.052e-02","GO:0010591 (41) <br> regulation of lamellipodium assembly <br> 1.072e-02","GO:0044090 (21) <br> positive regulation of vacuole organization <br> 1.073e-02","GO:0010506 (339) <br> regulation of autophagy <br> 1.075e-02","GO:1903364 (158) <br> positive regulation of cellular protein catabolic process <br> 1.135e-02","GO:0097191 (223) <br> extrinsic apoptotic signaling pathway <br> 1.159e-02","GO:0071346 (128) <br> cellular response to interferon-gamma <br> 1.173e-02","GO:0032232 (31) <br> negative regulation of actin filament bundle assembly <br> 1.173e-02","GO:0034599 (274) <br> cellular response to oxidative stress <br> 1.175e-02","GO:1900370 (23) <br> positive regulation of post-transcriptional gene silencing by RNA <br> 1.277e-02","GO:0060148 (23) <br> positive regulation of post-transcriptional gene silencing <br> 1.277e-02","GO:0001765 (9) <br> membrane raft assembly <br> 1.285e-02","GO:0090557 (11) <br> establishment of endothelial intestinal barrier <br> 1.313e-02","GO:1905820 (25) <br> positive regulation of chromosome separation <br> 1.318e-02","GO:0051146 (268) <br> striated muscle cell differentiation <br> 1.341e-02","GO:0006399 (197) <br> tRNA metabolic process <br> 1.344e-02","GO:0006415 (18) <br> translational termination <br> 1.349e-02","GO:0042059 (31) <br> negative regulation of epidermal growth factor receptor signaling pathway <br> 1.350e-02","GO:0031668 (250) <br> cellular response to extracellular stimulus <br> 1.395e-02","GO:0007080 (54) <br> mitotic metaphase plate congression <br> 1.410e-02","GO:0072522 (200) <br> purine-containing compound biosynthetic process <br> 1.410e-02","GO:0009059 (4770) <br> macromolecule biosynthetic process <br> 1.415e-02","GO:0051255 (12) <br> spindle midzone assembly <br> 1.499e-02","GO:0048259 (105) <br> regulation of receptor-mediated endocytosis <br> 1.525e-02","GO:0043588 (290) <br> skin development <br> 1.529e-02","GO:0051240 (1434) <br> positive regulation of multicellular organismal process <br> 1.530e-02","GO:0034101 (132) <br> erythrocyte homeostasis <br> 1.560e-02","GO:0060284 (497) <br> regulation of cell development <br> 1.562e-02","GO:0034605 (65) <br> cellular response to heat <br> 1.575e-02","GO:0006449 (11) <br> regulation of translational termination <br> 1.580e-02","GO:0043558 (15) <br> regulation of translational initiation in response to stress <br> 1.581e-02","GO:1903008 (127) <br> organelle disassembly <br> 1.583e-02","GO:0010770 (80) <br> positive regulation of cell morphogenesis involved in differentiation <br> 1.592e-02","GO:1990748 (110) <br> cellular detoxification <br> 1.599e-02","GO:0043409 (169) <br> negative regulation of MAPK cascade <br> 1.611e-02","GO:0051016 (29) <br> barbed-end actin filament capping <br> 1.629e-02","GO:1903312 (98) <br> negative regulation of mRNA metabolic process <br> 1.634e-02","GO:0010556 (3926) <br> regulation of macromolecule biosynthetic process <br> 1.670e-02","GO:0070830 (66) <br> bicellular tight junction assembly <br> 1.687e-02","GO:0086065 (57) <br> cell communication involved in cardiac conduction <br> 1.702e-02","GO:0000184 (44) <br> nuclear-transcribed mRNA catabolic process, nonsense-mediated decay <br> 1.705e-02","GO:0030336 (265) <br> negative regulation of cell migration <br> 1.714e-02","GO:0009185 (112) <br> ribonucleoside diphosphate metabolic process <br> 1.715e-02","GO:1902099 (63) <br> regulation of metaphase/anaphase transition of cell cycle <br> 1.726e-02","GO:0031333 (147) <br> negative regulation of protein-containing complex assembly <br> 1.821e-02","GO:0106027 (88) <br> neuron projection organization <br> 1.869e-02","GO:0043467 (130) <br> regulation of generation of precursor metabolites and energy <br> 1.883e-02","GO:0035239 (854) <br> tube morphogenesis <br> 1.902e-02","GO:0060964 (39) <br> regulation of miRNA-mediated gene silencing <br> 1.903e-02","GO:0006606 (163) <br> protein import into nucleus <br> 1.920e-02","GO:0000082 (205) <br> G1/S transition of mitotic cell cycle <br> 1.923e-02","GO:0086042 (7) <br> cardiac muscle cell-cardiac muscle cell adhesion <br> 1.937e-02","GO:0032446 (907) <br> protein modification by small protein conjugation <br> 1.939e-02","GO:0014902 (122) <br> myotube differentiation <br> 1.958e-02","GO:0007623 (210) <br> circadian rhythm <br> 1.988e-02","GO:0009889 (4145) <br> regulation of biosynthetic process <br> 1.990e-02","GO:0070849 (46) <br> response to epidermal growth factor <br> 2.040e-02","GO:0034063 (27) <br> stress granule assembly <br> 2.073e-02","GO:0030838 (53) <br> positive regulation of actin filament polymerization <br> 2.091e-02","GO:1901988 (261) <br> negative regulation of cell cycle phase transition <br> 2.126e-02","GO:0031326 (4085) <br> regulation of cellular biosynthetic process <br> 2.129e-02","GO:0016072 (264) <br> rRNA metabolic process <br> 2.139e-02","GO:0071174 (39) <br> mitotic spindle checkpoint signaling <br> 2.151e-02","GO:0007094 (39) <br> mitotic spindle assembly checkpoint signaling <br> 2.151e-02","GO:0071173 (39) <br> spindle assembly checkpoint signaling <br> 2.151e-02","GO:0055001 (178) <br> muscle cell development <br> 2.169e-02","GO:0030518 (116) <br> intracellular steroid hormone receptor signaling pathway <br> 2.260e-02","GO:0051256 (8) <br> mitotic spindle midzone assembly <br> 2.270e-02","GO:0032272 (83) <br> negative regulation of protein polymerization <br> 2.318e-02","GO:0051292 (9) <br> nuclear pore complex assembly <br> 2.320e-02","GO:0051094 (1268) <br> positive regulation of developmental process <br> 2.423e-02","GO:2000810 (20) <br> regulation of bicellular tight junction assembly <br> 2.430e-02","GO:0086073 (6) <br> bundle of His cell-Purkinje myocyte adhesion involved in cell communication <br> 2.533e-02","GO:0090407 (561) <br> organophosphate biosynthetic process <br> 2.537e-02","GO:0002262 (162) <br> myeloid cell homeostasis <br> 2.548e-02","GO:0010038 (350) <br> response to metal ion <br> 2.597e-02","GO:0031328 (1964) <br> positive regulation of cellular biosynthetic process <br> 2.611e-02","GO:0097237 (118) <br> cellular response to toxic substance <br> 2.686e-02","GO:0034341 (149) <br> response to interferon-gamma <br> 2.691e-02","GO:0070647 (1067) <br> protein modification by small protein conjugation or removal <br> 2.718e-02","GO:0007249 (288) <br> I-kappaB kinase/NF-kappaB signaling <br> 2.729e-02","GO:0031577 (40) <br> spindle checkpoint signaling <br> 2.811e-02","GO:0042063 (294) <br> gliogenesis <br> 2.844e-02","GO:0009168 (20) <br> purine ribonucleoside monophosphate biosynthetic process <br> 2.927e-02","GO:0035295 (1054) <br> tube development <br> 3.077e-02","GO:1901214 (312) <br> regulation of neuron death <br> 3.079e-02","GO:0032092 (86) <br> positive regulation of protein binding <br> 3.136e-02","GO:0043933 (3395) <br> protein-containing complex organization <br> 3.159e-02","GO:0035967 (118) <br> cellular response to topologically incorrect protein <br> 3.182e-02","GO:0046822 (108) <br> regulation of nucleocytoplasmic transport <br> 3.192e-02","GO:0051223 (498) <br> regulation of protein transport <br> 3.207e-02","GO:0044387 (7) <br> negative regulation of protein kinase activity by regulation of protein phosphorylation <br> 3.233e-02","GO:0006177 (14) <br> GMP biosynthetic process <br> 3.235e-02","GO:0009987 (19395) <br> cellular process <br> 3.253e-02","GO:0043502 (78) <br> regulation of muscle adaptation <br> 3.255e-02","GO:0033047 (48) <br> regulation of mitotic sister chromatid segregation <br> 3.307e-02","GO:0043122 (255) <br> regulation of I-kappaB kinase/NF-kappaB signaling <br> 3.332e-02","GO:0140253 (59) <br> cell-cell fusion <br> 3.428e-02","GO:0000768 (59) <br> syncytium formation by plasma membrane fusion <br> 3.428e-02","GO:1904261 (5) <br> positive regulation of basement membrane assembly involved in embryonic body morphogenesis <br> 3.428e-02","GO:1904259 (5) <br> regulation of basement membrane assembly involved in embryonic body morphogenesis <br> 3.428e-02","GO:2001197 (5) <br> basement membrane assembly involved in embryonic body morphogenesis <br> 3.428e-02","GO:0110011 (5) <br> regulation of basement membrane organization <br> 3.428e-02","GO:0006359 (30) <br> regulation of transcription by RNA polymerase III <br> 3.448e-02","GO:2001251 (90) <br> negative regulation of chromosome organization <br> 3.454e-02","GO:0007507 (574) <br> heart development <br> 3.624e-02","GO:0045841 (41) <br> negative regulation of mitotic metaphase/anaphase transition <br> 3.642e-02","GO:0010976 (149) <br> positive regulation of neuron projection development <br> 3.657e-02","GO:0048008 (56) <br> platelet-derived growth factor receptor signaling pathway <br> 3.671e-02","GO:0010629 (1579) <br> negative regulation of gene expression <br> 3.673e-02","GO:0035633 (35) <br> maintenance of blood-brain barrier <br> 3.694e-02","GO:0007520 (46) <br> myoblast fusion <br> 3.817e-02","GO:2001234 (224) <br> negative regulation of apoptotic signaling pathway <br> 3.883e-02","GO:0021543 (172) <br> pallium development <br> 3.981e-02","GO:0044843 (235) <br> cell cycle G1/S phase transition <br> 3.989e-02","GO:0034383 (27) <br> low-density lipoprotein particle clearance <br> 4.010e-02","GO:1901185 (35) <br> negative regulation of ERBB signaling pathway <br> 4.026e-02","GO:0009891 (1999) <br> positive regulation of biosynthetic process <br> 4.122e-02","GO:0060397 (8) <br> growth hormone receptor signaling pathway via JAK-STAT <br> 4.170e-02","GO:0070918 (52) <br> primary sncRNA processing <br> 4.192e-02","GO:0007166 (2811) <br> cell surface receptor signaling pathway <br> 4.205e-02","GO:0010041 (4) <br> response to iron(III) ion <br> 4.257e-02","GO:0061077 (70) <br> chaperone-mediated protein folding <br> 4.304e-02","GO:0008356 (19) <br> asymmetric cell division <br> 4.407e-02","GO:0010948 (305) <br> negative regulation of cell cycle process <br> 4.446e-02","GO:0071364 (42) <br> cellular response to epidermal growth factor stimulus <br> 4.641e-02","GO:0000460 (36) <br> maturation of 5.8S rRNA <br> 4.719e-02","GO:0071383 (206) <br> cellular response to steroid hormone stimulus <br> 4.858e-02","GO:0070201 (523) <br> regulation of establishment of protein localization <br> 4.975e-02"],"key":["GO:0006996","GO:0007010","GO:0051128","GO:0030029","GO:0030036","GO:0044260","GO:0071840","GO:0044267","GO:0016043","GO:0008104","GO:1901564","GO:0007049","GO:0033036","GO:0032268","GO:0033043","GO:0006793","GO:0051246","GO:0006796","GO:0051130","GO:0019538","GO:0051641","GO:0051179","GO:0048522","GO:0097435","GO:0022402","GO:0044087","GO:0048518","GO:0065009","GO:0043412","GO:0050790","GO:0035556","GO:0007015","GO:0044089","GO:0051493","GO:0036211","GO:0006464","GO:0051247","GO:0044248","GO:0000278","GO:0034613","GO:0070727","GO:0009894","GO:0032270","GO:0000902","GO:0009056","GO:0006928","GO:2000112","GO:0016192","GO:1903047","GO:0051649","GO:0030030","GO:0006417","GO:1902903","GO:0016310","GO:0032879","GO:0009057","GO:0051301","GO:0016477","GO:0032970","GO:0031329","GO:0065008","GO:0007163","GO:0050794","GO:0010638","GO:0051234","GO:0034248","GO:1902531","GO:0120036","GO:0048856","GO:0034330","GO:0048523","GO:0045184","GO:0044265","GO:0000226","GO:0007264","GO:0060341","GO:0051656","GO:0048583","GO:0033554","GO:0046907","GO:0019220","GO:0051174","GO:0051716","GO:0043254","GO:0022411","GO:0051640","GO:0048870","GO:0051674","GO:0032956","GO:0006950","GO:0031325","GO:0009966","GO:0048468","GO:0006518","GO:0032502","GO:0023051","GO:1901575","GO:0043043","GO:0051248","GO:0015031","GO:0006412","GO:0032269","GO:0006468","GO:1901566","GO:0031399","GO:0010646","GO:0006810","GO:0051173","GO:0031323","GO:0007017","GO:0007275","GO:0040011","GO:0110053","GO:0071705","GO:0009653","GO:0080090","GO:0070887","GO:0000904","GO:0031032","GO:0009893","GO:0010564","GO:0019058","GO:0006413","GO:0043603","GO:0031175","GO:0048869","GO:0010604","GO:0007154","GO:0016032","GO:0045216","GO:0043604","GO:0048666","GO:0009896","GO:0032880","GO:0071702","GO:0007051","GO:0140694","GO:0048731","GO:0030154","GO:0051171","GO:0051726","GO:0032989","GO:0010639","GO:0044093","GO:0006402","GO:0048858","GO:0007059","GO:0006915","GO:1903829","GO:0050789","GO:0050896","GO:0065007","GO:0042325","GO:0007155","GO:0032990","GO:0012501","GO:0008219","GO:0150115","GO:0034645","GO:0000910","GO:0051017","GO:1903311","GO:0061013","GO:0061572","GO:0023052","GO:0120039","GO:0010562","GO:0045937","GO:0034329","GO:0044270","GO:0097581","GO:0019439","GO:0048519","GO:0051129","GO:0051258","GO:0006446","GO:0031331","GO:0010033","GO:0051270","GO:0034655","GO:0046700","GO:0031401","GO:0051495","GO:0043069","GO:0007265","GO:0140014","GO:0006401","GO:0048812","GO:0060627","GO:0022008","GO:0022607","GO:0007399","GO:2000145","GO:0030334","GO:0060548","GO:1902117","GO:0044092","GO:0071310","GO:0032271","GO:0070925","GO:0031344","GO:0043067","GO:0008154","GO:0043085","GO:0032984","GO:0042327","GO:0061024","GO:0031334","GO:0010941","GO:1902904","GO:0042981","GO:0043488","GO:0007044","GO:0022604","GO:0000070","GO:0048193","GO:1902850","GO:0055086","GO:0042060","GO:0043487","GO:1901361","GO:0120035","GO:0009611","GO:0030182","GO:0044085","GO:0000819","GO:0043066","GO:0048699","GO:0034250","GO:0042221","GO:0051225","GO:0051494","GO:0033365","GO:0040012","GO:0010256","GO:0043086","GO:0030163","GO:0045727","GO:0051056","GO:1990778","GO:0007165","GO:0001932","GO:1902905","GO:0051653","GO:0098813","GO:0051983","GO:0061640","GO:0032535","GO:0051336","GO:0010628","GO:0051050","GO:0120032","GO:0052126","GO:0060491","GO:0051701","GO:0018193","GO:0030010","GO:0006897","GO:0032501","GO:0043244","GO:0044770","GO:0051049","GO:0044772","GO:0051338","GO:0031400","GO:0048285","GO:0006900","GO:0050793","GO:0007169","GO:0030866","GO:0017148","GO:0019222","GO:0000281","GO:0031346","GO:0007043","GO:0009117","GO:0000280","GO:1901699","GO:0009719","GO:0051276","GO:0001885","GO:0045936","GO:0048667","GO:0010563","GO:0001934","GO:1905818","GO:0032506","GO:0120031","GO:0007346","GO:0098609","GO:0051668","GO:1905477","GO:0030031","GO:0031532","GO:0032386","GO:0006753","GO:0044257","GO:0071495","GO:0090066","GO:1901576","GO:0008064","GO:0031589","GO:0030100","GO:0006470","GO:0060255","GO:1901698","GO:0030832","GO:0034660","GO:2000113","GO:0030041","GO:0030335","GO:0007052","GO:1902115","GO:0043087","GO:0007160","GO:0019637","GO:0071417","GO:0043632","GO:0061014","GO:1901565","GO:0030048","GO:0032273","GO:0034249","GO:0044249","GO:0043009","GO:0002064","GO:0009058","GO:0090130","GO:2000147","GO:0043555","GO:0044409","GO:0032886","GO:0031647","GO:0016197","GO:0035088","GO:0061245","GO:0010631","GO:0098876","GO:0009792","GO:0051304","GO:0019941","GO:0006511","GO:0034470","GO:0030865","GO:0051293","GO:0090132","GO:0060429","GO:0006886","GO:0030032","GO:0072659","GO:0051603","GO:0051348","GO:0046718","GO:0050658","GO:0050657","GO:0019693","GO:0043549","GO:0051272","GO:0016311","GO:0001701","GO:0010243","GO:0048646","GO:0040017","GO:0048513","GO:0070507","GO:0006913","GO:0051169","GO:0043434","GO:0023056","GO:0051261","GO:1903313","GO:1901990","GO:0007417","GO:0031324","GO:0051236","GO:0043393","GO:1901880","GO:0045446","GO:0034504","GO:0048041","GO:0009888","GO:0000956","GO:0051172","GO:1905475","GO:0071375","GO:0030833","GO:0018209","GO:0031109","GO:0150116","GO:1901879","GO:0071407","GO:0018105","GO:0032869","GO:0032388","GO:0006914","GO:0061919","GO:0010647","GO:0006903","GO:1902532","GO:0022603","GO:0006898","GO:0042176","GO:0090307","GO:0007167","GO:0009968","GO:0061157","GO:0000165","GO:0009725","GO:0051347","GO:0031529","GO:0042326","GO:0010608","GO:0033674","GO:0110020","GO:0043297","GO:0097190","GO:0016049","GO:0044271","GO:0046777","GO:0045197","GO:0009790","GO:0043242","GO:1901987","GO:0001952","GO:0050000","GO:1904377","GO:1904375","GO:0051239","GO:0008286","GO:0043408","GO:1901653","GO:0030038","GO:0043149","GO:0050779","GO:0048584","GO:0009967","GO:1902743","GO:0062197","GO:0046785","GO:0023057","GO:0001667","GO:0051231","GO:1901652","GO:0032870","GO:0003158","GO:0016050","GO:0006403","GO:0050821","GO:0008283","GO:1903362","GO:0010648","GO:0007409","GO:0120193","GO:0014070","GO:0006892","GO:0072657","GO:1902533","GO:0016358","GO:0032868","GO:0010035","GO:0051303","GO:0007229","GO:0051028","GO:0045732","GO:0045859","GO:0046037","GO:0044403","GO:0042692","GO:0010632","GO:0050792","GO:0033673","GO:0006901","GO:0009150","GO:0030042","GO:0007420","GO:0010468","GO:0046578","GO:0006163","GO:0051492","GO:0009259","GO:0120192","GO:0048524","GO:0010975","GO:0060322","GO:0080134","GO:0071496","GO:0008360","GO:0090257","GO:0061564","GO:0045787","GO:0051345","GO:1901888","GO:0051650","GO:0000288","GO:0009605","GO:0006457","GO:0030835","GO:0015931","GO:0072521","GO:0035303","GO:0009628","GO:0007093","GO:0042592","GO:2000058","GO:0009126","GO:0051222","GO:0044837","GO:0032231","GO:1903078","GO:0031110","GO:0061061","GO:0090068","GO:0031116","GO:0031330","GO:2001233","GO:0120181","GO:0120182","GO:0120183","GO:0120180","GO:0045070","GO:0033044","GO:0051310","GO:1901701","GO:0032465","GO:1900180","GO:0043161","GO:0045069","GO:0044281","GO:0009892","GO:0061028","GO:0048199","GO:0070848","GO:0030855","GO:0007020","GO:0044419","GO:0030834","GO:0016344","GO:0010498","GO:0003012","GO:0033146","GO:0051648","GO:0032456","GO:0045930","GO:0051098","GO:0048585","GO:0048207","GO:0048208","GO:0040001","GO:0051294","GO:0097178","GO:0043547","GO:0019079","GO:0120034","GO:0061842","GO:0007173","GO:1903050","GO:0046390","GO:0040007","GO:1901700","GO:0002183","GO:1901991","GO:0000022","GO:0051099","GO:0002181","GO:1904951","GO:0009152","GO:0035304","GO:0051893","GO:0090109","GO:0032091","GO:0009991","GO:0072698","GO:0001558","GO:0034097","GO:0080135","GO:0031113","GO:0016236","GO:0150105","GO:0002934","GO:1903900","GO:0009895","GO:0031112","GO:0009408","GO:0072697","GO:0034109","GO:0000075","GO:0051306","GO:0045454","GO:0009165","GO:0086001","GO:0090114","GO:0009167","GO:0010605","GO:2001236","GO:0000915","GO:0000912","GO:0007005","GO:0038127","GO:0001933","GO:0051693","GO:0007266","GO:0006893","GO:1901293","GO:0009123","GO:0010769","GO:1902745","GO:0006082","GO:0006469","GO:0045786","GO:0030155","GO:0031023","GO:0034314","GO:0032467","GO:0010634","GO:0051497","GO:0010810","GO:0030953","GO:1900368","GO:0006974","GO:0002011","GO:0006508","GO:0007091","GO:0030520","GO:0072359","GO:0009260","GO:0098911","GO:0009127","GO:0044380","GO:0071345","GO:0043436","GO:0031122","GO:0098901","GO:0006997","GO:0032922","GO:0002159","GO:0060147","GO:0060966","GO:0071363","GO:0045860","GO:0018108","GO:0044237","GO:0051271","GO:0032434","GO:0033206","GO:0051305","GO:1903076","GO:0018212","GO:0010965","GO:0090316","GO:0104004","GO:0071214","GO:0019752","GO:0033143","GO:0040008","GO:0048729","GO:0006164","GO:0048813","GO:0034332","GO:0033045","GO:0030522","GO:0044784","GO:0006259","GO:2000146","GO:2000637","GO:0086002","GO:0016567","GO:0030837","GO:0009161","GO:0010557","GO:0051642","GO:0051170","GO:0060996","GO:0006979","GO:0009132","GO:0061337","GO:0061136","GO:0030071","GO:0010591","GO:0044090","GO:0010506","GO:1903364","GO:0097191","GO:0071346","GO:0032232","GO:0034599","GO:1900370","GO:0060148","GO:0001765","GO:0090557","GO:1905820","GO:0051146","GO:0006399","GO:0006415","GO:0042059","GO:0031668","GO:0007080","GO:0072522","GO:0009059","GO:0051255","GO:0048259","GO:0043588","GO:0051240","GO:0034101","GO:0060284","GO:0034605","GO:0006449","GO:0043558","GO:1903008","GO:0010770","GO:1990748","GO:0043409","GO:0051016","GO:1903312","GO:0010556","GO:0070830","GO:0086065","GO:0000184","GO:0030336","GO:0009185","GO:1902099","GO:0031333","GO:0106027","GO:0043467","GO:0035239","GO:0060964","GO:0006606","GO:0000082","GO:0086042","GO:0032446","GO:0014902","GO:0007623","GO:0009889","GO:0070849","GO:0034063","GO:0030838","GO:1901988","GO:0031326","GO:0016072","GO:0071174","GO:0007094","GO:0071173","GO:0055001","GO:0030518","GO:0051256","GO:0032272","GO:0051292","GO:0051094","GO:2000810","GO:0086073","GO:0090407","GO:0002262","GO:0010038","GO:0031328","GO:0097237","GO:0034341","GO:0070647","GO:0007249","GO:0031577","GO:0042063","GO:0009168","GO:0035295","GO:1901214","GO:0032092","GO:0043933","GO:0035967","GO:0046822","GO:0051223","GO:0044387","GO:0006177","GO:0009987","GO:0043502","GO:0033047","GO:0043122","GO:0140253","GO:0000768","GO:1904261","GO:1904259","GO:2001197","GO:0110011","GO:0006359","GO:2001251","GO:0007507","GO:0045841","GO:0010976","GO:0048008","GO:0010629","GO:0035633","GO:0007520","GO:2001234","GO:0021543","GO:0044843","GO:0034383","GO:1901185","GO:0009891","GO:0060397","GO:0070918","GO:0007166","GO:0010041","GO:0061077","GO:0008356","GO:0010948","GO:0071364","GO:0000460","GO:0071383","GO:0070201"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[20.8547085497431,19.6429340246934,20.2085877960986,18.864656569561,18.7055504633317,21.3222866420815,21.7080297784974,21.1281505031929,21.6792322444023,20.4028868881456,21.4511386599994,19.8969694448129,20.5966905748038,20.2586423611478,19.3251431241161,20.464578834946,20.3420343983675,20.454965220887,19.2155522047458,21.2646563902221,20.5596909045512,21.4493506407295,21.3062754144974,18.8202439587487,19.4020793192827,19.0824988637835,21.4107321783704,20.5672678709767,20.8736212522412,20.2455411710308,20.3862284722067,18.0539340374567,18.2011556693834,18.2889404059498,20.8059733096351,20.8059733096351,19.6777349626702,20.1659001717058,19.0249365905889,19.9919387768348,19.997829396907,19.1679989309601,19.6006949367672,19.1719246025406,20.3531242243581,20.0996535935105,18.1607785644195,19.7692686236239,18.7795858929826,20.1642046175619,19.7048579881692,17.8659558065317,17.7941823919817,19.9714178732024,20.4301211493866,19.5569006856614,18.5665372569278,19.6499763854013,17.8622745505421,18.9722550706133,20.8216112888489,17.0425738758488,22.0755942244702,18.2347830558071,21.1331848217317,18.0571124546786,19.8363532261073,19.6708560194737,21.3198696761996,18.6589261540067,21.1218854090215,19.9760833009366,19.3422911397883,18.5319808127071,18.2375459754686,18.7541706757172,17.9270031699803,20.8611505031046,19.9667337081026,19.8912615253555,19.58225556087,19.5841124056931,21.6092206957528,17.8986295758108,18.1251328561888,18.3229519409635,19.8090563770885,19.8090563770885,17.7058710060953,20.847610072841,20.6518557458523,20.5255710545725,20.1050237313276,19.0071275958239,21.4278431882964,20.6809751106411,20.1103690832062,18.7449666515227,19.2003323326785,19.8998130046544,18.6902013318817,19.1157801624292,19.7557017851666,19.8490262399862,19.7065327143822,20.6755723496184,21.0939859721281,20.5803083593462,21.3691455037472,19.0219863045961,21.1004115767538,19.9382269127741,17.3071889925447,20.1849443457206,20.4197581460106,21.3315022542897,20.5548791650762,18.7188279862047,16.8222146795141,20.8241437517082,18.6668185160786,17.5332018280727,16.1523280086149,19.3591988024863,19.09786534936,20.9462106361808,20.7212083352719,21.4621588517154,17.9582033458579,16.8768537716112,18.9660880845565,19.2912836129535,18.262147933236,18.9316065728168,20.3978225784418,16.7900023015197,17.8134536510378,20.9821711600585,20.9393896429181,21.2995168292475,19.268574456577,18.7831746424171,17.7058710060953,19.7305533981685,17.2959575795351,18.5579865833401,17.6722355648885,19.9600092494455,18.0822628930065,22.1780367145862,21.8169086175002,22.2413961654309,19.4403484327659,19.6743004830516,18.5980946257729,19.9952150204314,20.0930561050808,15.7090215871995,19.3636677464856,16.6798722841062,16.4814853684313,17.5932840108147,16.8768537716112,16.5115854735424,21.4520315619783,18.5493774709702,19.0086215817833,19.0086215817833,17.975219480059,18.172444713275,15.5644039696379,18.2152742453374,21.3441954418206,18.6329271477974,17.3512048639261,15.3831759264344,18.0634459974889,20.5512568820243,19.192650328488,18.0346981573455,18.1549056295933,19.1548212202776,16.8222146795141,18.9966185240434,17.6891601077563,17.4752355820436,17.5426231954712,18.5165594103085,18.2942322275427,19.7990413721648,21.054881363787,20.2894885886194,19.0909045087562,19.0056317909676,19.1334359838834,15.3397884126306,19.2960106600444,20.2695330698625,16.884486975454,19.0160643770272,18.5428818203241,19.6385112341163,16.8458946382948,19.3625520632973,17.1658618982991,18.8846297537959,18.9059095074014,16.8062014701714,19.7597082577078,16.5699357330514,19.6125290745836,16.7570292193963,15.6018860534727,17.5045325447657,16.625974973076,17.5189440263249,16.471308742156,18.3735689215712,17.9920161079506,16.8614603770872,18.2915890540975,18.5076618574901,18.3636050964249,19.5635261480143,21.1165561378763,16.8691783884681,18.969175438885,19.6278288297653,16.5699357330514,20.988461845188,16.0602198573189,16.5507513357352,19.2924671078222,19.152168489186,18.3434405375302,18.7795858929826,19.1508399554176,16.3092571582485,17.5045325447657,17.6377240883285,21.3600506008709,19.2781881859394,16.6621348212509,14.7907258648956,17.388575995039,15.6564117262202,15.825004640016,17.726466984073,19.1455112942199,19.322838205313,19.0101137531149,16.8062014701714,16.6798722841062,16.8222146795141,16.8062014701714,19.4845144493855,16.3653377457502,18.5558398139169,21.6022562901932,16.1650226729754,18.3125829626229,19.8608300883505,18.0282222896175,19.0056317909676,18.2096454189511,18.2096454189511,14.8820117014956,20.2869495000605,18.4941969186411,14.0961442491203,16.6351023083939,21.5958713493962,15.2723067404551,17.6550920625975,16.3762954362776,18.1666248483086,18.0728886271239,18.6429906801522,19.7313730524165,19.2318279913243,15.0232762450614,17.9513341853314,18.3785217396688,17.954773290037,18.7431176624848,15.2723067404551,14.0961442491203,18.3510396876875,18.1371245536588,19.0352070936785,18.9814480012294,15.8874237642864,18.3859150060615,15.8720643257765,17.6507713330935,18.1954645211632,18.9123840842271,19.5153738461767,18.1840060996778,21.3238949752454,16.5016227680512,17.7467467596604,16.8768537716112,17.4503225782439,21.5104036235208,19.2817737430326,16.5115854735424,18.3883699100769,16.8222146795141,16.5888635180867,18.2729323128843,16.1266008282873,16.8768537716112,17.7667195392501,17.0958424386257,19.0782742848907,18.5231916695226,18.670746119908,15.6202823424339,19.5483277811012,16.1900834303366,15.6018860534727,16.8222146795141,21.3089264138074,18.5813625967166,16.8996286064323,21.3412337447138,17.480163120858,18.3357958241329,12.7847893425189,16.4403351484814,17.232464869954,17.4752355820436,17.0827178306593,14.5180474895371,14.5180474895371,17.4351497482074,16.2977734456596,18.6248177827926,15.7261543098391,18.6489902543207,18.626850042882,17.9920161079506,14.9113014863856,14.6258282642228,17.4503225782439,19.3736626702624,19.26252688917,15.1774535138295,17.388575995039,18.8202439587487,17.2846322728973,16.3653377457502,16.5313035845537,16.5313035845537,17.8659558065317,18.8062524032922,18.3710852662863,18.0085986166085,17.869626749228,19.1693088948601,19.3216840939803,18.3760477399893,20.7377263762327,16.4507346585004,17.5332018280727,17.5332018280727,17.8842084896359,19.8542050385501,16.0602198573189,16.0737400865481,17.5658897337121,19.204155208303,20.3274059293165,16.5603760262376,16.8380459769915,15.2723067404551,15.8720643257765,17.4752355820436,15.4044151730077,20.0095219958548,16.1004102564265,20.259163314556,16.6973887091722,17.4094747072172,16.2977734456596,17.6377240883285,16.0737400865481,15.0500590267378,15.6018860534727,18.3908200845062,17.5332018280727,16.8458946382948,16.884486975454,18.3203675801052,18.3203675801052,19.8512485236043,14.2345175300099,18.2320141729182,19.0597974062312,17.1533948742987,17.9057809314112,15.1277454070193,19.0895075731358,19.4413942461222,15.5453070901116,18.8375144993125,18.922032745784,18.4054227851936,14.6938508911943,17.7467467596604,19.2865380210281,18.1782383883591,15.6916899076018,15.2017202315928,18.4459078882545,18.1811254754696,21.0996720728518,17.0493465147958,14.2781699698326,19.2936494458378,15.4865223327699,17.9409621155194,16.0188918229245,15.5063724372317,15.0232762450614,16.1004102564265,20.4033461666366,15.9473099062644,18.6043121628591,17.6933576581111,15.8089797392543,15.8089797392543,15.6202823424339,20.1488315500731,19.6947595414818,14.5906996788533,17.5796572794936,15.4044151730077,19.5540498924541,17.954773290037,10.7797707327282,18.1430788638952,18.4222498103304,16.1135644599747,17.4899641684333,16.8458946382948,16.7818316810823,20.0210983047449,17.2789339363093,19.5492833746344,18.0217139542686,15.2951326334938,19.0175475387395,15.8720643257765,18.8562383619264,19.1226011162752,17.0958424386257,17.2146209988566,18.3125829626229,15.4664084400791,15.9176711280911,16.1650226729754,17.1217205230222,18.5813625967166,13.236279828353,17.4146490786193,17.7467467596604,17.0357677197234,16.5115854735424,17.0425738758488,12.4471552623873,17.7547722088211,14.8820117014956,18.7921014743194,21.2572069237747,16.7818316810823,17.8585829253593,15.5453070901116,17.8286697745514,15.152799219118,14.8820117014956,17.9986745106348,18.8730179355262,19.5114364530828,17.5519785497145,16.4298592558445,17.0493465147958,18.16370502316,17.684949205538,18.400573835677,16.8536992814473,16.625974973076,14.9113014863856,20.4580329248004,16.9585931325711,14.402535702524,17.0425738758488,17.9374863511375,16.1523280086149,19.3170565881304,16.2147179483544,19.8113554757747,16.6075411810639,14.2345175300099,17.4702898630316,9.65202509301152,15.7764011513431,14.8217448063313,15.5259652640403,18.512118469496,17.1843515094562,13.6586847106596,17.2959575795351,17.7305479349705,9.09103163078953,9.09103163078953,9.09103163078953,9.09103163078953,13.5950915152841,16.9585931325711,15.0763886917563,19.3580789722659,15.5644039696379,16.250900190535,18.0411418678934,15.4664084400791,19.9125242261296,20.7570304837231,14.4419633180214,12.7847893425189,18.6469935852872,18.6166556755762,13.891619822505,19.8121208444945,14.7268035556219,3.77952755905512,18.2347830558071,17.9650364933867,13.8363278826937,16.7736129918804,15.1774535138295,17.1344784292061,17.7627491144075,19.7652945534021,11.8731306335789,11.8731306335789,13.9970167186474,13.9970167186474,14.2345175300099,17.3457806327537,16.1650226729754,15.8408580779897,13.7199905291574,15.7927798038515,17.0627924973149,16.7653456892551,19.1006386265001,19.7850322505749,13.891619822505,16.7818316810823,10.4664612166769,16.6351023083939,16.471308742156,17.5473090686239,16.579431290721,15.5063724372317,14.9113014863856,14.9113014863856,15.6202823424339,18.1811254754696,14.6258282642228,17.9785961908987,19.0468404725712,18.5897564416008,14.6938508911943,17.5093536439159,12.679039288794,10.7797707327282,16.2745274114414,17.5887572421632,13.891619822505,15.8408580779897,11.0515743331527,15.6202823424339,16.6441709175865,15.0232762450614,13.9451483225709,17.2146209988566,15.1774535138295,13.5290443998576,14.1435704252252,20.6845625780069,16.4298592558445,9.09103163078953,9.09103163078953,18.2862862612016,16.0188918229245,17.5704947342904,14.2345175300099,16.250900190535,14.9400549338198,17.232464869954,15.3176234456008,15.6741548202393,13.891619822505,19.1401592432685,16.8768537716112,17.8474452688872,18.7523353011109,16.3431677806744,13.9970167186474,14.0961442491203,16.3653377457502,13.3142209623286,16.9440857974467,11.0515743331527,14.0961442491203,18.9010312098947,14.7590867667101,19.9061858724592,14.8820117014956,14.6602019413075,19.2805797371324,16.7060656693444,11.2911286539018,12.7847893425189,14.7590867667101,18.9091510378131,19.1102958367164,14.7590867667101,13.236279828353,16.3092571582485,15.0763886917563,7.20137928972192,14.1435704252252,14.1435704252252,18.5855665215127,17.9234899862472,17.7902940264943,22.2298008127361,17.4146490786193,16.2977734456596,9.65202509301152,9.65202509301152,15.7927798038515,17.8019248165931,14.9400549338198,16.5115854735424,17.6022916256912,17.6022916256912,19.0740350722802,15.1774535138295,18.6063778570541,18.553689380115,16.7736129918804,16.2861973405104,14.5180474895371,15.1774535138295,17.2789339363093,14.9682903911511,19.1862113812352,17.3726864188191,12.7847893425189,14.5180474895371,18.9379481001757,15.0500590267378,14.8217448063313,19.9559558959111,13.6586847106596,16.5699357330514,15.6018860534727,17.9616243985144,16.1776072099474,15.6564117262202,16.8142310637125,14.8217448063313,14.0961442491203,12.679039288794,17.6507713330935,16.4814853684313,17.0220535731758,16.1395213611136,13.5290443998576,17.334866389836,12.8846903643046,12.8846903643046,10.4664612166769,11.0515743331527,13.0691852338268,17.3015849292252,16.8301528193582,12.3192674406608,13.5290443998576,17.1965401157473,14.6258282642228,16.8536992814473,21.1009043033778,11.2911286539018,15.8089797392543,17.4198036237959,19.626039050177,16.1900834303366,18.2011556693834,14.9682903911511,11.0515743331527,11.8731306335789,16.1266008282873,15.3397884126306,15.8874237642864,16.5888635180867,13.3888274758207,15.6916899076018,20.8705924622063,14.9960252981067,14.7268035556219,14.2345175300099,17.2846322728973,15.9176711280911,14.9113014863856,16.3653377457502,15.5063724372317,16.1650226729754,18.9458294270912,13.9970167186474,16.5313035845537,16.8920784419944,9.65202509301152,19.0264090785838,16.0602198573189,16.9294247168184,20.9351051236607,14.3206880840709,13.236279828353,14.5906996788533,17.2616938676726,20.9177996918035,17.2789339363093,13.9970167186474,13.9970167186474,13.9970167186474,16.671031515582,15.976363046117,10.097845750688,15.4044151730077,10.4664612166769,19.4672475112412,12.5667614807513,9.09103163078953,18.3710852662863,16.5214787726,17.6975419377751,20.024931617907,16.004852243275,16.3871699094041,19.2417364474579,17.4094747072172,14.0473197023866,17.4402264269597,12.5667614807513,19.2255938683295,17.5284661085774,15.4664084400791,20.6966749970307,16.004852243275,15.8565435187603,18.2039918223307,9.65202509301152,11.6977535466344,22.6771653543307,15.2951326334938,14.402535702524,17.2265428793892,14.7907258648956,14.7907258648956,8.33927853761798,8.33927853761798,8.33927853761798,8.33927853761798,13.46035842381,15.5453070901116,18.4030006233289,14.0961442491203,16.3871699094041,14.6938508911943,19.7492629943839,13.7791594529932,14.3206880840709,17.0289277268582,16.616788179269,17.1023579289126,13.236279828353,13.7791594529932,20.0470441691129,10.097845750688,14.5547841218036,20.4680546340753,7.20137928972192,15.1022795884124,12.4471552623873,17.494837929168,14.1435704252252,13.8363278826937,16.8996286064323,18.2729323128843],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData257a9fd5","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[55.7284052393749,55.3971818407949,64.6597467634394,64.5740183543952,62.1892098846188,64.4804964536196,69.7488968639753,56.0401449086267,57.8988926865407,59.5550096794409,55.8141336484192,55.0036105083645,57.894995940675,55.1127193926026,57.1585109720676,58.8068344732366,54.9802300331706,68.4980414411024,55.9894872123733,62.1970033763501,62.1814163928875,57.209168668321,54.8360504361416,62.0411335417242,62.1736229011562,68.9344769780549,56.0830091131489,55.993383958239,57.8910991948094,59.5511129335753,57.1624077179333,64.4843931994853,59.5277324583814,62.2009001222157,62.1853131387531,59.0445359710411,67.5121647370935,62.1775196470218,58.3742956821497,54.6178326676653,68.6188405629375,55.6933345265841,68.501938186968,58.10542021742,57.0805760547547,69.8229350354226,68.6928787343847,68.6149438170718,61.53455657919,62.1931066304844,68.6227373088031,59.5589064253066,58.2301160851207,67.8472848815392,61.2033331806099,55.6777475431215,60.9110772406863,64.0167836956075,58.3781924280154,59.8472656193645,61.9164376740234,57.0844728006203,63.7907724354,62.0450302875898,58.2534965603146,60.0109289457217,54.6139359217997,67.0991096753349,57.0727825630234,55.8647913446726,57.2676198563058,63.381614119507,68.1395408214628,58.1794583888673,63.4088913405665,69.5345758413647,66.0898524961321,66.1911678886389,58.1210072008826,56.9675704246509,57.8794089572124,58.8107312191022,55.6699540513902,68.1434375673284,55.9037588033291,55.0464747128866,61.0981210422374,71.0932741876237,59.9174070449462,62.2632480560661,62.8477599359132,55.8959653115978,58.8224214566992,58.6353776551481,59.3952430989494,57.2559296187088,63.4205815781634,55.9700034830451,57.8872024489437,57.8833057030781,54.4346856119799,58.183355134733,67.8433881356736,66.9666203159028,58.1872518805986,61.7917418063227,63.8297398940565,59.6602218178134,69.3475320398136,67.960290511643,56.9948476457104,58.1911486264643,68.159024550791,58.0976267256887,55.506290725033,57.1195435134111,59.6641185636791,58.0859364880918,58.0157950625101,63.7946691812656,55.771269443897,65.1663237259736,65.2559488808835,68.2954106560887,55.0503714587522,56.3012268816251,58.514578533313,55.0776486798118,55.6037093716742,55.8725848364039,55.5101874708987,55.9622099913138,61.5384533250556,68.0148449537621,62.274938293663,68.0226384454934,56.9558801870539,69.4722279075143,54.4502725954425,57.3923157240065,55.4010785866605,58.0274853001071,58.2807737813742,59.5316292042471,59.5043519831875,68.8409550772794,58.2651867979116,54.1580166555189,55.4907037415704,66.3158637563397,67.761556472495,62.0606172710524,63.3270596773879,63.4049945947009,61.1955396888786,55.6660573055246,54.6490066345905,58.8146279649679,58.8302149484305,63.6115221255802,69.8190382895569,57.088369546486,55.8842750740008,55.5218777084956,68.0187416996277,63.8492236233847,54.6840773473813,59.3757593696212,60.5564733669124,57.1078532758142,54.2982995066822,54.2944027608166,55.2023445475125,61.8423995025761,55.4984972333017,64.6558500175738,67.7225890138385,57.4001092157378,63.8842943361755,54.3294734736074,59.7732274479172,55.7985466649566,57.7274358684522,60.3577393277644,57.2637231104401,61.8969539446952,68.6578080215939,66.2846897894145,55.8998620574634,61.7761548228601,57.906686178272,55.8686880905382,68.712362463713,61.1448819926252,62.8711404111071,57.2364458893806,55.9739002289107,62.1307586966341,59.4264170658746,54.3021962525479,56.0323514168954,58.6704483679389,62.9179013614949,54.2905060149509,66.113232971326,57.5209083375729,63.8803975903099,55.8531011070757,68.3187911312826,58.0820397422261,55.5296712002269,56.3129171192221,58.0547625211666,65.3182968147339,69.8385220188852,56.258362677103,54.0099403126243,56.0050741958359,54.6373163969936,57.2910003314997,68.6889819885191,63.1906735720902,57.2754133480371,58.4444371077314,67.8745621025987,57.2793100939027,63.8648106068473,70.3373054896881,59.4965584914562,58.0898332339574,55.4946004874361,68.1239538380002,58.3431217152245,58.3275347317619,66.6821578677105,56.2544659312374,57.2598263645745,55.7946499190909,65.2481553891522,61.8579864860387,53.9865598374304,55.9661067371794,57.7313326143178,55.5257744543612,60.5876473338376],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.9035903939572,15.1529131277165,14.3305370676501,13.9194621032505,13.8432424124575,13.7197997080126,13.5725070470381,13.4043767239534,13.0179790390059,13.0073709242437,12.8791054977506,12.7281660693699,12.7175173128439,12.6037896565277,12.5259445940012,12.4258009998182,12.4258009998182,12.2864696408384,11.8476949861581,11.4910253575436,11.026885206856,10.8545165749611,10.8136453381481,10.8136453381481,10.6647658804524,10.5034058156163,10.2176268196185,10.1195752502977,10.0803139373186,10.0430159356978,10.0003445657876,9.98593525856364,8.84963429128125,8.67701105051561,8.61662718058435,8.49696573511157,8.41497017723026,8.39547930229044,8.35143357014754,8.26486137069224,8.21967328779133,8.13350671319254,7.98506058624432,7.97919150699728,7.89172911461576,7.88656458671068,7.62662792812698,7.35945548305089,7.2036707605441,7.06229325578126,6.96558300476573,6.82399165830643,6.57245586728757,6.48095393764389,6.37327081699869,6.30344584999012,6.15003369424254,5.97375976761052,5.91791079480047,5.91577171707171,5.87459721009967,5.81073914113068,5.81073914113068,5.78739133749101,5.78009845188246,5.72812792765182,5.71439692665261,5.57096222596218,5.54625421459567,5.37189742551145,5.34896761541967,5.33070781275234,5.24689419421062,5.17265846647635,5.14483569520607,5.14483569520607,5.01698108470491,4.99390700596996,4.83793991424007,4.61691226584119,4.47602161337296,4.46501784924334,4.45020554716546,4.3911995101015,4.38470273521238,4.38308430109869,4.36917550466905,4.22533133736166,4.18037950785597,4.17347761809801,4.15726790952507,4.10082021855417,4.02651961377615,3.90926021977455,3.74024576172954,3.70810015341318,3.67061939147858,3.63739183843795,3.62763793767271,3.56995004984007,3.56995004984007,3.56120263618196,3.52125152581892,3.5172325119401,3.5172325119401,3.50940005544699,3.47780923490343,3.46278869772552,3.45333921030809,3.41445225385258,3.35099949285783,3.32370729632721,3.28863781270669,3.28863781270669,3.2878029223401,3.28761980962398,3.28030528139846,3.2453002908563,3.20918623709128,3.1206859588658,3.07809696181388,3.0490791027462,3.02007106241249,2.96431663721296,2.95843704148656,2.93845737097146,2.92219688183321,2.91576704129595,2.90614172566838,2.89522855655349,2.89522855655349,2.8743515643633,2.85003235030995,2.84376510900918,2.83724309534004,2.80720260876726,2.75002312616301,2.72243304947909,2.70895834294261,2.68382178474841,2.6762083019208,2.63336002210524,2.62904623844732,2.62319174787627,2.61312419237851,2.50847465494442,2.48601778910206,2.4453077020549,2.4004891774539,2.39501790977565,2.37926168820709,2.31242551538693,2.25884959298336,2.24277302954659,2.2162907063776,2.2139529358144,2.21336332795037,2.20515154709453,2.18290170058197,2.12493574978507,2.12390213722223,2.07921139266562,2.06742393907411,2.06687480159541,2.05490197213092,1.91466154813494,1.90912925415014,1.9041856928362,1.88759479369194,1.88456503392397,1.86950817781541,1.86147882433011,1.8490527619734,1.82823562790173,1.81225705432978,1.76265361046163,1.72105794391378,1.69907459810034,1.69905089385307,1.67605694235591,1.66633608507288,1.64028730923292,1.63991315873598,1.63547526939697,1.59847354877801,1.54386520962792,1.54225893746463,1.54225893746463,1.49789612141816,1.49563817777049,1.49459852093301,1.46530820015817,1.4617423330263,1.45793973332444,1.44416863177357,1.44053707754304,1.43458803158394,1.42867366874244,1.4140124157564,1.32215539811858,1.30493133173063],"text":["GO:0005829 (5420) <br> cytosol <br> 1.142e-171","GO:0005737 (12241) <br> cytoplasm <br> 4.985e-128","GO:0070161 (848) <br> anchoring junction <br> 5.192e-63","GO:0070062 (2110) <br> extracellular exosome <br> 4.416e-59","GO:0043230 (2134) <br> extracellular organelle <br> 5.077e-59","GO:0065010 (2134) <br> extracellular membrane-bounded organelle <br> 5.077e-59","GO:1903561 (2133) <br> extracellular vesicle <br> 1.618e-58","GO:0005925 (423) <br> focal adhesion <br> 1.340e-49","GO:0030055 (432) <br> cell-substrate junction <br> 1.390e-49","GO:0031982 (3957) <br> vesicle <br> 6.390e-48","GO:0005856 (2355) <br> cytoskeleton <br> 1.410e-46","GO:0005622 (17809) <br> intracellular anatomical structure <br> 1.071e-43","GO:0030054 (2086) <br> cell junction <br> 5.550e-43","GO:0005654 (4215) <br> nucleoplasm <br> 5.529e-39","GO:0015629 (502) <br> actin cytoskeleton <br> 4.093e-36","GO:0031252 (421) <br> cell leading edge <br> 1.474e-35","GO:0005615 (3364) <br> extracellular space <br> 2.469e-33","GO:0099080 (1349) <br> supramolecular complex <br> 7.075e-30","GO:0005911 (503) <br> cell-cell junction <br> 1.050e-29","GO:0043232 (6424) <br> intracellular non-membrane-bounded organelle <br> 8.260e-28","GO:0043228 (6425) <br> non-membrane-bounded organelle <br> 8.629e-28","GO:0016020 (9845) <br> membrane <br> 6.637e-26","GO:0005576 (4315) <br> extracellular region <br> 1.433e-25","GO:0042995 (2319) <br> cell projection <br> 2.229e-25","GO:0043226 (16739) <br> organelle <br> 2.787e-25","GO:0120025 (2212) <br> plasma membrane bounded cell projection <br> 8.320e-25","GO:0005938 (319) <br> cell cortex <br> 8.524e-25","GO:0005912 (174) <br> adherens junction <br> 2.409e-24","GO:0030027 (203) <br> lamellipodium <br> 1.166e-22","GO:0031981 (5500) <br> nuclear lumen <br> 2.264e-22","GO:0015630 (1339) <br> microtubule cytoskeleton <br> 2.881e-22","GO:0070013 (6625) <br> intracellular organelle lumen <br> 5.264e-22","GO:0031974 (6626) <br> membrane-enclosed lumen <br> 5.469e-22","GO:0043233 (6626) <br> organelle lumen <br> 5.469e-22","GO:0043229 (15936) <br> intracellular organelle <br> 1.009e-21","GO:0031410 (2463) <br> cytoplasmic vesicle <br> 3.296e-20","GO:0097708 (2465) <br> intracellular vesicle <br> 3.679e-20","GO:0043227 (15617) <br> membrane-bounded organelle <br> 2.601e-19","GO:0030863 (110) <br> cortical cytoskeleton <br> 3.660e-19","GO:0001726 (181) <br> ruffle <br> 5.471e-19","GO:0099512 (1001) <br> supramolecular fiber <br> 4.765e-18","GO:0005819 (418) <br> spindle <br> 7.517e-18","GO:0099081 (1010) <br> supramolecular polymer <br> 9.339e-18","GO:0030141 (866) <br> secretory granule <br> 5.790e-17","GO:0012505 (4707) <br> endomembrane system <br> 1.249e-16","GO:1904813 (124) <br> ficolin-1-rich granule lumen <br> 7.032e-16","GO:0101002 (184) <br> ficolin-1-rich granule <br> 4.672e-15","GO:0099503 (1031) <br> secretory vesicle <br> 1.204e-14","GO:0036464 (247) <br> cytoplasmic ribonucleoprotein granule <br> 1.435e-14","GO:0043231 (14570) <br> intracellular membrane-bounded organelle <br> 1.906e-14","GO:0099513 (769) <br> polymeric cytoskeletal fiber <br> 2.676e-14","GO:0031983 (325) <br> vesicle lumen <br> 3.941e-14","GO:0030496 (205) <br> midbody <br> 9.594e-14","GO:0098590 (1237) <br> plasma membrane region <br> 9.832e-14","GO:0035770 (264) <br> ribonucleoprotein granule <br> 1.321e-13","GO:0005815 (826) <br> microtubule organizing center <br> 1.870e-13","GO:0034774 (320) <br> secretory granule lumen <br> 1.916e-13","GO:0060205 (323) <br> cytoplasmic vesicle lumen <br> 2.490e-13","GO:0030864 (80) <br> cortical actin cytoskeleton <br> 2.979e-13","GO:0032432 (79) <br> actin filament bundle <br> 3.751e-13","GO:0042641 (79) <br> actomyosin <br> 3.751e-13","GO:0012506 (1195) <br> vesicle membrane <br> 5.170e-13","GO:0048471 (718) <br> perinuclear region of cytoplasm <br> 1.420e-12","GO:0043005 (1347) <br> neuron projection <br> 3.228e-12","GO:0030659 (1176) <br> cytoplasmic vesicle membrane <br> 9.400e-12","GO:0032991 (7399) <br> protein-containing complex <br> 1.398e-11","GO:0001725 (71) <br> stress fiber <br> 1.536e-11","GO:0097517 (71) <br> contractile actin filament bundle <br> 1.536e-11","GO:0010494 (82) <br> cytoplasmic stress granule <br> 2.164e-11","GO:0005874 (444) <br> microtubule <br> 3.138e-11","GO:0016328 (70) <br> lateral plasma membrane <br> 6.059e-11","GO:0045121 (327) <br> membrane raft <br> 7.593e-11","GO:0098857 (328) <br> membrane microdomain <br> 8.312e-11","GO:0030424 (637) <br> axon <br> 9.057e-11","GO:0045178 (268) <br> basal part of cell <br> 9.992e-11","GO:1902494 (1505) <br> catalytic complex <br> 1.033e-10","GO:0071944 (6237) <br> cell periphery <br> 1.414e-09","GO:0072686 (174) <br> mitotic spindle <br> 2.104e-09","GO:0030175 (104) <br> filopodium <br> 2.418e-09","GO:0009925 (250) <br> basal plasma membrane <br> 3.184e-09","GO:0030016 (232) <br> myofibril <br> 3.846e-09","GO:0031253 (341) <br> cell projection membrane <br> 4.023e-09","GO:0005813 (628) <br> centrosome <br> 4.452e-09","GO:0098858 (215) <br> actin-based cell projection <br> 5.434e-09","GO:0005886 (5757) <br> plasma membrane <br> 6.030e-09","GO:0005634 (10084) <br> nucleus <br> 7.353e-09","GO:0035371 (25) <br> microtubule plus-end <br> 1.035e-08","GO:1990752 (35) <br> microtubule end <br> 1.049e-08","GO:0032587 (97) <br> ruffle membrane <br> 1.283e-08","GO:0043292 (242) <br> contractile fiber <br> 1.298e-08","GO:0044291 (72) <br> cell-cell contact zone <br> 2.363e-08","GO:0005884 (114) <br> actin filament <br> 4.371e-08","GO:0031256 (175) <br> leading edge membrane <br> 6.256e-08","GO:0031090 (3668) <br> organelle membrane <br> 8.664e-08","GO:0031674 (141) <br> I band <br> 1.082e-07","GO:0016323 (225) <br> basolateral plasma membrane <br> 1.500e-07","GO:0045202 (1305) <br> synapse <br> 2.676e-07","GO:0005903 (105) <br> brush border <br> 3.304e-07","GO:0030018 (128) <br> Z disc <br> 4.234e-07","GO:0030017 (211) <br> sarcomere <br> 4.972e-07","GO:0000922 (171) <br> spindle pole <br> 7.079e-07","GO:0030425 (610) <br> dendrite <br> 1.062e-06","GO:0098588 (2125) <br> bounding membrane of organelle <br> 1.208e-06","GO:0097447 (612) <br> dendritic tree <br> 1.214e-06","GO:0030426 (165) <br> growth cone <br> 1.335e-06","GO:0042470 (109) <br> melanosome <br> 1.546e-06","GO:0048770 (109) <br> pigment granule <br> 1.546e-06","GO:0032153 (72) <br> cell division site <br> 1.632e-06","GO:0140535 (783) <br> intracellular protein-containing complex <br> 1.659e-06","GO:0098687 (374) <br> chromosomal region <br> 1.870e-06","GO:0010008 (527) <br> endosome membrane <br> 1.930e-06","GO:0030427 (171) <br> site of polarized growth <br> 2.686e-06","GO:0098862 (161) <br> cluster of actin-based cell projections <br> 2.843e-06","GO:0030139 (343) <br> endocytic vesicle <br> 4.247e-06","GO:0005768 (999) <br> endosome <br> 4.477e-06","GO:0014704 (50) <br> intercalated disc <br> 4.670e-06","GO:0032154 (54) <br> cleavage furrow <br> 5.664e-06","GO:0030135 (294) <br> coated vesicle <br> 6.720e-06","GO:0030117 (96) <br> membrane coat <br> 7.164e-06","GO:0048475 (96) <br> coated membrane <br> 7.164e-06","GO:0005844 (71) <br> polysome <br> 9.617e-06","GO:0070820 (163) <br> tertiary granule <br> 1.014e-05","GO:0070971 (31) <br> endoplasmic reticulum exit site <br> 1.452e-05","GO:0098978 (331) <br> glutamatergic synapse <br> 2.416e-05","GO:0005635 (484) <br> nuclear envelope <br> 3.342e-05","GO:0008290 (8) <br> F-actin capping protein complex <br> 3.428e-05","GO:0030981 (5) <br> cortical microtubule cytoskeleton <br> 3.546e-05","GO:0005643 (96) <br> nuclear pore <br> 4.063e-05","GO:0005794 (1630) <br> Golgi apparatus <br> 4.124e-05","GO:0005876 (82) <br> spindle microtubule <br> 4.139e-05","GO:0005769 (390) <br> early endosome <br> 4.274e-05","GO:0005901 (82) <br> caveola <br> 5.952e-05","GO:0036477 (835) <br> somatodendritic compartment <br> 6.601e-05","GO:0098794 (609) <br> postsynapse <br> 6.707e-05","GO:0043296 (151) <br> apical junction complex <br> 6.962e-05","GO:0098797 (705) <br> plasma membrane protein complex <br> 7.928e-05","GO:0009898 (170) <br> cytoplasmic side of plasma membrane <br> 9.408e-05","GO:0150034 (276) <br> distal axon <br> 1.232e-04","GO:0000932 (96) <br> P-body <br> 1.819e-04","GO:0016600 (9) <br> flotillin complex <br> 1.958e-04","GO:0005739 (1683) <br> mitochondrion <br> 2.135e-04","GO:0030120 (56) <br> vesicle coat <br> 2.305e-04","GO:0030667 (309) <br> secretory granule membrane <br> 2.357e-04","GO:0031975 (1260) <br> envelope <br> 2.692e-04","GO:0031967 (1260) <br> organelle envelope <br> 2.692e-04","GO:0110016 (8) <br> B-WICH complex <br> 2.747e-04","GO:0030662 (178) <br> coated vesicle membrane <br> 3.011e-04","GO:0000323 (742) <br> lytic vacuole <br> 3.039e-04","GO:0005764 (742) <br> lysosome <br> 3.039e-04","GO:0090543 (33) <br> Flemming body <br> 3.095e-04","GO:0098562 (195) <br> cytoplasmic side of membrane <br> 3.328e-04","GO:0043034 (16) <br> costamere <br> 3.445e-04","GO:0044853 (113) <br> plasma membrane raft <br> 3.521e-04","GO:0045177 (431) <br> apical part of cell <br> 3.851e-04","GO:0035748 (6) <br> myelin sheath abaxonal region <br> 4.457e-04","GO:0005811 (101) <br> lipid droplet <br> 4.746e-04","GO:0001931 (13) <br> uropod <br> 5.145e-04","GO:0031254 (13) <br> cell trailing edge <br> 5.145e-04","GO:0031258 (22) <br> lamellipodium membrane <br> 5.155e-04","GO:0045335 (140) <br> phagocytic vesicle <br> 5.157e-04","GO:1904724 (55) <br> tertiary granule lumen <br> 5.244e-04","GO:0012507 (57) <br> ER to Golgi transport vesicle membrane <br> 5.685e-04","GO:0005881 (77) <br> cytoplasmic microtubule <br> 6.178e-04","GO:0005773 (838) <br> vacuole <br> 7.574e-04","GO:0098796 (1348) <br> membrane protein complex <br> 8.354e-04","GO:0051233 (36) <br> spindle midzone <br> 8.931e-04","GO:0002102 (30) <br> podosome <br> 9.548e-04","GO:0031616 (15) <br> spindle pole centrosome <br> 1.086e-03","GO:0034399 (130) <br> nuclear periphery <br> 1.100e-03","GO:0014069 (320) <br> postsynaptic density <br> 1.152e-03","GO:0000779 (159) <br> condensed chromosome, centromeric region <br> 1.196e-03","GO:0000776 (149) <br> kinetochore <br> 1.214e-03","GO:0005681 (193) <br> spliceosomal complex <br> 1.241e-03","GO:0042582 (154) <br> azurophil granule <br> 1.273e-03","GO:0005766 (154) <br> primary lysosome <br> 1.273e-03","GO:0070160 (134) <br> tight junction <br> 1.336e-03","GO:0098552 (661) <br> side of membrane <br> 1.412e-03","GO:0016604 (879) <br> nuclear body <br> 1.433e-03","GO:0055038 (93) <br> recycling endosome membrane <br> 1.455e-03","GO:0000793 (259) <br> condensed chromosome <br> 1.559e-03","GO:0032279 (326) <br> asymmetric synapse <br> 1.778e-03","GO:0005851 (6) <br> eukaryotic translation initiation factor 2B complex <br> 1.895e-03","GO:0019897 (173) <br> extrinsic component of plasma membrane <br> 1.955e-03","GO:0033588 (6) <br> elongator holoenzyme complex <br> 2.071e-03","GO:0016327 (21) <br> apicolateral plasma membrane <br> 2.108e-03","GO:0042612 (8) <br> MHC class I protein complex <br> 2.326e-03","GO:0099572 (343) <br> postsynaptic specialization <br> 2.349e-03","GO:0090443 (4) <br> FAR/SIN/STRIPAK complex <br> 2.381e-03","GO:0005885 (11) <br> Arp2/3 protein complex <br> 2.437e-03","GO:0042383 (133) <br> sarcolemma <br> 3.101e-03","GO:0030057 (25) <br> desmosome <br> 3.266e-03","GO:0005875 (162) <br> microtubule associated complex <br> 3.587e-03","GO:0106002 (5) <br> mCRD-mediated mRNA stability complex <br> 3.977e-03","GO:0035578 (91) <br> azurophil granule lumen <br> 4.027e-03","GO:0044297 (558) <br> cell body <br> 4.176e-03","GO:0016234 (76) <br> inclusion body <br> 4.871e-03","GO:0005905 (72) <br> clathrin-coated pit <br> 5.510e-03","GO:0043197 (172) <br> dendritic spine <br> 5.718e-03","GO:0031901 (175) <br> early endosome membrane <br> 6.077e-03","GO:0000781 (166) <br> chromosome, telomeric region <br> 6.110e-03","GO:0005923 (126) <br> bicellular tight junction <br> 6.118e-03","GO:0031143 (18) <br> pseudopodium <br> 6.235e-03","GO:0044309 (175) <br> neuron spine <br> 6.563e-03","GO:0000775 (235) <br> chromosome, centromeric region <br> 7.500e-03","GO:0072357 (7) <br> PTW/PP1 phosphatase complex <br> 7.518e-03","GO:0018444 (3) <br> translation release factor complex <br> 8.333e-03","GO:0055037 (193) <br> recycling endosome <br> 8.562e-03","GO:0005871 (50) <br> kinesin complex <br> 8.573e-03","GO:0098984 (350) <br> neuron to neuron synapse <br> 8.812e-03","GO:0030134 (90) <br> COPII-coated ER to Golgi transport vesicle <br> 1.217e-02","GO:0005775 (175) <br> vacuolar lumen <br> 1.233e-02","GO:0008305 (31) <br> integrin complex <br> 1.247e-02","GO:0030127 (15) <br> COPII vesicle coat <br> 1.295e-02","GO:0071013 (85) <br> catalytic step 2 spliceosome <br> 1.304e-02","GO:1904949 (96) <br> ATPase complex <br> 1.350e-02","GO:0008180 (34) <br> COP9 signalosome <br> 1.376e-02","GO:0000164 (15) <br> protein phosphatase type 1 complex <br> 1.416e-02","GO:0005916 (10) <br> fascia adherens <br> 1.485e-02","GO:0001772 (44) <br> immunological synapse <br> 1.541e-02","GO:0016459 (57) <br> myosin complex <br> 1.727e-02","GO:0099738 (39) <br> cell cortex region <br> 1.901e-02","GO:0044611 (3) <br> nuclear pore inner ring <br> 2.000e-02","GO:0016342 (32) <br> catenin complex <br> 2.000e-02","GO:0030904 (21) <br> retromer complex <br> 2.108e-02","GO:0098636 (45) <br> protein complex involved in cell adhesion <br> 2.156e-02","GO:0016363 (109) <br> nuclear matrix <br> 2.289e-02","GO:0055028 (3) <br> cortical microtubule <br> 2.291e-02","GO:1990234 (838) <br> transferase complex <br> 2.315e-02","GO:0031965 (306) <br> nuclear membrane <br> 2.521e-02","GO:0030136 (197) <br> clathrin-coated vesicle <br> 2.858e-02","GO:0005765 (413) <br> lysosomal membrane <br> 2.869e-02","GO:0098852 (413) <br> lytic vacuole membrane <br> 2.869e-02","GO:0030688 (15) <br> preribosome, small subunit precursor <br> 3.178e-02","GO:0030684 (79) <br> preribosome <br> 3.194e-02","GO:0097149 (3) <br> centralspindlin complex <br> 3.202e-02","GO:0008091 (9) <br> spectrin <br> 3.425e-02","GO:0016324 (365) <br> apical plasma membrane <br> 3.453e-02","GO:0005850 (5) <br> eukaryotic translation initiation factor 2 complex <br> 3.484e-02","GO:0070938 (10) <br> contractile ring <br> 3.596e-02","GO:0042587 (7) <br> glycogen granule <br> 3.626e-02","GO:0000145 (19) <br> exocyst <br> 3.676e-02","GO:0005902 (90) <br> microvillus <br> 3.727e-02","GO:0019898 (318) <br> extrinsic component of membrane <br> 3.855e-02","GO:0005774 (460) <br> vacuolar membrane <br> 4.763e-02","GO:0034451 (109) <br> centriolar satellite <br> 4.955e-02"],"key":["GO:0005829","GO:0005737","GO:0070161","GO:0070062","GO:0043230","GO:0065010","GO:1903561","GO:0005925","GO:0030055","GO:0031982","GO:0005856","GO:0005622","GO:0030054","GO:0005654","GO:0015629","GO:0031252","GO:0005615","GO:0099080","GO:0005911","GO:0043232","GO:0043228","GO:0016020","GO:0005576","GO:0042995","GO:0043226","GO:0120025","GO:0005938","GO:0005912","GO:0030027","GO:0031981","GO:0015630","GO:0070013","GO:0031974","GO:0043233","GO:0043229","GO:0031410","GO:0097708","GO:0043227","GO:0030863","GO:0001726","GO:0099512","GO:0005819","GO:0099081","GO:0030141","GO:0012505","GO:1904813","GO:0101002","GO:0099503","GO:0036464","GO:0043231","GO:0099513","GO:0031983","GO:0030496","GO:0098590","GO:0035770","GO:0005815","GO:0034774","GO:0060205","GO:0030864","GO:0032432","GO:0042641","GO:0012506","GO:0048471","GO:0043005","GO:0030659","GO:0032991","GO:0001725","GO:0097517","GO:0010494","GO:0005874","GO:0016328","GO:0045121","GO:0098857","GO:0030424","GO:0045178","GO:1902494","GO:0071944","GO:0072686","GO:0030175","GO:0009925","GO:0030016","GO:0031253","GO:0005813","GO:0098858","GO:0005886","GO:0005634","GO:0035371","GO:1990752","GO:0032587","GO:0043292","GO:0044291","GO:0005884","GO:0031256","GO:0031090","GO:0031674","GO:0016323","GO:0045202","GO:0005903","GO:0030018","GO:0030017","GO:0000922","GO:0030425","GO:0098588","GO:0097447","GO:0030426","GO:0042470","GO:0048770","GO:0032153","GO:0140535","GO:0098687","GO:0010008","GO:0030427","GO:0098862","GO:0030139","GO:0005768","GO:0014704","GO:0032154","GO:0030135","GO:0030117","GO:0048475","GO:0005844","GO:0070820","GO:0070971","GO:0098978","GO:0005635","GO:0008290","GO:0030981","GO:0005643","GO:0005794","GO:0005876","GO:0005769","GO:0005901","GO:0036477","GO:0098794","GO:0043296","GO:0098797","GO:0009898","GO:0150034","GO:0000932","GO:0016600","GO:0005739","GO:0030120","GO:0030667","GO:0031975","GO:0031967","GO:0110016","GO:0030662","GO:0000323","GO:0005764","GO:0090543","GO:0098562","GO:0043034","GO:0044853","GO:0045177","GO:0035748","GO:0005811","GO:0001931","GO:0031254","GO:0031258","GO:0045335","GO:1904724","GO:0012507","GO:0005881","GO:0005773","GO:0098796","GO:0051233","GO:0002102","GO:0031616","GO:0034399","GO:0014069","GO:0000779","GO:0000776","GO:0005681","GO:0042582","GO:0005766","GO:0070160","GO:0098552","GO:0016604","GO:0055038","GO:0000793","GO:0032279","GO:0005851","GO:0019897","GO:0033588","GO:0016327","GO:0042612","GO:0099572","GO:0090443","GO:0005885","GO:0042383","GO:0030057","GO:0005875","GO:0106002","GO:0035578","GO:0044297","GO:0016234","GO:0005905","GO:0043197","GO:0031901","GO:0000781","GO:0005923","GO:0031143","GO:0044309","GO:0000775","GO:0072357","GO:0018444","GO:0055037","GO:0005871","GO:0098984","GO:0030134","GO:0005775","GO:0008305","GO:0030127","GO:0071013","GO:1904949","GO:0008180","GO:0000164","GO:0005916","GO:0001772","GO:0016459","GO:0099738","GO:0044611","GO:0016342","GO:0030904","GO:0098636","GO:0016363","GO:0055028","GO:1990234","GO:0031965","GO:0030136","GO:0005765","GO:0098852","GO:0030688","GO:0030684","GO:0097149","GO:0008091","GO:0016324","GO:0005850","GO:0070938","GO:0042587","GO:0000145","GO:0005902","GO:0019898","GO:0005774","GO:0034451"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.2503512083362,22.1748783283837,18.9363657722217,20.1145095846538,20.1285941750016,20.1285941750016,20.1280107275765,17.9718339896155,18.001990970823,20.8799550072757,20.2507995824627,22.585069851013,20.1002515064211,20.9549594388116,18.2152742453374,17.9650364933867,20.6856365889365,19.5473714214816,18.2180794348776,21.4472011073401,21.4473803984182,21.9322973416308,20.9827195977937,20.231755679778,22.517896061399,20.1732189286943,17.5612687740268,16.6351023083939,16.8768537716112,21.2674104348233,19.5377654274122,21.4826536894232,21.4828271931595,21.4828271931595,22.4644291323247,20.3061070310994,20.3071065080618,22.4423927774989,15.8874237642864,16.6973887091722,19.157468189673,17.954773290037,19.1693088948601,18.964541499708,21.0852764547599,16.0871361666277,16.7232602595652,19.1964973973329,17.178216104206,22.3665672625306,18.8044923244643,17.5887572421632,16.8920784419944,19.435105650505,17.2789339363093,18.9010312098947,17.5658897337121,17.5796572794936,15.3397884126306,15.3176234456008,15.3176234456008,19.3901386912533,18.7112581239676,19.5454563992209,19.3692307174758,21.6092206957528,15.1277454070193,15.1277454070193,15.3831759264344,18.0411418678934,15.1022795884124,17.5977954387839,17.6022916256912,18.5472159712769,17.3015849292252,19.6879790543733,21.4131403561583,16.6351023083939,15.7927798038515,17.1965401157473,17.0827178306593,17.6593987636157,18.5275938935892,16.965790105779,21.3204744211359,21.959168016757,13.0691852338268,13.7791594529932,15.6741548202393,17.1471184345574,15.152799219118,15.9473099062644,16.6441709175865,20.7894610999521,16.2977734456596,17.0357677197234,19.5045146528864,15.8089797392543,16.1395213611136,16.9367746702346,16.6075411810639,18.4874102772636,20.1233325242321,18.4919387498456,16.5507513357352,15.8720643257765,15.8720643257765,15.152799219118,18.828909277739,17.7941823919817,18.2836265978252,16.6075411810639,16.5115854735424,17.6679704286443,19.1548212202776,14.4804534637839,14.6258282642228,17.4402264269597,15.6564117262202,15.6564117262202,15.1277454070193,16.5313035845537,13.5290443998576,17.615689722033,18.16370502316,10.097845750688,8.33927853761798,15.6564117262202,19.7897205009428,15.3831759264344,17.854880874634,15.3831759264344,18.9156086894326,18.4851399454722,16.4086739788765,18.6863344205339,16.5982332186728,17.3457806327537,15.6564117262202,10.4664612166769,19.8303418083352,14.6938508911943,17.5141574314169,19.4590349653877,19.4590349653877,10.097845750688,16.671031515582,18.7560033506328,18.7560033506328,13.6586847106596,16.8142310637125,12.0338125235308,15.9325651257445,17.9986745106348,9.09103163078953,15.7430923844698,11.5049403625855,11.5049403625855,12.7847893425189,16.2861973405104,14.6602019413075,14.7268035556219,15.2723067404551,18.9204298627924,19.5464142945856,13.8363278826937,13.46035842381,11.8731306335789,16.1650226729754,17.5658897337121,16.49158971849,16.3871699094041,16.7981253888998,16.4403351484814,16.4403351484814,16.2147179483544,18.5980946257729,18.9844971122736,15.6018860534727,17.2500776210341,17.5932840108147,9.09103163078953,16.625974973076,9.09103163078953,12.679039288794,10.097845750688,17.6679704286443,7.20137928972192,11.0515743331527,16.2024531014276,13.0691852338268,16.5214787726,8.33927853761798,15.5644039696379,18.3636050964249,15.2491361405053,15.152799219118,16.616788179269,16.6441709175865,16.5603760262376,16.1135644599747,12.3192674406608,16.6441709175865,17.1023579289126,9.65202509301152,3.77952755905512,16.7981253888998,14.4804534637839,17.6975419377751,15.5453070901116,16.6441709175865,13.5290443998576,11.8731306335789,15.4460240033122,15.6564117262202,13.7199905291574,11.8731306335789,10.7797707327282,14.2345175300099,14.7268035556219,13.9970167186474,3.77952755905512,13.5950915152841,12.679039288794,14.2781699698326,15.8720643257765,3.77952755905512,18.9204298627924,17.4996940136884,16.8301528193582,17.9374863511375,17.9374863511375,11.8731306335789,15.3176234456008,3.77952755905512,10.4664612166769,17.7587666942319,8.33927853761798,10.7797707327282,9.65202509301152,12.4471552623873,15.5453070901116,17.5566317485932,18.0915697419912,15.8720643257765],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData257a9fd5","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[29.9045134070955,37.3267555238949,9.31704498443527,9.62874019143997,19.9653324937332,2.132470462977,48.7075267696539,26.8615889487122,17.4055356062072,26.6784680145969,23.031634092642,23.0394264728171,3.82731315106504,28.3538297522472,29.7291848531554,42.1697198027304,20.9978228669363,9.3521106952233,23.0550112331673,37.3735098049456,3.99874551491762,16.727598530972,16.7237023408844,28.0187574047171,3.71042744843828,15.800305290133,9.25080975294678,17.495147978221,19.9770210639959,16.9886432668384,19.9809172540835,48.7192153399166,28.3499335621596,42.0411455298409,29.8577591260448,9.76121065441697,48.3919353725617,19.984813444171,41.6632150913478,16.5717509274696,20.5692419573049,19.9692286838208,16.6224013986079,21.671863752084,19.9731248739084,37.3851983752083,16.4587614149304,15.5821186452297,6.99101950216272,39.2865391379369,16.5756471175572,6.8702376094484,3.72990839887607,42.4424531088595,21.1926323713142,37.9852116486923,42.4697264394724,3.75718172948898,47.7880259089901,9.35600688531086,29.538271538865,6.87802998962352,42.4463492989471,47.1997012057687,19.2912916085856,23.0628036133424,8.60404219841203,10.2365458450991,9.41055354653668,32.5850921873359,42.5125845304356,9.50795829872565,39.321604848725,21.2004247514894,15.0600291734968,29.5109982082521,37.3657174247705,9.52354305907589,3.55847603502349,28.2798021405836,9.32094117452283,26.351188047242,21.8316075456739,3.11041417495423,47.8815344710915,4.33381786244767,17.6042413006727,8.22611175991883,19.2990839887607,8.53001458674841,2.27273330612911,9.57029734012659,10.1001791920346,29.8967210269204,26.6317137335462,23.6706092670016,16.2639519105525,9.78848398502988,3.7182198286134,3.72601220878851,42.0255607694907,40.2099361886884,2.1129895125392,37.7981945244895,40.6229323379696,21.5978361404204,3.50782556388522,28.575912587238,41.6047722400344,3.80783220062725,48.4270010833497,28.2914907108462,6.61308906366952,8.24948890044418,16.9418889857877,3.73380458896363,37.3774059950332,17.3509889449814,2.43637328980658,15.4262710417273,19.614675385853,9.41444973662424,6.99491569225028,48.0334858845063,15.1535377355982,28.2836983306711,9.89757730748152,47.7958182891652,25.026483417472,39.3255010388125,40.2060399986008,19.6029868155903,5.91567103799651,37.9501459379043,3.25846939828147,48.8361010425434,48.380246802299,4.33771405253523,34.6734500742674,27.9642107434913,30.5162152508423,37.5098764580102,29.935682927796,19.949747733383,34.6227996031291,3.71432363852584],"y":[17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,17,15.5764726764937,14.9667722840028,13.7206840404994,13.4233177654004,13.1825488113752,13.0426977374324,13.0168355266911,12.4849982862494,12.0559901005848,11.8621412330684,11.7993917072582,11.7535298986232,11.4492371907424,11.4008970023625,11.3020138313789,10.9403086764541,10.7228272259569,10.6576693837479,10.5415247439324,10.2683565659673,9.89102745434099,9.18912573238609,8.94767877266428,8.67235541406082,8.20356953662584,8.16349849774066,8.11664608731116,8.10576290318292,8.03476111733297,7.99379335050522,7.97420229239471,7.75037379758991,7.44315734860856,7.30813936340553,6.95822424381019,6.89026734210375,6.68065716854416,6.59370217974501,6.59370217974501,6.45352942596417,6.1194861814958,5.58960441395575,5.56527039596159,5.35396371449347,5.29904867497857,5.27504629494621,5.27504629494621,5.20561105903504,5.08875683377267,5.02410710098282,4.75313018593384,4.70449680669544,4.69087087228816,4.63134194802027,4.56569431276822,4.51426133848859,4.13184100081199,4.04381444017796,4.02654561186271,4.02332094635953,3.99649411451391,3.87710429082798,3.86277465042412,3.69990129065414,3.68616914401623,3.48004346823189,3.46064217521539,3.44124082020466,3.38368458096576,3.32049942332852,3.27119890821381,3.19290082244742,3.084672193024,2.90372326495754,2.83073780528346,2.78050537430177,2.7612362014333,2.74063380237495,2.71367778839552,2.70998474112299,2.48861556666923,2.48333910074506,2.44504533650312,2.40706784338818,2.39116267211674,2.32187138204968,2.29726981238147,2.28463816874852,2.24096459394289,2.23726378592968,2.22805358477373,2.17790199225209,2.12562120454277,2.09499573461073,2.06314878458301,2.00661636546004,1.98914260734364,1.98914260734364,1.87858439718649,1.8284399386164,1.79617927728908,1.77646962358773,1.75589927719664,1.73762177718732,1.68094012686099,1.67930475718593,1.65045925952217,1.63223108743456,1.61492239475626,1.58253134789075,1.50127913009163,1.47737091248736,1.45126010101015,1.42385247556901,1.40652060907956,1.35324275838396,1.3417376551542,1.33022840200606,1.30731862664678],"text":["GO:0045296 (333) <br> cadherin binding <br> 8.036e-80","GO:0050839 (560) <br> cell adhesion molecule binding <br> 5.487e-67","GO:0005515 (14833) <br> protein binding <br> 2.754e-66","GO:0008092 (999) <br> cytoskeletal protein binding <br> 1.499e-41","GO:0019899 (2048) <br> enzyme binding <br> 2.970e-37","GO:0000166 (2176) <br> nucleotide binding <br> 1.892e-36","GO:1901265 (2177) <br> nucleoside phosphate binding <br> 2.048e-36","GO:0036094 (2519) <br> small molecule binding <br> 1.524e-34","GO:0017076 (1932) <br> purine nucleotide binding <br> 3.873e-34","GO:0035639 (1850) <br> purine ribonucleoside triphosphate binding <br> 1.143e-33","GO:0032553 (1936) <br> ribonucleotide binding <br> 1.227e-32","GO:0032555 (1919) <br> purine ribonucleotide binding <br> 6.122e-32","GO:0003779 (448) <br> actin binding <br> 3.932e-31","GO:0043168 (2422) <br> anion binding <br> 2.804e-30","GO:0044877 (1283) <br> protein-containing complex binding <br> 5.427e-30","GO:0097367 (2292) <br> carbohydrate derivative binding <br> 1.469e-27","GO:0030554 (1577) <br> adenyl nucleotide binding <br> 6.411e-27","GO:0005524 (1503) <br> ATP binding <br> 9.798e-27","GO:0032559 (1566) <br> adenyl ribonucleotide binding <br> 3.446e-25","GO:0051015 (218) <br> actin filament binding <br> 1.527e-23","GO:0003824 (5746) <br> catalytic activity <br> 3.640e-22","GO:0016818 (757) <br> hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides <br> 1.158e-17","GO:0016817 (757) <br> hydrolase activity, acting on acid anhydrides <br> 1.158e-17","GO:0042802 (2105) <br> identical protein binding <br> 1.629e-17","GO:0003723 (3107) <br> RNA binding <br> 3.294e-17","GO:0016462 (754) <br> pyrophosphatase activity <br> 3.417e-17","GO:0005488 (18309) <br> binding <br> 4.950e-17","GO:0017111 (698) <br> nucleoside-triphosphatase activity <br> 2.652e-16","GO:0019902 (195) <br> phosphatase binding <br> 1.080e-15","GO:0016887 (356) <br> ATP hydrolysis activity <br> 1.902e-14","GO:0019903 (150) <br> protein phosphatase binding <br> 3.773e-14","GO:1901363 (7428) <br> heterocyclic compound binding <br> 6.568e-14","GO:0043167 (6066) <br> ion binding <br> 9.064e-14","GO:0097159 (7501) <br> organic cyclic compound binding <br> 9.620e-14","GO:0045182 (156) <br> translation regulator activity <br> 3.273e-13","GO:0008135 (98) <br> translation factor activity, RNA binding <br> 8.790e-13","GO:0140657 (546) <br> ATP-dependent activity <br> 1.374e-12","GO:0019904 (689) <br> protein domain specific binding <br> 1.587e-12","GO:0090079 (121) <br> translation regulator activity, nucleic acid binding <br> 1.764e-12","GO:0016772 (954) <br> transferase activity, transferring phosphorus-containing groups <br> 3.554e-12","GO:0030234 (1245) <br> enzyme regulator activity <br> 3.973e-12","GO:0019900 (771) <br> kinase binding <br> 4.989e-12","GO:0016787 (2457) <br> hydrolase activity <br> 1.147e-11","GO:0031267 (268) <br> small GTPase binding <br> 1.893e-11","GO:0019901 (690) <br> protein kinase binding <br> 2.200e-11","GO:0051020 (298) <br> GTPase binding <br> 2.874e-11","GO:0016740 (2336) <br> transferase activity <br> 5.391e-11","GO:0016301 (795) <br> kinase activity <br> 1.285e-10","GO:0004712 (446) <br> protein serine/threonine/tyrosine kinase activity <br> 6.470e-10","GO:0060090 (404) <br> molecular adaptor activity <br> 1.128e-09","GO:0016773 (695) <br> phosphotransferase activity, alcohol group as acceptor <br> 2.126e-09","GO:0004672 (586) <br> protein kinase activity <br> 6.258e-09","GO:0003729 (351) <br> mRNA binding <br> 6.863e-09","GO:0098631 (65) <br> cell adhesion mediator activity <br> 7.645e-09","GO:0030674 (308) <br> protein-macromolecule adaptor activity <br> 7.839e-09","GO:0051920 (8) <br> peroxiredoxin activity <br> 9.231e-09","GO:0098641 (18) <br> cadherin binding involved in cell-cell adhesion <br> 1.014e-08","GO:0003743 (58) <br> translation initiation factor activity <br> 1.061e-08","GO:0140096 (2322) <br> catalytic activity, acting on a protein <br> 1.777e-08","GO:0005525 (384) <br> GTP binding <br> 3.604e-08","GO:0044548 (14) <br> S100 protein binding <br> 4.919e-08","GO:0004674 (438) <br> protein serine/threonine kinase activity <br> 1.101e-07","GO:0098632 (54) <br> cell-cell adhesion mediator activity <br> 1.287e-07","GO:0106310 (360) <br> protein serine kinase activity <br> 2.086e-07","GO:0019001 (406) <br> guanyl nucleotide binding <br> 2.549e-07","GO:0032561 (406) <br> guanyl ribonucleotide binding <br> 2.549e-07","GO:0005200 (104) <br> structural constituent of cytoskeleton <br> 3.519e-07","GO:0008379 (7) <br> thioredoxin peroxidase activity <br> 7.595e-07","GO:0005543 (465) <br> phospholipid binding <br> 2.573e-06","GO:0047485 (107) <br> protein N-terminus binding <br> 2.721e-06","GO:0098772 (1970) <br> molecular function regulator activity <br> 4.426e-06","GO:0008017 (271) <br> microtubule binding <br> 5.023e-06","GO:0060589 (497) <br> nucleoside-triphosphatase regulator activity <br> 5.308e-06","GO:0030695 (497) <br> GTPase regulator activity <br> 5.308e-06","GO:0015631 (379) <br> tubulin binding <br> 6.229e-06","GO:0044389 (314) <br> ubiquitin-like protein ligase binding <br> 8.152e-06","GO:0051010 (21) <br> microtubule plus-end binding <br> 9.460e-06","GO:0008022 (190) <br> protein C-terminus binding <br> 1.766e-05","GO:0003676 (5449) <br> nucleic acid binding <br> 1.975e-05","GO:0043021 (154) <br> ribonucleoprotein complex binding <br> 2.038e-05","GO:0005516 (202) <br> calmodulin binding <br> 2.337e-05","GO:0035091 (270) <br> phosphatidylinositol binding <br> 2.718e-05","GO:0031625 (296) <br> ubiquitin protein ligase binding <br> 3.060e-05","GO:0001618 (76) <br> virus receptor activity <br> 7.382e-05","GO:0140272 (77) <br> exogenous protein binding <br> 9.040e-05","GO:0003924 (340) <br> GTPase activity <br> 9.407e-05","GO:0017166 (12) <br> vinculin binding <br> 9.477e-05","GO:0005085 (230) <br> guanyl-nucleotide exchange factor activity <br> 1.008e-04","GO:0019003 (71) <br> GDP binding <br> 1.327e-04","GO:0005178 (158) <br> integrin binding <br> 1.372e-04","GO:0000287 (224) <br> magnesium ion binding <br> 1.996e-04","GO:0008047 (561) <br> enzyme activator activity <br> 2.060e-04","GO:0008289 (793) <br> lipid binding <br> 3.311e-04","GO:0045294 (11) <br> alpha-catenin binding <br> 3.462e-04","GO:0035591 (80) <br> signaling adaptor activity <br> 3.620e-04","GO:0033592 (5) <br> RNA strand annealing activity <br> 4.133e-04","GO:0016684 (56) <br> oxidoreductase activity, acting on peroxide as acceptor <br> 4.781e-04","GO:0008143 (23) <br> poly(A) binding <br> 5.356e-04","GO:0003725 (75) <br> double-stranded RNA binding <br> 6.414e-04","GO:0003727 (84) <br> single-stranded RNA binding <br> 8.229e-04","GO:0097110 (67) <br> scaffold protein binding <br> 1.248e-03","GO:0070064 (18) <br> proline-rich region binding <br> 1.477e-03","GO:0000146 (31) <br> microfilament motor activity <br> 1.658e-03","GO:0051721 (31) <br> protein phosphatase 2A binding <br> 1.733e-03","GO:0070717 (29) <br> poly-purine tract binding <br> 1.817e-03","GO:0031072 (126) <br> heat shock protein binding <br> 1.933e-03","GO:0002162 (11) <br> dystroglycan binding <br> 1.950e-03","GO:0043531 (34) <br> ADP binding <br> 3.246e-03","GO:0086083 (5) <br> cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication <br> 3.286e-03","GO:0003774 (115) <br> cytoskeletal motor activity <br> 3.589e-03","GO:0140666 (13) <br> annealing activity <br> 3.917e-03","GO:0043024 (17) <br> ribosomal small subunit binding <br> 4.063e-03","GO:0004601 (52) <br> peroxidase activity <br> 4.766e-03","GO:0005096 (287) <br> GTPase activator activity <br> 5.043e-03","GO:0016874 (178) <br> ligase activity <br> 5.192e-03","GO:0003730 (97) <br> mRNA 3'-UTR binding <br> 5.742e-03","GO:0051018 (51) <br> protein kinase A binding <br> 5.791e-03","GO:0017056 (28) <br> structural constituent of nuclear pore <br> 5.915e-03","GO:0000774 (11) <br> adenyl-nucleotide exchange factor activity <br> 6.639e-03","GO:0016209 (84) <br> antioxidant activity <br> 7.488e-03","GO:0019208 (114) <br> phosphatase regulator activity <br> 8.035e-03","GO:0005544 (53) <br> calcium-dependent phospholipid binding <br> 8.647e-03","GO:0004713 (145) <br> protein tyrosine kinase activity <br> 9.849e-03","GO:0140358 (20) <br> P-type transmembrane transporter activity <br> 1.025e-02","GO:0015662 (20) <br> P-type ion transporter activity <br> 1.025e-02","GO:0043022 (74) <br> ribosome binding <br> 1.323e-02","GO:0008190 (10) <br> eukaryotic initiation factor 4E binding <br> 1.484e-02","GO:0140098 (401) <br> catalytic activity, acting on RNA <br> 1.599e-02","GO:0034237 (25) <br> protein kinase A regulatory subunit binding <br> 1.673e-02","GO:0060590 (13) <br> ATPase regulator activity <br> 1.754e-02","GO:0070063 (58) <br> RNA polymerase binding <br> 1.830e-02","GO:0019205 (45) <br> nucleobase-containing compound kinase activity <br> 2.085e-02","GO:0004386 (161) <br> helicase activity <br> 2.093e-02","GO:0051879 (45) <br> Hsp90 protein binding <br> 2.236e-02","GO:0001784 (44) <br> phosphotyrosine residue binding <br> 2.332e-02","GO:1901981 (174) <br> phosphatidylinositol phosphate binding <br> 2.427e-02","GO:0140640 (606) <br> catalytic activity, acting on a nucleic acid <br> 2.615e-02","GO:0003925 (41) <br> G protein activity <br> 3.153e-02","GO:0048306 (83) <br> calcium-dependent protein binding <br> 3.331e-02","GO:0042626 (101) <br> ATPase-coupled transmembrane transporter activity <br> 3.538e-02","GO:0046875 (29) <br> ephrin receptor binding <br> 3.768e-02","GO:0051219 (89) <br> phosphoprotein binding <br> 3.922e-02","GO:0045309 (55) <br> protein phosphorylated amino acid binding <br> 4.434e-02","GO:0019888 (95) <br> protein phosphatase regulator activity <br> 4.553e-02","GO:0048156 (43) <br> tau protein binding <br> 4.675e-02","GO:0003724 (76) <br> RNA helicase activity <br> 4.928e-02"],"key":["GO:0045296","GO:0050839","GO:0005515","GO:0008092","GO:0019899","GO:0000166","GO:1901265","GO:0036094","GO:0017076","GO:0035639","GO:0032553","GO:0032555","GO:0003779","GO:0043168","GO:0044877","GO:0097367","GO:0030554","GO:0005524","GO:0032559","GO:0051015","GO:0003824","GO:0016818","GO:0016817","GO:0042802","GO:0003723","GO:0016462","GO:0005488","GO:0017111","GO:0019902","GO:0016887","GO:0019903","GO:1901363","GO:0043167","GO:0097159","GO:0045182","GO:0008135","GO:0140657","GO:0019904","GO:0090079","GO:0016772","GO:0030234","GO:0019900","GO:0016787","GO:0031267","GO:0019901","GO:0051020","GO:0016740","GO:0016301","GO:0004712","GO:0060090","GO:0016773","GO:0004672","GO:0003729","GO:0098631","GO:0030674","GO:0051920","GO:0098641","GO:0003743","GO:0140096","GO:0005525","GO:0044548","GO:0004674","GO:0098632","GO:0106310","GO:0019001","GO:0032561","GO:0005200","GO:0008379","GO:0005543","GO:0047485","GO:0098772","GO:0008017","GO:0060589","GO:0030695","GO:0015631","GO:0044389","GO:0051010","GO:0008022","GO:0003676","GO:0043021","GO:0005516","GO:0035091","GO:0031625","GO:0001618","GO:0140272","GO:0003924","GO:0017166","GO:0005085","GO:0019003","GO:0005178","GO:0000287","GO:0008047","GO:0008289","GO:0045294","GO:0035591","GO:0033592","GO:0016684","GO:0008143","GO:0003725","GO:0003727","GO:0097110","GO:0070064","GO:0000146","GO:0051721","GO:0070717","GO:0031072","GO:0002162","GO:0043531","GO:0086083","GO:0003774","GO:0140666","GO:0043024","GO:0004601","GO:0005096","GO:0016874","GO:0003730","GO:0051018","GO:0017056","GO:0000774","GO:0016209","GO:0019208","GO:0005544","GO:0004713","GO:0140358","GO:0015662","GO:0043022","GO:0008190","GO:0140098","GO:0034237","GO:0060590","GO:0070063","GO:0019205","GO:0004386","GO:0051879","GO:0001784","GO:1901981","GO:0140640","GO:0003925","GO:0048306","GO:0042626","GO:0046875","GO:0051219","GO:0045309","GO:0019888","GO:0048156","GO:0003724"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[17.624547365014,18.3685967559048,22.3861442735098,19.1548212202776,20.0773111139784,20.1528368483919,20.1534079019973,20.3337679054467,20.0043397763782,19.9498493525277,20.0069337697521,19.9958691648831,18.0539340374567,20.2854233448952,19.4824959204812,20.2172629636424,19.7476477922584,19.6862778475427,19.7387244567285,16.9871587228391,21.318255378163,18.7831746424171,18.7831746424171,20.1115535954641,20.5896987039054,18.7777876288362,22.6150098782813,18.6727053102756,16.8142310637125,17.7223733864033,16.397962365319,21.6136850289855,21.3810276459757,21.6248412412041,16.4610588441362,15.6916899076018,18.3332373698971,18.6549612445074,16.0465732633297,19.0936936019979,19.4434831420617,18.8080099920536,20.3031033596201,17.3015849292252,18.6569452679025,17.4603433658104,20.2407879479432,18.8494623655922,18.0475537273348,17.9057809314112,18.6668185160786,18.4317660527774,17.701713027194,14.9682903911511,17.5093536439159,10.097845750688,12.3192674406608,14.7590867667101,20.2333547719424,17.8324464205714,11.6977535466344,18.0217139542686,14.6258282642228,17.7386721955122,17.9128933797703,17.9128933797703,15.7927798038515,9.65202509301152,18.1069338525357,15.8408580779897,20.0287523380054,17.3183279951438,18.2011556693834,18.2011556693834,17.8134536510378,17.5379208193078,12.679039288794,16.7736129918804,21.2565660260977,16.4403351484814,16.8691783884681,17.3127699537978,17.4503225782439,15.2491361405053,15.2723067404551,17.6550920625975,11.2911286539018,17.0694664581656,15.1277454070193,16.4814853684313,17.0289277268582,18.3710852662863,18.8460604232204,11.0515743331527,15.3397884126306,8.33927853761798,14.6938508911943,12.8846903643046,15.2256107955112,15.4253620090145,15.0232762450614,12.3192674406608,13.5290443998576,13.5290443998576,13.3888274758207,16.1135644599747,11.0515743331527,13.7199905291574,8.33927853761798,15.9619082967451,11.5049403625855,12.1819515281204,14.5547841218036,17.4042803625922,16.671031515582,15.6741548202393,14.5180474895371,13.3142209623286,11.0515743331527,15.4253620090145,15.9473099062644,14.5906996788533,16.3431677806744,12.5667614807513,12.5667614807513,15.2017202315928,10.7797707327282,17.8950391793916,13.0691852338268,11.5049403625855,14.7590867667101,14.2781699698326,16.5115854735424,14.2781699698326,14.2345175300099,16.6351023083939,18.4783043985539,14.0961442491203,15.4044151730077,15.7430923844698,13.3888274758207,15.5259652640403,14.6602019413075,15.6384558719977,14.1896720932675,15.2491361405053],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData257a9fd5","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[199.207695202663,199.430163037278,199.016450923784,198.879847867441,199.004742090383,199.562863149153,199.020353868251,199.121830424391],"y":[5.5715554327063,4.10214564356114,3.41300189710085,2.98527575490304,2.63861370232743,1.81930589068754,1.67958010326293,1.34507721375344],"text":["KEGG:04810 (207) <br> Regulation of actin cytoskeleton <br> 2.682e-06","KEGG:05100 (72) <br> Bacterial invasion of epithelial cells <br> 7.904e-05","KEGG:04520 (69) <br> Adherens junction <br> 3.864e-04","KEGG:04144 (237) <br> Endocytosis <br> 1.034e-03","KEGG:04510 (192) <br> Focal adhesion <br> 2.298e-03","KEGG:05205 (200) <br> Proteoglycans in cancer <br> 1.516e-02","KEGG:04530 (159) <br> Tight junction <br> 2.091e-02","KEGG:04670 (110) <br> Leukocyte transendothelial migration <br> 4.518e-02"],"key":["KEGG:04810","KEGG:05100","KEGG:04520","KEGG:04144","KEGG:04510","KEGG:05205","KEGG:04530","KEGG:04670"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":[16.9071378973067,15.152799219118,15.0763886917563,17.1152966943003,16.7900023015197,16.8536992814473,16.49158971849,15.8874237642864],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedData257a9fd5","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,50.0088542588985],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.9047281742518,71.3153887019656],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.2112626173189,193.957499557287],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.85337347264,200.003895873915],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[26.0044271294493,62.6100584381087,134.584381087303,198.928634673278],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"38b0ee27075":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"38b07c082a3a":{"x":{},"xend":{},"y":{},"yend":{}},"38b078e2440c":{"x":{},"xend":{},"y":{},"yend":{}},"38b0339d242b":{"x":{},"xend":{},"y":{},"yend":{}},"38b0343a4555":{"x":{},"xend":{},"y":{},"yend":{}},"38b0ef1ecf":{"x":{},"y":{}},"38b04653ddc":{"yintercept":{}}},"cur_data":"38b0ee27075","visdat":{"38b0ee27075":["function (y) ","x"],"38b07c082a3a":["function (y) ","x"],"38b078e2440c":["function (y) ","x"],"38b0339d242b":["function (y) ","x"],"38b0343a4555":["function (y) ","x"],"38b0ef1ecf":["function (y) ","x"],"38b04653ddc":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData257a9fd5"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;%
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-fd235ecc1e02da0ed1ec" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-fd235ecc1e02da0ed1ec">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52","53","54","55","56","57","58","59","60","61","62","63","64","65","66","67","68","69","70","71","72","73","74","75","76","77","78","79","80","81","82","83","84","85","86","87","88","89","90","91","92","93","94","95","96","97","98","99","100","101","102","103","104","105","106","107","108","109","110","111","112","113","114","115","116","117","118","119","120","121","122","123","124","125","126","127","128","129","130","131","132","133","134","135","136","137","138","139","140","141","142","143","144","145","146","147","148","149","150","151","152","153","154","155","156","157","158","159","160","161","162","163","164","165","166","167","168","169","170","171","172","173","174","175","176","177","178","179","180","181","182","183","184","185","186","187","188","189","190","191","192","193","194","195","196","197","198","199","200","201","202","203","204","205","206","207","208","209","210","211","212","213","214","215","216","217","218","219","220","221","222","223","224","225","226","227","228","229","230","231","232","233","234","235","236","237","238","239","240","241","242","243","244","245","246","247","248","249","250","251","252","253","254","255","256","257","258","259","260","261","262","263","264","265","266","267","268","269","270","271","272","273","274","275","276","277","278","279","280","281","282","283","284","285","286","287","288","289","290","291","292","293","294","295","296","297","298","299","300","301","302","303","304","305","306","307","308","309","310","311","312","313","314","315","316","317","318","319","320","321","322","323","324","325","326","327","328","329","330","331","332","333","334","335","336","337","338","339","340","341","342","343","344","345","346","347","348","349","350","351","352","353","354","355","356","357","358","359","360","361","362","363","364","365","366","367","368","369","370","371","372","373","374","375","376","377","378","379","380","381","382","383","384","385","386","387","388","389","390","391","392","393","394","395","396","397","398","399","400","401","402","403","404","405","406","407","408","409","410","411","412","413","414","415","416","417","418","419","420","421","422","423","424","425","426","427","428","429","430","431","432","433","434","435","436","437","438","439","440","441","442","443","444","445","446","447","448","449","450","451","452","453","454","455","456","457","458","459","460","461","462","463","464","465","466","467","468","469","470","471","472","473","474","475","476","477","478","479","480","481","482","483","484","485","486","487","488","489","490","491","492","493","494","495","496","497","498","499","500","501","502","503","504","505","506","507","508","509","510","511","512","513","514","515","516","517","518","519","520","521","522","523","524","525","526","527","528","529","530","531","532","533","534","535","536","537","538","539","540","541","542","543","544","545","546","547","548","549","550","551","552","553","554","555","556","557","558","559","560","561","562","563","564","565","566","567","568","569","570","571","572","573","574","575","576","577","578","579","580","581","582","583","584","585","586","587","588","589","590","591","592","593","594","595","596","597","598","599","600","601","602","603","604","605","606","607","608","609","610","611","612","613","614","615","616","617","618","619","620","621","622","623","624","625","626","627","628","629","630","631","632","633","634","635","636","637","638","639","640","641","642","643","644","645","646","647","648","649","650","651","652","653","654","655","656","657","658","659","660","661","662","663","664","665","666","667","668","669","670","671","672","673","674","675","676","677","678","679","680","681","682","683","684","685","686","687","688","689","690","691","692","693","694","695","696","697","698","699","700","701","702","703","704","705","706","707","708","709","710","711","712","713","714","715","716","717","718","719","720","721","722","723","724","725","726","727","728","729","730","731","732","733","734","735","736","737","738","739","740","741","742","743","744","745","746","747","748","749","750","751","752","753","754","755","756","757","758","759","760","761","762","763","764","765","766","767","768","769","770","771","772","773","774","775","776","777","778","779","780","781","782","783","784","785","786","787","788","789","790","791","792","793","794","795","796","797","798","799","800","801","802","803","804","805","806","807","808","809","810","811","812","813","814","815","816","817","818","819","820","821","822","823","824","825","826","827","828","829","830","831","832","833","834","835","836","837","838","839","840","841","842","843","844","845","846","847","848","849","850","851","852","853","854","855","856","857","858","859","860","861","862","863","864","865","866","867","868","869","870","871","872","873","874","875","876","877","878","879","880","881","882","883","884","885","886","887","888","889","890","891","892","893","894","895","896","897","898","899","900","901","902","903","904","905","906","907","908","909","910","911","912","913","914","915","916","917","918","919","920","921","922","923","924","925","926","927","928","929","930","931","932","933","934","935","936","937","938","939","940","941","942","943","944","945","946","947","948","949","950","951","952","953","954","955","956","957","958","959","960","961","962","963","964","965","966","967","968","969","970","971","972","973","974","975","976","977","978","979","980","981","982","983","984","985","986","987","988","989","990","991","992","993","994","995","996","997","998","999","1000","1001","1002","1003","1004","1005","1006","1007","1008","1009","1010","1011","1012","1013","1014","1015","1016","1017","1018","1019","1020","1021","1022","1023","1024","1025","1026","1027","1028","1029","1030","1031","1032","1033","1034","1035","1036","1037","1038","1039","1040","1041","1042","1043","1044","1045","1046","1047","1048","1049","1050","1051","1052","1053","1054","1055","1056","1057","1058","1059","1060","1061","1062","1063","1064","1065","1066","1067","1068","1069","1070","1071","1072","1073","1074","1075","1076","1077","1078","1079","1080","1081","1082","1083","1084","1085","1086","1087","1088","1089","1090","1091","1092","1093","1094","1095","1096","1097","1098","1099","1100","1101","1102","1103","1104","1105","1106","1107","1108","1109","1110","1111","1112","1113","1114","1115","1116","1117","1118","1119","1120","1121","1122","1123","1124","1125","1126","1127","1128","1129","1130","1131","1132","1133","1134","1135","1136","1137","1138","1139","1140","1141","1142","1143","1144","1145","1146","1147","1148","1149","1150","1151","1152","1153","1154","1155","1156","1157","1158","1159","1160","1161","1162","1163","1164","1165","1166","1167","1168","1169","1170","1171","1172","1173","1174","1175","1176","1177","1178","1179","1180","1181","1182","1183","1184","1185","1186","1187","1188","1189","1190","1191","1192","1193","1194","1195","1196","1197","1198","1199","1200","1201","1202","1203","1204"],["GO:CC","GO:CC","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:BP","GO:MF","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","KEGG","GO:CC","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","KEGG","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","KEGG","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","KEGG","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","KEGG","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:MF","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:MF","GO:MF","GO:MF","GO:BP","GO:CC","GO:CC","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","KEGG","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:MF","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","KEGG","GO:MF","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:CC","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP","GO:CC","GO:BP","GO:CC","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","KEGG","GO:MF","GO:BP","GO:MF","GO:BP","GO:CC","GO:BP","GO:MF","GO:CC","GO:BP"],[469,384,7163,9068,1879,2761,2739,2127,2715,4067,2920,2933,549,1026,1451,15598,491,283,1025,1959,7675,311,7682,4612,35,11989,836,1259,12699,6382,19757,3955,6335,277,12703,5399,5401,5645,470,3529,24853,6765,2963,9203,7118,3746,8640,536,9205,2749,15701,2752,15600,2129,2125,6769,16000,10311,4877,1888,15648,14844,21838,849,7577,240,2089,2123,12628,5405,3858,560,14840,18614,537,12301,9080,15322,10463,1024,2936,1450,837,514,2716,1444,2130,2126,12630,15886,10936,2443,15702,12691,118,1320,9981,3493,19028,4214,8642,2124,307,3659,1148,2866,184,29336,5742,26160,3777,16008,460,7676,3747,2406,3781,3780,6679,26026,5776,9081,3660,440,3543,15744,5816,9159,1862,1079,8261,18613,816,3063,15325,4783,15692,3978,9795,25711,4086,22777,4615,15146,9845,14845,13161,12702,103,3155,17116,16015,14895,9518,14442,3796,6472,15643,16064,12207,7586,15999,15159,16030,3776,9146,2884,8257,1959,4266,14805,3848,2128,2492,8804,3778,7647,24863,12102,4616,1452,15703,5347,2402,11992,8641,2446,6764,1111,10278,3579,24855,8297,4791,1874,456,1799,2766,2596,15642,1149,8255,7151,1526,2057,2938,817,3166,1993,11458,22673,11908,19684,2538,4617,4021,20855,10181,19115,309,2090,8108,3741,4767,4213,4715,6418,2403,4613,12432,8213,15158,4750,1117,3054,5639,13186,12433,3754,14967,4216,1568,9082,19681,183,3387,5050,2964,23191,15031,814,4614,7716,15640,16066,9175,9083,4784,504,12634,2393,15148,2972,2855,26852,15321,15410,3712,18612,11668,864,3055,9176,5116,3560,2433,23288,10001,312,3654,15520,1098,2440,4012,26378,17762,3487,18264,7648,22778,4713,13645,9844,12705,21888,6675,14841,15599,15713,2430,8263,4307,15719,10007,14305,8299,15888,12126,1282,3156,22948,2392,15103,17385,9571,7544,7599,3128,3276,29369,7813,17313,25367,12633,19341,8643,19137,8275,12124,3154,3742,1083,787,3533,12129,1021,1260,9170,454,11670,17767,3655,8265,5062,514,1251,445,26027,12064,294,12363,10381,2961,4927,7596,32,9237,14601,25976,16789,11511,10388,1847,4412,452,12362,24666,22776,1544,2146,3983,7730,12627,304,12123,11753,14999,9797,11621,15686,15887,2296,9400,11459,4508,12130,1889,7720,13455,15547,29114,3065,662,26028,512,16012,7069,22109,16240,18299,1263,8826,15776,4773,15541,22773,1215,16350,17256,1410,1253,16051,10382,6073,861,7672,2840,8803,12201,12978,15540,12980,15778,11602,8298,14662,2842,4439,5407,2443,15324,3069,8035,531,5891,6474,1696,121,8277,2960,3718,120,1023,24965,4067,15725,1022,625,13644,14968,4714,664,28670,8807,22772,3225,137,2115,22001,16026,28367,7677,8345,8732,2716,12697,19516,21090,24864,3518,8387,7703,2448,17032,1099,24964,8005,3578,3353,10010,29337,1100,7685,7814,2025,2548,2965,1478,3964,25365,12131,3060,6862,3608,19446,12455,17763,794,24854,7690,8645,9796,12692,12088,1903,773,3661,348,1101,7851,3659,21149,29371,12402,12776,9088,8411,5743,10125,17973,4776,1077,22137,10399,412,4134,826,15747,7026,1928,2486,9936,9580,4929,8034,15737,1479,21151,17195,2829,3353,7678,20371,15966,1074,15788,14319,15250,15249,1056,2539,6913,12396,7062,15721,5777,504,4496,14947,11464,14835,9078,480,18906,2853,15638,12313,7652,2891,15716,26380,25248,3291,8256,15694,12288,25144,13245,9959,14507,4208,333,15641,2914,28365,19404,8006,6089,8156,1932,23289,25143,19436,401,5998,6746,9071,8734,1880,2854,18439,3694,4792,2844,6251,25712,7595,2841,11576,5091,21318,3067,4268,17896,295,72,616,1184,4073,15787,8344,11669,4754,302,437,506,9602,22646,12227,21694,413,5649,12706,14353,13174,4132,12199,25245,677,15231,27352,27350,15697,3570,529,12298,1960,24929,7684,12169,3622,2149,15316,14896,286,4267,25878,405,3624,18583,14360,7653,11777,489,600,784,4006,15690,1599,24928,9072,1729,5650,3996,2394,15346,3567,26428,4441,4793,1677,3284,22870,5183,2835,20369,25713,5805,9070,4309,15746,141,3125,15523,13459,13573,896,71,13737,1944,12771,385,11884,4777,1059,15323,9601,1124,2843,3749,7686,3294,4649,14211,1445,1438,2183,3834,15885,3856,1120,66,408,22869,3186,14846,5092,17097,20874,2080,3557,19517,3599,21269,18256,2094,7161,13513,15785,2419,6323,25152,2439,299,16009,124,5563,3977,2437,8008,1872,5578,20313,10291,3996,3002,453,3662,11832,29283,3725,15683,13025,8605,192,1261,1265,2492,26191,4085,8157,2000,818,17803,21092,8163,8262,30397,509,22858,22859,22860,22857,13115,442,9206,15753,24967,8772,23524,12179,13114,12712,4212,416,17771,14606,19096,444,3623,8026,2941,12783,8007,2553,5794,201,4665,1588,9273,264,1405,1708,16007,8763,13638,15575,823,14897,14614,14615,102,11448,15738,101,21688,12395,334,10274,6436,22775,2038,410,18392,3073,2760,26163,14086,11454,24966,852,3547,898,2562,9808,25249,10,110,15576,30,9189,850,27875,1507,3751,9914,10292,16166,21129,8530,487,4277,20404,5031,388,982,457,9668,20875,1657,8160,5753,23283,863,1566,26911,4215,8159,3913,20403,296,2052,3787,3178,9677,35,513,15749,13250,3763,20958,21134,3765,4751,30400,317,314,2928,11201,663,16046,3157,2836,24602,3722,2021,4906,25880,2114,6822,1028,10166,2447,505,465,13512,7717,8101,9841,8774,11917,3801,4779,1859,15890,4944,6749,8072,23706,2904,2302,731,2483,3001,7889,20215,3857,1185,856,22157,3726,12759,19374,12315,1605,8169,22147,2921,9119,3836,842,16946,17717,19392,13574,532,6001,12684,2112,446,9081,3941,15720,1418,103,8755,547,1224,9295,15748,26189,2314,113,6092,5082,21327,22485,19249,6954,9270,11455,3447,100,3134,15029,2184,15104,4522,9847,9207,929,7891,12988,2561,501,2277,29370,1904,29847,20959,3700,5844,8010,3759,4708,16001,15639,1283,17746,2908,3731,11816,3377,18065,17875,7695,4738,12631,4672,26430,21695,19375,8606,9974,1073,418,619,23708,16947,543,1066,2930,21498,28672,6747,15615,2390,2405,11510,4090,605,8431,2991,20314,3662,28,2028,540,15710,439,14644,12429,15698,189,9669,17061,9975,2433,12405,26123,4907,11755,29098,12299,15519,26379,4707,5911,19083,21000,75,7815,3783,25349,870,9581,8264,9807,22502,12343,3795,10253,17715,2578,41,20985,8760,5326,3483,4209,2384,866,19097,9650,4519,8011,300,1006,1166,25246,8258,5669,19243,3003,19242,3586,16729,9228,7888,15711,867,2557,4218,8644,15736,324,15571,12022,30012,1436,21008,21410,917,4312,8260,11905,21717,9853,18983,3142,8376,11513,1075,409,3650,3766,10287,24533,8531,601,12512,1140,10784,14379,1136,3280,15684,12762,2197,4273,12371,9209,8387,12154,604,23042,300,27241,27239,30365,22638,2356,862,30415,486,6665,2912,3374,2042,13557,5093,14487,4774,22,10515,530,7320,3384,983,30398,9115,7084,13028,9882,24504,4211,17168,19132,3066,4315,17819,3596,7171,5067,326,4608,19393,8374,204,417,19412,441,1716,18713],["cytosol","cytoplasm","cadherin binding","cell adhesion molecule binding","protein binding","anchoring junction","extracellular exosome","extracellular organelle","extracellular membrane-bounded organelle","extracellular vesicle","organelle organization","cytoskeleton organization","focal adhesion","cell-substrate junction","vesicle","regulation of cellular component organization","cytoskeleton","intracellular anatomical structure","cell junction","cytoskeletal protein binding","actin filament-based process","nucleoplasm","actin cytoskeleton organization","enzyme binding","nucleotide binding","nucleoside phosphate binding","actin cytoskeleton","cell leading edge","cellular macromolecule metabolic process","small molecule binding","cellular component organization or biogenesis","purine nucleotide binding","purine ribonucleoside triphosphate binding","extracellular space","cellular protein metabolic process","ribonucleotide binding","purine ribonucleotide binding","cellular component organization","actin binding","protein localization","organonitrogen compound metabolic process","anion binding","cell cycle","macromolecule localization","protein-containing complex binding","supramolecular complex","regulation of cellular protein metabolic process","cell-cell junction","regulation of organelle organization","phosphorus metabolic process","regulation of protein metabolic process","phosphate-containing compound metabolic process","positive regulation of cellular component organization","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","protein metabolic process","cellular localization","carbohydrate derivative binding","adenyl nucleotide binding","ATP binding","localization","positive regulation of cellular process","supramolecular fiber organization","membrane","cell cycle process","extracellular region","cell projection","organelle","regulation of cellular component biogenesis","adenyl ribonucleotide binding","plasma membrane bounded cell projection","cell cortex","positive regulation of biological process","regulation of molecular function","adherens junction","macromolecule modification","actin filament binding","regulation of catalytic activity","intracellular signal transduction","lamellipodium","actin filament organization","nuclear lumen","microtubule cytoskeleton","catalytic activity","intracellular organelle lumen","membrane-enclosed lumen","organelle lumen","intracellular organelle","positive regulation of cellular component biogenesis","regulation of cytoskeleton organization","protein modification process","cellular protein modification process","positive regulation of protein metabolic process","cellular catabolic process","mitotic cell cycle","cytoplasmic vesicle","cellular protein localization","intracellular vesicle","cellular macromolecule localization","regulation of catabolic process","positive regulation of cellular protein metabolic process","membrane-bounded organelle","cell morphogenesis","catabolic process","cortical cytoskeleton","movement of cell or subcellular component","ruffle","regulation of cellular macromolecule biosynthetic process","vesicle-mediated transport","mitotic cell cycle process","supramolecular fiber","establishment of localization in cell","spindle","cell projection organization","supramolecular polymer","regulation of translation","hydrolase activity, acting on acid anhydrides, in phosphorus-containing anhydrides","hydrolase activity, acting on acid anhydrides","identical protein binding","regulation of supramolecular fiber organization","phosphorylation","regulation of localization","macromolecule catabolic process","RNA binding","pyrophosphatase activity","cell division","cell migration","regulation of actin filament-based process","binding","secretory granule","regulation of cellular catabolic process","regulation of biological quality","endomembrane system","establishment or maintenance of cell polarity","regulation of cellular process","positive regulation of organelle organization","establishment of localization","nucleoside-triphosphatase activity","regulation of cellular amide metabolic process","regulation of intracellular signal transduction","ficolin-1-rich granule lumen","plasma membrane bounded cell projection organization","phosphatase binding","anatomical structure development","cell junction organization","negative regulation of cellular process","establishment of protein localization","cellular macromolecule catabolic process","microtubule cytoskeleton organization","small GTPase mediated signal transduction","regulation of cellular localization","establishment of organelle localization","regulation of response to stimulus","cellular response to stress","intracellular transport","ficolin-1-rich granule","regulation of phosphate metabolic process","regulation of phosphorus metabolic process","cellular response to stimulus","regulation of protein-containing complex assembly","cellular component disassembly","organelle localization","cell motility","localization of cell","secretory vesicle","regulation of actin cytoskeleton organization","response to stress","positive regulation of cellular metabolic process","cytoplasmic ribonucleoprotein granule","regulation of signal transduction","cell development","ATP hydrolysis activity","intracellular membrane-bounded organelle","peptide metabolic process","developmental process","polymeric cytoskeletal fiber","regulation of signaling","organic substance catabolic process","peptide biosynthetic process","protein phosphatase binding","vesicle lumen","negative regulation of protein metabolic process","protein transport","translation","heterocyclic compound binding","negative regulation of cellular protein metabolic process","protein phosphorylation","ion binding","midbody","organic cyclic compound binding","plasma membrane region","organonitrogen compound biosynthetic process","regulation of protein modification process","regulation of cell communication","ribonucleoprotein granule","microtubule organizing center","secretory granule lumen","transport","cytoplasmic vesicle lumen","positive regulation of nitrogen compound metabolic process","cortical actin cytoskeleton","regulation of cellular metabolic process","translation regulator activity","actin filament bundle","actomyosin","microtubule-based process","vesicle membrane","multicellular organism development","translation factor activity, RNA binding","locomotion","regulation of actin filament organization","ATP-dependent activity","nitrogen compound transport","perinuclear region of cytoplasm","protein domain specific binding","anatomical structure morphogenesis","regulation of primary metabolic process","translation regulator activity, nucleic acid binding","cellular response to chemical stimulus","cell morphogenesis involved in differentiation","neuron projection","actomyosin structure organization","transferase activity, transferring phosphorus-containing groups","enzyme regulator activity","positive regulation of metabolic process","regulation of cell cycle process","viral life cycle","translational initiation","kinase binding","cellular amide metabolic process","neuron projection development","cellular developmental process","positive regulation of macromolecule metabolic process","cytoplasmic vesicle membrane","cell communication","viral process","cell-cell junction organization","amide biosynthetic process","hydrolase activity","neuron development","positive regulation of catabolic process","protein-containing complex","regulation of protein localization","organic substance transport","stress fiber","contractile actin filament bundle","small GTPase binding","spindle organization","non-membrane-bounded organelle assembly","system development","cytoplasmic stress granule","protein kinase binding","cell differentiation","regulation of nitrogen compound metabolic process","regulation of cell cycle","cellular component morphogenesis","GTPase binding","negative regulation of organelle organization","microtubule","positive regulation of molecular function","mRNA catabolic process","cell projection morphogenesis","chromosome segregation","apoptotic process","positive regulation of protein localization","regulation of biological process","response to stimulus","transferase activity","biological regulation","regulation of phosphorylation","lateral plasma membrane","cell adhesion","cell part morphogenesis","programmed cell death","cell death","membrane raft","cell-substrate junction organization","cellular macromolecule biosynthetic process","cytokinesis","membrane microdomain","actin filament bundle assembly","axon","basal part of cell","catalytic complex","regulation of mRNA metabolic process","regulation of mRNA catabolic process","kinase activity","actin filament bundle organization","signaling","plasma membrane bounded cell projection morphogenesis","positive regulation of phosphorus metabolic process","positive regulation of phosphate metabolic process","cell junction assembly","cellular nitrogen compound catabolic process","lamellipodium organization","aromatic compound catabolic process","negative regulation of biological process","negative regulation of cellular component organization","protein polymerization","regulation of translational initiation","positive regulation of cellular catabolic process","response to organic substance","regulation of cellular component movement","nucleobase-containing compound catabolic process","heterocycle catabolic process","positive regulation of protein modification process","positive regulation of cytoskeleton organization","negative regulation of programmed cell death","protein serine/threonine/tyrosine kinase activity","Ras protein signal transduction","mitotic nuclear division","RNA catabolic process","neuron projection morphogenesis","regulation of vesicle-mediated transport","molecular adaptor activity","neurogenesis","cellular component assembly","cell periphery","nervous system development","regulation of cell motility","regulation of cell migration","negative regulation of cell death","positive regulation of organelle assembly","negative regulation of molecular function","cellular response to organic substance","regulation of protein polymerization","organelle assembly","regulation of cell projection organization","regulation of programmed cell death","mitotic spindle","phosphotransferase activity, alcohol group as acceptor","filopodium","basal plasma membrane","actin polymerization or depolymerization","positive regulation of catalytic activity","myofibril","cell projection membrane","protein-containing complex disassembly","centrosome","positive regulation of phosphorylation","membrane organization","actin-based cell projection","positive regulation of protein-containing complex assembly","regulation of cell death","plasma membrane","protein kinase activity","mRNA binding","negative regulation of supramolecular fiber organization","regulation of apoptotic process","nucleus","regulation of mRNA stability","cell adhesion mediator activity","cell-substrate junction assembly","protein-macromolecule adaptor activity","regulation of cell morphogenesis","mitotic sister chromatid segregation","peroxiredoxin activity","Golgi vesicle transport","microtubule cytoskeleton organization involved in mitosis","nucleobase-containing small molecule metabolic process","wound healing","cadherin binding involved in cell-cell adhesion","microtubule plus-end","microtubule end","translation initiation factor activity","regulation of RNA stability","organic cyclic compound catabolic process","regulation of plasma membrane bounded cell projection organization","ruffle membrane","contractile fiber","response to wounding","neuron differentiation","cellular component biogenesis","sister chromatid segregation","negative regulation of apoptotic process","catalytic activity, acting on a protein","generation of neurons","positive regulation of cellular amide metabolic process","response to chemical","spindle assembly","negative regulation of cytoskeleton organization","cell-cell contact zone","protein localization to organelle","regulation of locomotion","endomembrane system organization","negative regulation of catalytic activity","GTP binding","protein catabolic process","positive regulation of translation","regulation of small GTPase mediated signal transduction","protein localization to cell periphery","signal transduction","regulation of protein phosphorylation","positive regulation of supramolecular fiber organization","actin filament","spindle localization","S100 protein binding","nuclear chromosome segregation","regulation of chromosome segregation","cytoskeleton-dependent cytokinesis","leading edge membrane","regulation of cellular component size","regulation of hydrolase activity","positive regulation of gene expression","positive regulation of transport","regulation of plasma membrane bounded cell projection assembly","organelle membrane","movement in host environment","regulation of cell projection assembly","I band","protein serine/threonine kinase activity","biological process involved in interaction with host","cell-cell adhesion mediator activity","peptidyl-amino acid modification","basolateral plasma membrane","establishment of cell polarity","endocytosis","multicellular organismal process","regulation of protein-containing complex disassembly","cell cycle phase transition","regulation of transport","mitotic cell cycle phase transition","regulation of transferase activity","protein serine kinase activity","negative regulation of protein modification process","organelle fission","vesicle budding from membrane","guanyl nucleotide binding","guanyl ribonucleotide binding","synapse","regulation of developmental process","transmembrane receptor protein tyrosine kinase signaling pathway","cortical actin cytoskeleton organization","brush border","negative regulation of translation","regulation of metabolic process","structural constituent of cytoskeleton","mitotic cytokinesis","positive regulation of cell projection organization","cell-cell junction assembly","nucleotide metabolic process","nuclear division","Z disc","cellular response to nitrogen compound","response to endogenous stimulus","chromosome organization","sarcomere","endothelial cell development","negative regulation of phosphate metabolic process","cell morphogenesis involved in neuron differentiation","negative regulation of phosphorus metabolic process","positive regulation of protein phosphorylation","regulation of chromosome separation","cytokinetic process","plasma membrane bounded cell projection assembly","regulation of mitotic cell cycle","spindle pole","thioredoxin peroxidase activity","cell-cell adhesion","localization within membrane","positive regulation of protein localization to membrane","cell projection assembly","actin cytoskeleton reorganization","regulation of intracellular transport","nucleoside phosphate metabolic process","cellular protein catabolic process","cellular response to endogenous stimulus","regulation of anatomical structure size","organic substance biosynthetic process","regulation of actin polymerization or depolymerization","cell-substrate adhesion","regulation of endocytosis","protein dephosphorylation","regulation of macromolecule metabolic process","dendrite","response to nitrogen compound","regulation of actin filament length","bounding membrane of organelle","dendritic tree","ncRNA metabolic process","negative regulation of cellular macromolecule biosynthetic process","growth cone","actin filament polymerization","positive regulation of cell migration","melanosome","pigment granule","mitotic spindle organization","cell division site","intracellular protein-containing complex","regulation of organelle assembly","regulation of GTPase activity","cell-matrix adhesion","organophosphate metabolic process","chromosomal region","cellular response to organonitrogen compound","modification-dependent macromolecule catabolic process","positive regulation of mRNA catabolic process","endosome membrane","organonitrogen compound catabolic process","actin filament-based movement","positive regulation of protein polymerization","negative regulation of cellular amide metabolic process","cellular biosynthetic process","chordate embryonic development","phospholipid binding","epithelial cell development","biosynthetic process","Regulation of actin cytoskeleton","site of polarized growth","protein N-terminus binding","cluster of actin-based cell projections","tissue migration","positive regulation of cell motility","regulation of translation in response to stress","entry into host","regulation of microtubule-based process","regulation of protein stability","endosomal transport","establishment or maintenance of apical/basal cell polarity","establishment or maintenance of bipolar cell polarity","epithelial cell migration","endocytic vesicle","vesicle-mediated transport to the plasma membrane","molecular function regulator activity","endosome","embryo development ending in birth or egg hatching","intercalated disc","chromosome separation","modification-dependent protein catabolic process","microtubule binding","ubiquitin-dependent protein catabolic process","ncRNA processing","nucleoside-triphosphatase regulator activity","GTPase regulator activity","cortical cytoskeleton organization","establishment of spindle localization","cleavage furrow","epithelium migration","epithelium development","intracellular protein transport","tubulin binding","lamellipodium assembly","protein localization to plasma membrane","proteolysis involved in cellular protein catabolic process","coated vesicle","negative regulation of transferase activity","viral entry into host cell","RNA transport","nucleic acid transport","membrane coat","coated membrane","ribose phosphate metabolic process","regulation of kinase activity","ubiquitin-like protein ligase binding","positive regulation of cellular component movement","dephosphorylation","in utero embryonic development","response to organonitrogen compound","anatomical structure formation involved in morphogenesis","positive regulation of locomotion","animal organ development","microtubule plus-end binding","polysome","regulation of microtubule cytoskeleton organization","nucleocytoplasmic transport","nuclear transport","response to peptide hormone","positive regulation of signaling","tertiary granule","protein depolymerization","positive regulation of mRNA metabolic process","regulation of mitotic cell cycle phase transition","central nervous system development","negative regulation of cellular metabolic process","establishment of RNA localization","regulation of protein binding","negative regulation of protein depolymerization","endothelial cell differentiation","protein localization to nucleus","focal adhesion assembly","tissue development","nuclear-transcribed mRNA catabolic process","negative regulation of nitrogen compound metabolic process","endoplasmic reticulum exit site","regulation of protein localization to membrane","cellular response to peptide hormone stimulus","regulation of actin filament polymerization","peptidyl-serine modification","microtubule polymerization or depolymerization","protein C-terminus binding","regulation of cell-substrate junction organization","regulation of protein depolymerization","cellular response to organic cyclic compound","nucleic acid binding","peptidyl-serine phosphorylation","ribonucleoprotein complex binding","cellular response to insulin stimulus","positive regulation of intracellular transport","calmodulin binding","autophagy","process utilizing autophagic mechanism","glutamatergic synapse","positive regulation of cell communication","vesicle targeting","phosphatidylinositol binding","negative regulation of intracellular signal transduction","regulation of anatomical structure morphogenesis","receptor-mediated endocytosis","regulation of protein catabolic process","ubiquitin protein ligase binding","mitotic spindle assembly","enzyme-linked receptor protein signaling pathway","negative regulation of signal transduction","mRNA destabilization","nuclear envelope","MAPK cascade","F-actin capping protein complex","cortical microtubule cytoskeleton","response to hormone","positive regulation of transferase activity","ruffle organization","negative regulation of phosphorylation","post-transcriptional regulation of gene expression","nuclear pore","Golgi apparatus","spindle microtubule","positive regulation of kinase activity","regulation of actomyosin structure organization","apical junction assembly","apoptotic signaling pathway","early endosome","cell growth","cellular nitrogen compound biosynthetic process","protein autophosphorylation","establishment or maintenance of epithelial cell apical/basal polarity","embryo development","negative regulation of protein-containing complex disassembly","regulation of cell cycle phase transition","regulation of cell-matrix adhesion","chromosome localization","positive regulation of protein localization to cell periphery","regulation of protein localization to cell periphery","regulation of multicellular organismal process","insulin receptor signaling pathway","caveola","regulation of MAPK cascade","somatodendritic compartment","cellular response to peptide","contractile actin filament bundle assembly","stress fiber assembly","postsynapse","apical junction complex","RNA destabilization","positive regulation of response to stimulus","virus receptor activity","positive regulation of signal transduction","regulation of lamellipodium organization","Bacterial invasion of epithelial cells","plasma membrane protein complex","cellular response to chemical stress","microtubule polymerization","negative regulation of signaling","exogenous protein binding","ameboidal-type cell migration","GTPase activity","cytoplasmic side of plasma membrane","vinculin binding","spindle elongation","guanyl-nucleotide exchange factor activity","response to peptide","cellular response to hormone stimulus","endothelium development","vesicle organization","distal axon","RNA localization","protein stabilization","cell population proliferation","regulation of cellular protein catabolic process","GDP binding","negative regulation of cell communication","integrin binding","axonogenesis","tight junction organization","response to organic cyclic compound","post-Golgi vesicle-mediated transport","protein localization to membrane","positive regulation of intracellular signal transduction","dendrite development","response to insulin","response to inorganic substance","establishment of chromosome localization","P-body","integrin-mediated signaling pathway","mRNA transport","positive regulation of protein catabolic process","regulation of protein kinase activity","flotillin complex","magnesium ion binding","GMP metabolic process","enzyme activator activity","biological process involved in symbiotic interaction","mitochondrion","muscle cell differentiation","regulation of epithelial cell migration","vesicle coat","regulation of viral process","negative regulation of kinase activity","secretory granule membrane","vesicle coating","purine ribonucleotide metabolic process","actin filament depolymerization","brain development","regulation of gene expression","regulation of Ras protein signal transduction","envelope","organelle envelope","purine nucleotide metabolic process","B-WICH complex","regulation of stress fiber assembly","ribonucleotide metabolic process","coated vesicle membrane","lytic vacuole","lysosome","tight junction assembly","Flemming body","positive regulation of viral process","regulation of neuron projection development","head development","regulation of response to stress","lipid binding","cytoplasmic side of membrane","cellular response to external stimulus","regulation of cell shape","regulation of muscle system process","axon development","costamere","alpha-catenin binding","positive regulation of cell cycle","positive regulation of hydrolase activity","plasma membrane raft","signaling adaptor activity","regulation of cell junction assembly","apical part of cell","Adherens junction","establishment of vesicle localization","nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay","RNA strand annealing activity","response to external stimulus","protein folding","negative regulation of actin filament depolymerization","myelin sheath abaxonal region","nucleobase-containing compound transport","purine-containing compound metabolic process","regulation of dephosphorylation","response to abiotic stimulus","mitotic cell cycle checkpoint signaling","lipid droplet","oxidoreductase activity, acting on peroxide as acceptor","homeostatic process","regulation of ubiquitin-dependent protein catabolic process","purine nucleoside monophosphate metabolic process","positive regulation of protein transport","actomyosin contractile ring organization","regulation of actin filament bundle assembly","uropod","cell trailing edge","lamellipodium membrane","phagocytic vesicle","positive regulation of protein localization to plasma membrane","tertiary granule lumen","regulation of microtubule polymerization or depolymerization","poly(A) binding","ER to Golgi transport vesicle membrane","muscle structure development","positive regulation of cell cycle process","positive regulation of microtubule polymerization","negative regulation of cellular catabolic process","regulation of apoptotic signaling pathway","cytoplasmic microtubule","focal adhesion disassembly","regulation of focal adhesion disassembly","positive regulation of focal adhesion disassembly","cell-substrate junction disassembly","positive regulation of viral genome replication","double-stranded RNA binding","regulation of chromosome organization","metaphase plate congression","cellular response to oxygen-containing compound","regulation of cytokinesis","regulation of protein localization to nucleus","proteasome-mediated ubiquitin-dependent protein catabolic process","regulation of viral genome replication","small molecule metabolic process","negative regulation of metabolic process","vacuole","establishment of endothelial barrier","vesicle targeting, to, from or within Golgi","response to growth factor","single-stranded RNA binding","membrane protein complex","epithelial cell differentiation","microtubule nucleation","biological process involved in interspecies interaction between organisms","regulation of actin filament depolymerization","spindle midzone","meiotic chromosome movement towards spindle pole","podosome","proteasomal protein catabolic process","muscle system process","regulation of intracellular estrogen receptor signaling pathway","Endocytosis","spindle pole centrosome","nuclear periphery","vesicle localization","endocytic recycling","negative regulation of mitotic cell cycle","regulation of binding","postsynaptic density","negative regulation of response to stimulus","vesicle targeting, rough ER to cis-Golgi","COPII vesicle coating","condensed chromosome, centromeric region","establishment of mitotic spindle localization","establishment of spindle orientation","kinetochore","ruffle assembly","positive regulation of GTPase activity","spliceosomal complex","scaffold protein binding","viral genome replication","positive regulation of plasma membrane bounded cell projection assembly","azurophil granule","primary lysosome","microtubule organizing center localization","epidermal growth factor receptor signaling pathway","tight junction","regulation of proteolysis involved in cellular protein catabolic process","ribose phosphate biosynthetic process","growth","response to oxygen-containing compound","cytoplasmic translational initiation","side of membrane","nuclear body","recycling endosome membrane","proline-rich region binding","negative regulation of mitotic cell cycle phase transition","mitotic spindle elongation","condensed chromosome","positive regulation of binding","microfilament motor activity","protein phosphatase 2A binding","cytoplasmic translation","positive regulation of establishment of protein localization","asymmetric synapse","purine ribonucleotide biosynthetic process","poly-purine tract binding","regulation of protein dephosphorylation","regulation of focal adhesion assembly","regulation of cell-substrate junction assembly","negative regulation of protein binding","eukaryotic translation initiation factor 2B complex","response to extracellular stimulus","protein localization to microtubule cytoskeleton","heat shock protein binding","dystroglycan binding","extrinsic component of plasma membrane","regulation of cell growth","response to cytokine","regulation of cellular response to stress","elongator holoenzyme complex","regulation of microtubule polymerization","macroautophagy","protein localization to cell-cell junction","apicolateral plasma membrane","desmosome organization","regulation of viral life cycle","negative regulation of catabolic process","positive regulation of microtubule polymerization or depolymerization","response to heat","protein localization to cell cortex","Focal adhesion","MHC class I protein complex","postsynaptic specialization","FAR/SIN/STRIPAK complex","homotypic cell-cell adhesion","cell cycle checkpoint signaling","Arp2/3 protein complex","mitotic sister chromatid separation","cell redox homeostasis","nucleotide biosynthetic process","cardiac muscle cell action potential","COPII-coated vesicle budding","purine ribonucleoside monophosphate metabolic process","negative regulation of macromolecule metabolic process","regulation of extrinsic apoptotic signaling pathway","actomyosin contractile ring assembly","assembly of actomyosin apparatus involved in cytokinesis","mitochondrion organization","ERBB signaling pathway","negative regulation of protein phosphorylation","actin filament capping","Rho protein signal transduction","Golgi to plasma membrane transport","nucleoside phosphate biosynthetic process","nucleoside monophosphate metabolic process","sarcolemma","regulation of cell morphogenesis involved in differentiation","positive regulation of lamellipodium organization","organic acid metabolic process","ADP binding","desmosome","cell adhesive protein binding involved in bundle of His cell-Purkinje myocyte communication","negative regulation of protein kinase activity","microtubule associated complex","cytoskeletal motor activity","negative regulation of cell cycle","regulation of cell adhesion","microtubule organizing center organization","Arp2/3 complex-mediated actin nucleation","positive regulation of cytokinesis","annealing activity","mCRD-mediated mRNA stability complex","positive regulation of epithelial cell migration","azurophil granule lumen","negative regulation of stress fiber assembly","regulation of cell-substrate adhesion","ribosomal small subunit binding","astral microtubule organization","regulation of post-transcriptional gene silencing by RNA","cellular response to DNA damage stimulus","cell body","morphogenesis of an epithelial sheet","proteolysis","metaphase/anaphase transition of mitotic cell cycle","intracellular estrogen receptor signaling pathway","circulatory system development","ribonucleotide biosynthetic process","peroxidase activity","inclusion body","regulation of ventricular cardiac muscle cell action potential","purine nucleoside monophosphate biosynthetic process","protein localization to cytoskeleton","cellular response to cytokine stimulus","oxoacid metabolic process","GTPase activator activity","cytoplasmic microtubule organization","regulation of cardiac muscle cell action potential","nucleus organization","circadian regulation of gene expression","ligase activity","desmosome assembly","regulation of post-transcriptional gene silencing","regulation of gene silencing by RNA","cellular response to growth factor stimulus","positive regulation of protein kinase activity","clathrin-coated pit","peptidyl-tyrosine phosphorylation","cellular metabolic process","dendritic spine","mRNA 3'-UTR binding","protein kinase A binding","structural constituent of nuclear pore","negative regulation of cellular component movement","early endosome membrane","chromosome, telomeric region","regulation of proteasomal ubiquitin-dependent protein catabolic process","bicellular tight junction","pseudopodium","meiotic cytokinesis","chromosome movement towards spindle pole","regulation of protein localization to plasma membrane","neuron spine","adenyl-nucleotide exchange factor activity","peptidyl-tyrosine modification","regulation of mitotic sister chromatid separation","positive regulation of intracellular protein transport","cellular response to environmental stimulus","cellular response to abiotic stimulus","carboxylic acid metabolic process","regulation of intracellular steroid hormone receptor signaling pathway","regulation of growth","antioxidant activity","chromosome, centromeric region","PTW/PP1 phosphatase complex","tissue morphogenesis","purine nucleotide biosynthetic process","dendrite morphogenesis","phosphatase regulator activity","adherens junction organization","regulation of sister chromatid segregation","translation release factor complex","intracellular receptor signaling pathway","metaphase/anaphase transition of cell cycle","recycling endosome","kinesin complex","DNA metabolic process","negative regulation of cell motility","calcium-dependent phospholipid binding","positive regulation of miRNA-mediated gene silencing","cardiac muscle cell action potential involved in contraction","neuron to neuron synapse","protein ubiquitination","negative regulation of actin filament polymerization","ribonucleoside monophosphate metabolic process","positive regulation of macromolecule biosynthetic process","centrosome localization","import into nucleus","protein tyrosine kinase activity","dendritic spine development","response to oxidative stress","nucleoside diphosphate metabolic process","P-type transmembrane transporter activity","P-type ion transporter activity","cardiac conduction","regulation of proteasomal protein catabolic process","regulation of mitotic metaphase/anaphase transition","regulation of lamellipodium assembly","positive regulation of vacuole organization","regulation of autophagy","positive regulation of cellular protein catabolic process","extrinsic apoptotic signaling pathway","cellular response to interferon-gamma","negative regulation of actin filament bundle assembly","cellular response to oxidative stress","COPII-coated ER to Golgi transport vesicle","vacuolar lumen","integrin complex","positive regulation of post-transcriptional gene silencing by RNA","positive regulation of post-transcriptional gene silencing","membrane raft assembly","COPII vesicle coat","catalytic step 2 spliceosome","establishment of endothelial intestinal barrier","positive regulation of chromosome separation","ribosome binding","striated muscle cell differentiation","tRNA metabolic process","translational termination","negative regulation of epidermal growth factor receptor signaling pathway","ATPase complex","COP9 signalosome","cellular response to extracellular stimulus","mitotic metaphase plate congression","purine-containing compound biosynthetic process","macromolecule biosynthetic process","protein phosphatase type 1 complex","eukaryotic initiation factor 4E binding","fascia adherens","spindle midzone assembly","Proteoglycans in cancer","regulation of receptor-mediated endocytosis","skin development","positive regulation of multicellular organismal process","immunological synapse","erythrocyte homeostasis","regulation of cell development","cellular response to heat","regulation of translational termination","regulation of translational initiation in response to stress","organelle disassembly","positive regulation of cell morphogenesis involved in differentiation","catalytic activity, acting on RNA","cellular detoxification","negative regulation of MAPK cascade","barbed-end actin filament capping","negative regulation of mRNA metabolic process","regulation of macromolecule biosynthetic process","protein kinase A regulatory subunit binding","bicellular tight junction assembly","cell communication involved in cardiac conduction","nuclear-transcribed mRNA catabolic process, nonsense-mediated decay","negative regulation of cell migration","ribonucleoside diphosphate metabolic process","regulation of metaphase/anaphase transition of cell cycle","myosin complex","ATPase regulator activity","negative regulation of protein-containing complex assembly","RNA polymerase binding","neuron projection organization","regulation of generation of precursor metabolites and energy","cell cortex region","tube morphogenesis","regulation of miRNA-mediated gene silencing","protein import into nucleus","G1/S transition of mitotic cell cycle","cardiac muscle cell-cardiac muscle cell adhesion","protein modification by small protein conjugation","myotube differentiation","circadian rhythm","regulation of biosynthetic process","nuclear pore inner ring","catenin complex","response to epidermal growth factor","stress granule assembly","nucleobase-containing compound kinase activity","positive regulation of actin filament polymerization","Tight junction","helicase activity","retromer complex","negative regulation of cell cycle phase transition","regulation of cellular biosynthetic process","rRNA metabolic process","mitotic spindle checkpoint signaling","mitotic spindle assembly checkpoint signaling","spindle assembly checkpoint signaling","protein complex involved in cell adhesion","muscle cell development","Hsp90 protein binding","intracellular steroid hormone receptor signaling pathway","mitotic spindle midzone assembly","nuclear matrix","cortical microtubule","transferase complex","negative regulation of protein polymerization","nuclear pore complex assembly","phosphotyrosine residue binding","positive regulation of developmental process","phosphatidylinositol phosphate binding","regulation of bicellular tight junction assembly","nuclear membrane","bundle of His cell-Purkinje myocyte adhesion involved in cell communication","organophosphate biosynthetic process","myeloid cell homeostasis","response to metal ion","positive regulation of cellular biosynthetic process","catalytic activity, acting on a nucleic acid","cellular response to toxic substance","response to interferon-gamma","protein modification by small protein conjugation or removal","I-kappaB kinase/NF-kappaB signaling","spindle checkpoint signaling","gliogenesis","clathrin-coated vesicle","lysosomal membrane","lytic vacuole membrane","purine ribonucleoside monophosphate biosynthetic process","tube development","regulation of neuron death","positive regulation of protein binding","G protein activity","protein-containing complex organization","preribosome, small subunit precursor","cellular response to topologically incorrect protein","regulation of nucleocytoplasmic transport","preribosome","centralspindlin complex","regulation of protein transport","negative regulation of protein kinase activity by regulation of protein phosphorylation","GMP biosynthetic process","cellular process","regulation of muscle adaptation","regulation of mitotic sister chromatid segregation","calcium-dependent protein binding","regulation of I-kappaB kinase/NF-kappaB signaling","spectrin","cell-cell fusion","syncytium formation by plasma membrane fusion","positive regulation of basement membrane assembly involved in embryonic body morphogenesis","regulation of basement membrane assembly involved in embryonic body morphogenesis","basement membrane assembly involved in embryonic body morphogenesis","regulation of basement membrane organization","regulation of transcription by RNA polymerase III","apical plasma membrane","negative regulation of chromosome organization","eukaryotic translation initiation factor 2 complex","ATPase-coupled transmembrane transporter activity","contractile ring","heart development","glycogen granule","negative regulation of mitotic metaphase/anaphase transition","positive regulation of neuron projection development","platelet-derived growth factor receptor signaling pathway","negative regulation of gene expression","exocyst","maintenance of blood-brain barrier","microvillus","ephrin receptor binding","myoblast fusion","extrinsic component of membrane","negative regulation of apoptotic signaling pathway","phosphoprotein binding","pallium development","cell cycle G1/S phase transition","low-density lipoprotein particle clearance","negative regulation of ERBB signaling pathway","positive regulation of biosynthetic process","growth hormone receptor signaling pathway via JAK-STAT","primary sncRNA processing","cell surface receptor signaling pathway","response to iron(III) ion","chaperone-mediated protein folding","asymmetric cell division","protein phosphorylated amino acid binding","negative regulation of cell cycle process","Leukocyte transendothelial migration","protein phosphatase regulator activity","cellular response to epidermal growth factor stimulus","tau protein binding","maturation of 5.8S rRNA","vacuolar membrane","cellular response to steroid hormone stimulus","RNA helicase activity","centriolar satellite","regulation of establishment of protein localization"],["1.1e-171","5.0e-128"," 8.0e-80"," 5.5e-67"," 2.8e-66"," 5.2e-63"," 4.4e-59"," 5.1e-59"," 5.1e-59"," 1.6e-58"," 1.8e-57"," 7.9e-51"," 1.3e-49"," 1.4e-49"," 6.4e-48"," 5.7e-47"," 1.4e-46"," 1.1e-43"," 5.5e-43"," 1.5e-41"," 7.4e-40"," 5.5e-39"," 9.4e-39"," 3.0e-37"," 1.9e-36"," 2.0e-36"," 4.1e-36"," 1.5e-35"," 1.6e-35"," 1.5e-34"," 2.3e-34"," 3.9e-34"," 1.1e-33"," 2.5e-33"," 8.2e-33"," 1.2e-32"," 6.1e-32"," 1.5e-31"," 3.9e-31"," 1.0e-30"," 2.6e-30"," 2.8e-30"," 3.6e-30"," 5.1e-30"," 5.4e-30"," 7.1e-30"," 8.9e-30"," 1.0e-29"," 1.8e-29"," 3.1e-29"," 6.2e-29"," 6.9e-29"," 1.2e-28"," 8.3e-28"," 8.6e-28"," 1.1e-27"," 1.2e-27"," 1.5e-27"," 6.4e-27"," 9.8e-27"," 1.2e-26"," 1.6e-26"," 2.4e-26"," 6.6e-26"," 1.3e-25"," 1.4e-25"," 2.2e-25"," 2.8e-25"," 3.2e-25"," 3.4e-25"," 8.3e-25"," 8.5e-25"," 1.3e-24"," 1.6e-24"," 2.4e-24"," 1.3e-23"," 1.5e-23"," 6.2e-23"," 7.7e-23"," 1.2e-22"," 1.5e-22"," 2.3e-22"," 2.9e-22"," 3.6e-22"," 5.3e-22"," 5.5e-22"," 5.5e-22"," 1.0e-21"," 2.9e-21"," 3.3e-21"," 3.5e-21"," 3.5e-21"," 1.4e-20"," 2.3e-20"," 3.3e-20"," 3.3e-20"," 3.5e-20"," 3.7e-20"," 6.4e-20"," 2.1e-19"," 2.4e-19"," 2.6e-19"," 2.7e-19"," 2.9e-19"," 3.7e-19"," 3.8e-19"," 5.5e-19"," 1.1e-18"," 1.2e-18"," 2.1e-18"," 4.8e-18"," 5.1e-18"," 7.5e-18"," 9.1e-18"," 9.3e-18"," 1.1e-17"," 1.2e-17"," 1.2e-17"," 1.6e-17"," 1.9e-17"," 2.0e-17"," 2.4e-17"," 2.4e-17"," 3.3e-17"," 3.4e-17"," 3.5e-17"," 3.6e-17"," 4.0e-17"," 5.0e-17"," 5.8e-17"," 8.0e-17"," 1.1e-16"," 1.2e-16"," 1.7e-16"," 2.1e-16"," 2.3e-16"," 2.4e-16"," 2.7e-16"," 2.9e-16"," 4.7e-16"," 7.0e-16"," 8.1e-16"," 1.1e-15"," 1.1e-15"," 1.1e-15"," 1.5e-15"," 1.5e-15"," 1.8e-15"," 1.9e-15"," 1.9e-15"," 2.1e-15"," 3.5e-15"," 3.9e-15"," 4.3e-15"," 4.5e-15"," 4.7e-15"," 5.3e-15"," 6.0e-15"," 7.0e-15"," 7.8e-15"," 8.7e-15"," 9.4e-15"," 1.0e-14"," 1.0e-14"," 1.2e-14"," 1.2e-14"," 1.3e-14"," 1.4e-14"," 1.4e-14"," 1.5e-14"," 1.9e-14"," 1.9e-14"," 1.9e-14"," 2.2e-14"," 2.6e-14"," 2.7e-14"," 3.3e-14"," 3.6e-14"," 3.7e-14"," 3.8e-14"," 3.9e-14"," 4.3e-14"," 4.4e-14"," 4.7e-14"," 6.6e-14"," 7.5e-14"," 7.7e-14"," 9.1e-14"," 9.6e-14"," 9.6e-14"," 9.8e-14"," 9.8e-14"," 1.0e-13"," 1.1e-13"," 1.3e-13"," 1.9e-13"," 1.9e-13"," 2.2e-13"," 2.5e-13"," 2.5e-13"," 3.0e-13"," 3.2e-13"," 3.3e-13"," 3.8e-13"," 3.8e-13"," 3.9e-13"," 5.2e-13"," 7.7e-13"," 8.8e-13"," 1.1e-12"," 1.3e-12"," 1.4e-12"," 1.4e-12"," 1.4e-12"," 1.6e-12"," 1.6e-12"," 1.7e-12"," 1.8e-12"," 1.8e-12"," 2.0e-12"," 3.2e-12"," 3.3e-12"," 3.6e-12"," 4.0e-12"," 4.1e-12"," 4.1e-12"," 4.7e-12"," 4.8e-12"," 5.0e-12"," 5.0e-12"," 5.1e-12"," 5.4e-12"," 9.3e-12"," 9.4e-12"," 9.5e-12"," 9.8e-12"," 1.1e-11"," 1.1e-11"," 1.1e-11"," 1.3e-11"," 1.3e-11"," 1.4e-11"," 1.4e-11"," 1.4e-11"," 1.5e-11"," 1.5e-11"," 1.9e-11"," 1.9e-11"," 1.9e-11"," 2.1e-11"," 2.2e-11"," 2.2e-11"," 2.2e-11"," 2.3e-11"," 2.6e-11"," 2.8e-11"," 2.9e-11"," 3.1e-11"," 3.1e-11"," 3.2e-11"," 3.4e-11"," 3.4e-11"," 3.8e-11"," 3.9e-11"," 4.1e-11"," 4.1e-11"," 4.3e-11"," 5.4e-11"," 5.4e-11"," 5.6e-11"," 6.1e-11"," 6.2e-11"," 6.2e-11"," 6.8e-11"," 7.4e-11"," 7.6e-11"," 7.8e-11"," 7.8e-11"," 7.9e-11"," 8.3e-11"," 8.7e-11"," 9.1e-11"," 1.0e-10"," 1.0e-10"," 1.1e-10"," 1.3e-10"," 1.3e-10"," 1.6e-10"," 1.7e-10"," 1.8e-10"," 1.9e-10"," 1.9e-10"," 2.4e-10"," 2.8e-10"," 3.3e-10"," 3.4e-10"," 3.4e-10"," 3.5e-10"," 3.6e-10"," 3.7e-10"," 3.8e-10"," 4.1e-10"," 4.7e-10"," 4.8e-10"," 5.2e-10"," 6.0e-10"," 6.3e-10"," 6.4e-10"," 6.5e-10"," 6.7e-10"," 9.0e-10"," 9.4e-10"," 9.4e-10"," 1.1e-09"," 1.1e-09"," 1.3e-09"," 1.4e-09"," 1.4e-09"," 1.4e-09"," 1.5e-09"," 1.5e-09"," 1.5e-09"," 1.6e-09"," 1.6e-09"," 1.7e-09"," 1.7e-09"," 1.8e-09"," 1.8e-09"," 1.9e-09"," 2.1e-09"," 2.1e-09"," 2.4e-09"," 3.2e-09"," 3.5e-09"," 3.8e-09"," 3.8e-09"," 4.0e-09"," 4.2e-09"," 4.5e-09"," 4.8e-09"," 5.1e-09"," 5.4e-09"," 5.4e-09"," 5.8e-09"," 6.0e-09"," 6.3e-09"," 6.9e-09"," 6.9e-09"," 7.3e-09"," 7.4e-09"," 7.6e-09"," 7.6e-09"," 7.7e-09"," 7.8e-09"," 8.0e-09"," 9.0e-09"," 9.2e-09"," 9.4e-09"," 9.8e-09"," 9.9e-09"," 1.0e-08"," 1.0e-08"," 1.0e-08"," 1.0e-08"," 1.1e-08"," 1.1e-08"," 1.2e-08"," 1.2e-08"," 1.3e-08"," 1.3e-08"," 1.5e-08"," 1.5e-08"," 1.6e-08"," 1.6e-08"," 1.8e-08"," 1.8e-08"," 2.0e-08"," 2.0e-08"," 2.0e-08"," 2.2e-08"," 2.3e-08"," 2.4e-08"," 2.5e-08"," 2.5e-08"," 2.6e-08"," 2.7e-08"," 3.6e-08"," 3.7e-08"," 3.7e-08"," 3.8e-08"," 3.9e-08"," 3.9e-08"," 4.2e-08"," 4.2e-08"," 4.4e-08"," 4.5e-08"," 4.9e-08"," 5.3e-08"," 5.5e-08"," 5.7e-08"," 6.3e-08"," 6.3e-08"," 6.6e-08"," 7.0e-08"," 8.2e-08"," 8.4e-08"," 8.7e-08"," 9.3e-08"," 1.1e-07"," 1.1e-07"," 1.1e-07"," 1.2e-07"," 1.3e-07"," 1.4e-07"," 1.5e-07"," 1.6e-07"," 1.6e-07"," 1.7e-07"," 1.8e-07"," 1.8e-07"," 1.9e-07"," 2.0e-07"," 2.1e-07"," 2.1e-07"," 2.1e-07"," 2.3e-07"," 2.3e-07"," 2.5e-07"," 2.5e-07"," 2.7e-07"," 3.0e-07"," 3.0e-07"," 3.2e-07"," 3.3e-07"," 3.5e-07"," 3.5e-07"," 3.5e-07"," 3.5e-07"," 3.6e-07"," 3.8e-07"," 3.9e-07"," 4.0e-07"," 4.2e-07"," 4.3e-07"," 4.8e-07"," 4.9e-07"," 5.0e-07"," 5.1e-07"," 5.4e-07"," 5.6e-07"," 5.8e-07"," 5.9e-07"," 6.0e-07"," 6.3e-07"," 6.6e-07"," 6.8e-07"," 7.1e-07"," 7.6e-07"," 8.0e-07"," 8.1e-07"," 8.1e-07"," 8.2e-07"," 8.3e-07"," 8.3e-07"," 8.4e-07"," 8.6e-07"," 8.8e-07"," 9.1e-07"," 9.3e-07"," 9.7e-07"," 1.0e-06"," 1.0e-06"," 1.0e-06"," 1.1e-06"," 1.1e-06"," 1.1e-06"," 1.1e-06"," 1.2e-06"," 1.2e-06"," 1.3e-06"," 1.3e-06"," 1.3e-06"," 1.4e-06"," 1.5e-06"," 1.5e-06"," 1.5e-06"," 1.6e-06"," 1.6e-06"," 1.7e-06"," 1.7e-06"," 1.8e-06"," 1.8e-06"," 1.8e-06"," 1.9e-06"," 1.9e-06"," 1.9e-06"," 1.9e-06"," 1.9e-06"," 2.0e-06"," 2.2e-06"," 2.3e-06"," 2.3e-06"," 2.4e-06"," 2.5e-06"," 2.6e-06"," 2.6e-06"," 2.7e-06"," 2.7e-06"," 2.7e-06"," 2.7e-06"," 2.8e-06"," 3.1e-06"," 3.1e-06"," 3.1e-06"," 3.2e-06"," 3.3e-06"," 3.7e-06"," 3.9e-06"," 4.0e-06"," 4.0e-06"," 4.2e-06"," 4.2e-06"," 4.4e-06"," 4.4e-06"," 4.5e-06"," 4.6e-06"," 4.7e-06"," 4.7e-06"," 4.7e-06"," 5.0e-06"," 5.1e-06"," 5.3e-06"," 5.3e-06"," 5.3e-06"," 5.4e-06"," 5.4e-06"," 5.7e-06"," 5.7e-06"," 6.0e-06"," 6.2e-06"," 6.2e-06"," 6.4e-06"," 6.5e-06"," 6.6e-06"," 6.7e-06"," 6.7e-06"," 7.0e-06"," 7.1e-06"," 7.1e-06"," 7.2e-06"," 7.2e-06"," 7.7e-06"," 8.1e-06"," 8.2e-06"," 8.2e-06"," 9.0e-06"," 9.0e-06"," 9.0e-06"," 9.3e-06"," 9.4e-06"," 9.4e-06"," 9.5e-06"," 9.6e-06"," 9.6e-06"," 9.7e-06"," 9.7e-06"," 9.8e-06"," 9.9e-06"," 1.0e-05"," 1.0e-05"," 1.0e-05"," 1.0e-05"," 1.0e-05"," 1.1e-05"," 1.1e-05"," 1.1e-05"," 1.1e-05"," 1.2e-05"," 1.2e-05"," 1.2e-05"," 1.3e-05"," 1.4e-05"," 1.4e-05"," 1.5e-05"," 1.5e-05"," 1.6e-05"," 1.6e-05"," 1.6e-05"," 1.7e-05"," 1.8e-05"," 1.9e-05"," 1.9e-05"," 2.0e-05"," 2.0e-05"," 2.0e-05"," 2.0e-05"," 2.1e-05"," 2.1e-05"," 2.3e-05"," 2.4e-05"," 2.4e-05"," 2.4e-05"," 2.6e-05"," 2.6e-05"," 2.7e-05"," 2.7e-05"," 2.9e-05"," 2.9e-05"," 3.0e-05"," 3.1e-05"," 3.1e-05"," 3.2e-05"," 3.2e-05"," 3.3e-05"," 3.3e-05"," 3.4e-05"," 3.4e-05"," 3.5e-05"," 3.6e-05"," 3.8e-05"," 3.8e-05"," 3.9e-05"," 4.0e-05"," 4.1e-05"," 4.1e-05"," 4.1e-05"," 4.1e-05"," 4.2e-05"," 4.2e-05"," 4.2e-05"," 4.3e-05"," 4.3e-05"," 4.5e-05"," 4.7e-05"," 4.7e-05"," 4.9e-05"," 5.0e-05"," 5.2e-05"," 5.3e-05"," 5.3e-05"," 5.3e-05"," 5.7e-05"," 5.7e-05"," 5.9e-05"," 6.0e-05"," 6.2e-05"," 6.6e-05"," 6.6e-05"," 6.6e-05"," 6.6e-05"," 6.7e-05"," 7.0e-05"," 7.0e-05"," 7.3e-05"," 7.4e-05"," 7.4e-05"," 7.9e-05"," 7.9e-05"," 7.9e-05"," 8.4e-05"," 9.0e-05"," 9.0e-05"," 9.0e-05"," 9.2e-05"," 9.4e-05"," 9.4e-05"," 9.5e-05"," 9.9e-05"," 1.0e-04"," 1.0e-04"," 1.1e-04"," 1.1e-04"," 1.2e-04"," 1.2e-04"," 1.2e-04"," 1.3e-04"," 1.3e-04"," 1.3e-04"," 1.3e-04"," 1.4e-04"," 1.4e-04"," 1.4e-04"," 1.5e-04"," 1.6e-04"," 1.6e-04"," 1.6e-04"," 1.6e-04"," 1.7e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.8e-04"," 1.9e-04"," 1.9e-04"," 1.9e-04"," 2.0e-04"," 2.0e-04"," 2.0e-04"," 2.1e-04"," 2.1e-04"," 2.1e-04"," 2.2e-04"," 2.3e-04"," 2.3e-04"," 2.3e-04"," 2.4e-04"," 2.4e-04"," 2.5e-04"," 2.5e-04"," 2.5e-04"," 2.5e-04"," 2.6e-04"," 2.7e-04"," 2.7e-04"," 2.7e-04"," 2.7e-04"," 2.7e-04"," 2.8e-04"," 3.0e-04"," 3.0e-04"," 3.0e-04"," 3.0e-04"," 3.1e-04"," 3.1e-04"," 3.2e-04"," 3.3e-04"," 3.3e-04"," 3.3e-04"," 3.3e-04"," 3.3e-04"," 3.3e-04"," 3.4e-04"," 3.4e-04"," 3.4e-04"," 3.4e-04"," 3.5e-04"," 3.5e-04"," 3.5e-04"," 3.5e-04"," 3.6e-04"," 3.7e-04"," 3.9e-04"," 3.9e-04"," 4.1e-04"," 4.1e-04"," 4.1e-04"," 4.2e-04"," 4.3e-04"," 4.3e-04"," 4.5e-04"," 4.5e-04"," 4.6e-04"," 4.6e-04"," 4.7e-04"," 4.7e-04"," 4.7e-04"," 4.8e-04"," 4.8e-04"," 4.9e-04"," 5.0e-04"," 5.1e-04"," 5.1e-04"," 5.1e-04"," 5.1e-04"," 5.1e-04"," 5.2e-04"," 5.2e-04"," 5.2e-04"," 5.2e-04"," 5.3e-04"," 5.4e-04"," 5.7e-04"," 5.7e-04"," 5.9e-04"," 6.0e-04"," 6.0e-04"," 6.1e-04"," 6.2e-04"," 6.3e-04"," 6.3e-04"," 6.3e-04"," 6.3e-04"," 6.4e-04"," 6.4e-04"," 6.5e-04"," 7.0e-04"," 7.1e-04"," 7.1e-04"," 7.2e-04"," 7.2e-04"," 7.3e-04"," 7.3e-04"," 7.5e-04"," 7.6e-04"," 7.8e-04"," 7.9e-04"," 8.0e-04"," 8.2e-04"," 8.4e-04"," 8.5e-04"," 8.5e-04"," 8.5e-04"," 8.7e-04"," 8.9e-04"," 9.0e-04"," 9.5e-04"," 9.7e-04"," 9.8e-04"," 1.0e-03"," 1.0e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.1e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.2e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.3e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.4e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.5e-03"," 1.6e-03"," 1.6e-03"," 1.7e-03"," 1.7e-03"," 1.8e-03"," 1.8e-03"," 1.8e-03"," 1.8e-03"," 1.8e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 1.9e-03"," 2.0e-03"," 2.0e-03"," 2.0e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.1e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.2e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.3e-03"," 2.4e-03"," 2.4e-03"," 2.4e-03"," 2.4e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.5e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.6e-03"," 2.7e-03"," 2.7e-03"," 2.9e-03"," 2.9e-03"," 3.0e-03"," 3.0e-03"," 3.1e-03"," 3.1e-03"," 3.2e-03"," 3.2e-03"," 3.2e-03"," 3.3e-03"," 3.3e-03"," 3.3e-03"," 3.6e-03"," 3.6e-03"," 3.6e-03"," 3.7e-03"," 3.7e-03"," 3.9e-03"," 3.9e-03"," 3.9e-03"," 4.0e-03"," 4.0e-03"," 4.0e-03"," 4.1e-03"," 4.1e-03"," 4.1e-03"," 4.1e-03"," 4.1e-03"," 4.1e-03"," 4.2e-03"," 4.2e-03"," 4.3e-03"," 4.5e-03"," 4.5e-03"," 4.6e-03"," 4.7e-03"," 4.8e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 4.9e-03"," 5.0e-03"," 5.0e-03"," 5.1e-03"," 5.1e-03"," 5.2e-03"," 5.2e-03"," 5.2e-03"," 5.2e-03"," 5.3e-03"," 5.3e-03"," 5.4e-03"," 5.5e-03"," 5.5e-03"," 5.6e-03"," 5.6e-03"," 5.7e-03"," 5.7e-03"," 5.8e-03"," 5.9e-03"," 6.1e-03"," 6.1e-03"," 6.1e-03"," 6.1e-03"," 6.1e-03"," 6.2e-03"," 6.3e-03"," 6.3e-03"," 6.4e-03"," 6.6e-03"," 6.6e-03"," 6.8e-03"," 6.8e-03"," 6.9e-03"," 6.9e-03"," 6.9e-03"," 6.9e-03"," 7.2e-03"," 7.4e-03"," 7.5e-03"," 7.5e-03"," 7.5e-03"," 7.6e-03"," 7.8e-03"," 7.8e-03"," 8.0e-03"," 8.2e-03"," 8.3e-03"," 8.3e-03"," 8.4e-03"," 8.4e-03"," 8.6e-03"," 8.6e-03"," 8.6e-03"," 8.6e-03"," 8.6e-03"," 8.7e-03"," 8.8e-03"," 8.8e-03"," 9.1e-03"," 9.3e-03"," 9.5e-03"," 9.6e-03"," 9.6e-03"," 9.6e-03"," 9.8e-03"," 9.9e-03"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.0e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.1e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.2e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.3e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.4e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.5e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.6e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.7e-02"," 1.8e-02"," 1.8e-02"," 1.8e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 1.9e-02"," 2.0e-02"," 2.0e-02"," 2.0e-02"," 2.0e-02"," 2.0e-02"," 2.0e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.1e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.2e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.3e-02"," 2.4e-02"," 2.4e-02"," 2.4e-02"," 2.5e-02"," 2.5e-02"," 2.5e-02"," 2.5e-02"," 2.6e-02"," 2.6e-02"," 2.6e-02"," 2.7e-02"," 2.7e-02"," 2.7e-02"," 2.7e-02"," 2.8e-02"," 2.8e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 2.9e-02"," 3.1e-02"," 3.1e-02"," 3.1e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.2e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.3e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.4e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.5e-02"," 3.6e-02"," 3.6e-02"," 3.6e-02"," 3.6e-02"," 3.7e-02"," 3.7e-02"," 3.7e-02"," 3.7e-02"," 3.7e-02"," 3.7e-02"," 3.8e-02"," 3.8e-02"," 3.9e-02"," 3.9e-02"," 3.9e-02"," 4.0e-02"," 4.0e-02"," 4.0e-02"," 4.0e-02"," 4.1e-02"," 4.2e-02"," 4.2e-02"," 4.2e-02"," 4.3e-02"," 4.3e-02"," 4.4e-02"," 4.4e-02"," 4.4e-02"," 4.5e-02"," 4.6e-02"," 4.6e-02"," 4.7e-02"," 4.7e-02"," 4.8e-02"," 4.9e-02"," 4.9e-02"," 5.0e-02"," 5.0e-02"],[5420,12241,333,560,14833,848,2110,2134,2134,2133,3874,1453,423,432,3957,2276,2355,17809,2086,999,804,4215,715,2048,2176,2177,502,421,5766,2519,8070,1932,1850,3364,4882,1936,1919,7868,448,2665,6446,2422,1774,3125,1283,1349,2370,503,1137,2803,2536,2781,1046,6424,6425,5487,3031,2292,1577,1503,6436,5687,778,9845,1206,4315,2319,16739,946,1566,2212,319,6224,3050,174,3936,218,2345,2629,203,448,5500,1339,5746,6625,6626,6626,15936,497,529,3719,3719,1493,2199,906,2463,1913,2465,1922,1009,1406,15617,1012,2559,110,2085,181,483,1604,755,1001,2196,418,1525,1010,393,757,757,2105,374,1882,2725,1359,3107,754,646,1461,392,18309,866,871,3768,4707,226,11193,509,4903,698,449,1691,124,1485,195,5754,691,4856,1889,1152,630,510,741,410,3895,1875,1766,184,1386,1388,7399,402,471,542,1655,1655,1031,352,3851,3271,247,2947,2094,356,14570,894,6317,769,3351,2103,736,150,325,1034,1778,707,7428,970,1587,6066,205,7501,1237,1708,1527,3336,264,826,320,4742,323,3083,80,6004,156,79,79,904,1195,4768,98,1833,269,546,2233,718,689,2702,5812,121,3019,722,1347,196,954,1245,3776,695,313,129,771,1167,957,4184,3465,1176,6508,419,203,867,2457,1108,519,7399,845,2654,71,71,268,192,379,4313,82,690,4160,5654,1089,757,298,352,444,1556,267,642,344,1865,457,12276,8885,2336,13001,1242,70,1489,661,1918,2074,327,99,1171,179,328,158,637,268,1505,326,203,795,161,6451,638,895,895,424,487,91,502,5876,678,277,82,451,3010,1028,442,481,999,196,887,446,348,301,315,623,531,404,1642,4587,6237,2430,952,893,983,80,1112,2391,204,900,635,1448,174,695,104,250,199,1170,232,341,245,628,816,829,215,194,1592,5757,586,351,167,1419,10084,188,65,93,308,307,173,8,310,157,562,429,18,25,35,58,201,530,619,97,242,558,1366,4834,202,869,2322,1436,167,4336,122,165,72,1109,997,550,755,384,996,142,307,336,5957,1097,177,114,59,14,284,96,106,175,357,992,1135,896,194,3668,179,196,141,438,194,54,1285,225,147,641,7354,130,538,1724,440,893,360,500,500,62,406,406,1305,2425,613,41,105,174,7313,104,77,340,148,485,454,128,683,1557,1059,211,67,417,564,418,735,77,41,553,475,171,7,913,877,110,567,109,339,495,833,1316,491,5774,160,362,203,296,6787,610,1100,161,2125,612,568,196,165,169,523,109,109,127,72,783,203,367,234,943,374,626,697,94,527,1350,132,93,196,5700,653,465,206,5861,207,171,107,161,302,547,22,154,256,301,232,51,51,293,343,141,1970,999,674,50,100,686,271,675,429,497,497,63,54,54,296,1180,1084,379,73,284,778,294,265,147,163,163,96,96,393,770,314,561,434,394,1010,1134,563,3513,21,71,155,313,313,398,1715,163,122,123,320,1037,2506,166,198,77,109,301,83,1940,125,2371,31,181,288,141,336,123,190,68,93,569,5449,313,154,199,204,202,541,541,331,1711,44,270,508,930,243,404,296,71,951,1243,90,484,788,8,5,839,575,56,362,1104,96,1630,82,489,98,74,592,390,490,4765,227,45,1110,87,414,119,88,67,125,2666,114,82,664,835,349,105,105,609,151,94,2169,76,1513,53,72,705,323,83,1356,77,418,340,170,12,10,230,477,582,126,304,276,199,191,1958,264,71,1351,158,438,78,901,109,799,975,234,253,538,86,96,112,130,238,653,9,224,27,561,289,1683,362,225,56,161,226,309,19,364,62,762,5452,191,1260,1260,391,8,90,383,178,742,742,72,33,62,431,809,1312,793,195,317,153,227,484,16,11,347,573,113,80,200,431,69,173,63,5,2788,214,48,6,226,413,129,1130,134,101,56,1658,171,44,300,7,103,13,13,22,140,60,55,89,23,57,621,248,33,267,358,77,6,6,6,6,32,75,214,69,1166,91,137,444,86,1796,3570,838,49,22,685,84,1348,670,37,1659,57,36,3,30,509,421,36,237,15,130,190,73,240,366,320,1599,15,15,159,39,39,149,44,276,193,67,130,107,154,154,34,104,134,229,189,959,1624,37,661,879,93,18,191,9,259,174,31,31,157,316,326,168,29,88,63,63,94,6,490,54,126,11,173,425,921,657,6,56,308,21,21,10,139,325,37,107,11,192,8,343,4,94,175,11,67,38,253,73,31,42,3361,153,6,6,528,119,321,44,137,64,256,79,133,97,37,988,34,25,5,203,162,115,388,740,145,39,41,13,5,147,91,28,212,17,11,41,826,558,58,1787,62,55,1099,182,52,76,12,22,58,831,966,287,58,27,142,69,178,4,42,42,655,409,72,373,12865,172,97,51,28,289,175,166,141,126,18,7,7,104,175,11,376,64,161,328,328,940,73,665,84,235,7,640,190,140,114,51,73,3,264,65,193,50,1023,281,53,22,51,350,849,68,60,1859,33,167,145,93,420,131,20,20,96,195,60,41,21,339,158,223,128,31,274,90,175,31,23,23,9,15,85,11,25,74,268,197,18,31,96,34,250,54,200,4770,15,10,10,12,200,105,290,1434,44,132,497,65,11,15,127,80,401,110,169,29,98,3926,25,66,57,44,265,112,63,57,13,147,58,88,130,39,854,39,163,205,7,907,122,210,4145,3,32,46,27,45,53,159,161,21,261,4085,264,39,39,39,45,178,45,116,8,109,3,838,83,9,44,1268,174,20,306,6,561,162,350,1964,606,118,149,1067,288,40,294,197,413,413,20,1054,312,86,41,3395,15,118,108,79,3,498,7,14,19395,78,48,83,255,9,59,59,5,5,5,5,30,365,90,5,101,10,574,7,41,149,56,1579,19,35,90,29,46,318,224,89,172,235,27,35,1999,8,52,2811,4,70,19,55,305,110,95,42,43,36,460,206,76,109,523],[754,1071,140,152,1133,169,311,313,313,312,462,243,104,105,434,292,310,1166,263,157,139,433,129,277,248,248,96,97,537,268,645,252,244,116,470,249,246,616,90,306,463,283,232,334,148,189,242,92,164,304,251,295,150,547,547,411,326,239,211,204,506,505,116,679,172,112,240,1088,132,206,231,75,507,303,53,377,51,248,242,57,80,469,165,438,533,533,533,1042,87,96,354,354,161,240,133,237,219,237,219,139,152,1014,113,264,39,191,47,78,190,115,106,235,77,169,106,68,81,81,225,73,203,250,165,102,80,102,153,56,1181,69,121,175,389,51,741,80,380,75,71,163,23,161,48,435,88,414,204,143,97,79,100,70,345,200,193,26,139,139,519,42,62,71,160,160,72,59,340,283,37,277,188,54,953,89,462,104,304,179,77,40,32,110,191,75,217,105,169,425,46,218,131,154,169,301,37,103,31,373,31,241,28,401,30,28,28,117,136,368,24,166,48,63,208,81,68,237,388,26,137,81,136,40,120,136,306,96,58,25,104,86,111,329,229,131,456,51,40,80,206,129,74,532,108,233,25,25,46,42,57,332,15,95,325,375,128,81,52,61,50,157,45,88,60,182,61,418,582,226,809,119,20,132,72,185,196,35,27,95,38,35,34,61,31,131,55,38,102,34,447,86,105,105,58,62,17,63,214,90,51,19,65,267,110,58,61,116,41,96,68,54,53,47,67,71,61,165,138,424,222,103,93,102,24,104,221,25,107,73,146,35,90,24,28,29,122,35,49,36,78,95,95,37,37,155,394,79,24,36,142,683,34,19,24,50,35,37,6,48,35,64,58,11,13,15,15,35,62,70,23,35,70,132,140,40,91,215,145,32,94,30,35,15,121,103,73,77,52,110,29,47,50,407,115,36,27,10,6,48,26,27,28,40,103,41,86,27,285,27,27,25,59,28,13,116,24,32,73,233,24,71,149,62,96,54,57,58,20,40,40,118,180,71,15,11,24,311,17,22,38,28,55,54,23,76,151,113,30,15,50,71,50,83,22,16,17,64,32,5,84,89,23,45,24,47,55,93,132,46,361,23,39,37,41,291,67,101,23,182,67,71,25,23,9,59,16,16,28,19,73,34,50,30,84,51,70,81,21,64,132,12,22,31,355,78,57,26,363,21,23,24,12,41,60,7,23,40,46,39,16,16,40,20,27,167,100,79,11,24,79,32,78,57,60,60,17,14,15,40,80,109,39,12,41,86,40,36,22,26,26,15,15,47,82,46,60,50,54,96,102,60,257,9,8,29,40,40,51,156,13,21,22,47,99,104,26,30,13,18,45,19,108,22,201,7,28,41,21,42,26,33,17,18,68,110,40,25,31,30,32,65,65,26,154,15,38,54,82,12,42,43,19,88,99,19,49,73,5,5,89,65,8,42,65,17,139,19,58,20,19,69,49,54,299,36,14,98,13,54,22,21,16,22,143,22,17,64,78,45,23,23,62,26,19,184,17,138,6,18,48,42,20,113,17,47,17,10,6,7,25,55,67,16,43,27,27,32,156,28,16,112,20,43,16,30,18,53,93,34,34,33,20,13,17,21,29,70,5,32,10,62,41,102,30,31,14,28,30,15,7,21,12,75,108,25,85,85,42,4,20,41,25,8,8,15,11,8,46,78,121,76,10,39,24,21,45,6,7,46,61,16,15,28,45,27,18,14,3,193,33,9,5,16,43,21,58,25,10,7,58,21,12,35,6,19,4,4,4,11,15,9,19,5,7,20,36,11,25,16,13,5,5,5,5,6,9,33,17,100,18,25,53,18,125,128,8,8,7,50,8,114,49,12,144,11,10,2,10,58,16,12,14,5,19,30,17,35,35,37,139,6,6,25,10,10,24,5,36,13,10,13,13,5,5,10,19,22,24,15,61,129,10,15,81,8,7,30,6,34,22,5,10,18,35,37,14,5,18,14,14,17,4,17,14,22,5,15,45,90,69,4,14,40,9,7,4,24,25,11,15,4,42,3,38,4,19,28,2,16,6,17,11,10,11,120,10,5,5,31,20,35,8,21,12,17,17,7,13,4,38,6,5,3,26,18,16,47,62,20,9,11,4,4,22,4,10,27,4,6,11,81,57,13,149,15,14,92,14,6,13,4,8,14,28,37,35,14,7,24,7,12,3,11,11,53,46,14,40,679,9,16,7,8,35,25,23,17,20,6,2,2,17,9,5,40,15,22,31,31,36,16,60,7,16,3,41,14,20,17,13,16,3,35,15,10,9,95,34,3,8,6,37,82,9,14,43,9,26,23,18,23,19,5,5,10,20,11,7,8,41,18,11,10,10,17,7,7,7,8,8,3,6,11,5,8,5,22,19,5,9,10,7,12,11,14,284,6,3,4,6,16,19,26,37,10,12,37,9,4,4,21,11,39,8,22,8,9,79,5,14,6,10,32,17,11,5,5,18,13,17,19,7,71,10,25,29,4,85,8,11,82,3,6,12,9,11,4,15,15,7,34,81,34,11,11,11,8,17,11,20,5,10,3,63,13,5,11,93,23,6,34,3,48,13,13,47,52,8,13,96,35,11,34,24,5,5,7,84,26,14,5,62,6,14,19,14,3,44,3,6,259,14,12,3,32,3,9,9,4,4,4,4,5,35,17,2,9,5,40,4,11,22,12,62,5,10,14,8,8,32,27,16,25,31,4,9,47,5,12,170,3,14,3,11,37,17,15,11,10,10,5,28,9,16,45]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap" class="section level4 tabset tabset-fade">
<h4 class="tabset tabset-fade">Treemap</h4>
<div id="bp" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/BP-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/MF-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/CC-cond1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/scatter-cond1-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="sample1-1" class="section level3 tabset tabset-fade">
<h3 class="tabset tabset-fade"><strong>5.2. sample1</strong></h3>
<div id="gostplot-1" class="section level4">
<h4>Gostplot</h4>
<p>UP regulated proteins in sample1 were used to generate these gene
ontologies. In this case we also use the specific unique genes
identify.</p>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant &amp; get(paste0(test, &quot;_ratio&quot;)) &lt; 0) %&gt;% pull(name)

if(!params$Imputation){
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  filter(!!sym(paste0(&quot;unique_&quot;, cond_2)) == &quot;+&quot;) %&gt;% 
  pull(name) %&gt;% unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  print(&quot;NOT enough data.&quot;)
  GO &lt;- NULL
}else{
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  if(is.null(GO)){
    print(&quot;NOT enough data.&quot;)
  }else{
    gostplot(GO, capped = TRUE, interactive = TRUE)
  }
}</code></pre>
<div id="htmlwidget-ef2b5ccbfe7532abed56" style="width:960px;height:768px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-ef2b5ccbfe7532abed56">{"x":{"data":[{"x":[193.875683520811,193.867891517338,123.642460209128,148.810631429712,83.4513062909313,170.499673099216,151.604064675093,115.530984592834,177.97610043239,131.699391801104,79.7578966443193,136.503161942742,172.034697783567,112.753135354401,112.74144734919,112.698591330084,124.667108665941],"y":[4.60278683687009,3.42675047362512,2.76028556631907,1.89238737075329,1.88057853847729,1.73082049036823,1.6077421361967,1.6077421361967,1.598612104058,1.54222529770554,1.44124404369501,1.40050937617544,1.3601019119277,1.35561023971613,1.35561023971613,1.35561023971613,1.3082643506534],"text":["GO:2001295 (2) <br> malonyl-CoA biosynthetic process <br> 2.496e-05","GO:2001293 (6) <br> malonyl-CoA metabolic process <br> 3.743e-04","GO:0043603 (1167) <br> cellular amide metabolic process <br> 1.737e-03","GO:0070488 (2) <br> neutrophil aggregation <br> 1.281e-02","GO:0006084 (33) <br> acetyl-CoA metabolic process <br> 1.317e-02","GO:1901137 (612) <br> carbohydrate derivative biosynthetic process <br> 1.859e-02","GO:0071616 (45) <br> acyl-CoA biosynthetic process <br> 2.468e-02","GO:0035384 (45) <br> thioester biosynthetic process <br> 2.468e-02","GO:1903311 (326) <br> regulation of mRNA metabolic process <br> 2.520e-02","GO:0048024 (105) <br> regulation of mRNA splicing, via spliceosome <br> 2.869e-02","GO:0002523 (20) <br> leukocyte migration involved in inflammatory response <br> 3.620e-02","GO:0051289 (57) <br> protein homotetramerization <br> 3.976e-02","GO:1901564 (6446) <br> organonitrogen compound metabolic process <br> 4.364e-02","GO:0034033 (60) <br> purine nucleoside bisphosphate biosynthetic process <br> 4.410e-02","GO:0034030 (60) <br> ribonucleoside bisphosphate biosynthetic process <br> 4.410e-02","GO:0033866 (60) <br> nucleoside bisphosphate biosynthetic process <br> 4.410e-02","GO:0044255 (1009) <br> cellular lipid metabolic process <br> 4.917e-02"],"key":["GO:2001295","GO:2001293","GO:0043603","GO:0070488","GO:0006084","GO:1901137","GO:0071616","GO:0035384","GO:1903311","GO:0048024","GO:0002523","GO:0051289","GO:1901564","GO:0034033","GO:0034030","GO:0033866","GO:0044255"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(255,153,0,1)","opacity":0.8,"size":[3.77952755905512,10.414746449235,19.7556054091027,3.77952755905512,14.3787130795644,18.9244516335556,14.9496653615287,14.9496653615287,18.0668830168052,16.3782122472689,13.3854897407262,15.3659461499023,21.771831432208,15.4543140306055,15.4543140306055,15.4543140306055,19.5720950699891],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(255,153,0,1)"}},"hoveron":"points","set":"SharedData86d1ea8a","name":"GO:BP","legendgroup":"GO:BP","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[55.7284052393749,64.4843931994853,59.5277324583814,62.2009001222157,64.5740183543952,69.7488968639753,64.4804964536196,62.1892098846188,55.3971818407949,62.1970033763501,62.1814163928875,57.8521317361529,62.9802492953453,59.5511129335753,57.1585109720676,56.0401449086267,55.1127193926026,54.9802300331706,57.8988926865407,64.6597467634394,54.5282075127554],"y":[5.9219875864274,5.24478726045691,5.24281275453984,5.24281275453984,4.15228537598879,4.07308578275031,4.06966720403886,4.06966720403886,3.83814200587954,3.49315631840096,3.49140540518972,2.80711201393597,2.72761577017114,2.06101372594057,1.78847559946377,1.52437734652424,1.49033903542906,1.47952617842411,1.4613827631128,1.3652995431001,1.35706004948539],"text":["GO:0005829 (5420) <br> cytosol <br> 1.197e-06","GO:0070013 (6625) <br> intracellular organelle lumen <br> 5.691e-06","GO:0031974 (6626) <br> membrane-enclosed lumen <br> 5.717e-06","GO:0043233 (6626) <br> organelle lumen <br> 5.717e-06","GO:0070062 (2110) <br> extracellular exosome <br> 7.042e-05","GO:1903561 (2133) <br> extracellular vesicle <br> 8.451e-05","GO:0065010 (2134) <br> extracellular membrane-bounded organelle <br> 8.518e-05","GO:0043230 (2134) <br> extracellular organelle <br> 8.518e-05","GO:0005737 (12241) <br> cytoplasm <br> 1.452e-04","GO:0043232 (6424) <br> intracellular non-membrane-bounded organelle <br> 3.213e-04","GO:0043228 (6425) <br> non-membrane-bounded organelle <br> 3.225e-04","GO:0022626 (104) <br> cytosolic ribosome <br> 1.559e-03","GO:0044391 (189) <br> ribosomal subunit <br> 1.872e-03","GO:0031981 (5500) <br> nuclear lumen <br> 8.689e-03","GO:0015629 (502) <br> actin cytoskeleton <br> 1.628e-02","GO:0005925 (423) <br> focal adhesion <br> 2.990e-02","GO:0005654 (4215) <br> nucleoplasm <br> 3.233e-02","GO:0005615 (3364) <br> extracellular space <br> 3.315e-02","GO:0030055 (432) <br> cell-substrate junction <br> 3.456e-02","GO:0070161 (848) <br> anchoring junction <br> 4.312e-02","GO:0001533 (58) <br> cornified envelope <br> 4.395e-02"],"key":["GO:0005829","GO:0070013","GO:0031974","GO:0043233","GO:0070062","GO:1903561","GO:0065010","GO:0043230","GO:0005737","GO:0043232","GO:0043228","GO:0022626","GO:0044391","GO:0031981","GO:0015629","GO:0005925","GO:0005654","GO:0005615","GO:0030055","GO:0070161","GO:0001533"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(16,150,24,1)","opacity":0.8,"size":[21.5777172766428,21.8023091869008,21.8024769875207,21.8024769875207,20.4818967806045,20.4948978190041,20.495459668663,20.495459668663,22.4724113318876,21.7680236702223,21.7681970505336,16.3629836313714,17.2807127312487,21.5942051342832,18.6600085813673,18.4276289321127,21.2923520950053,21.0323972011483,18.4563996606908,19.3499769913994,15.3959838971932],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(16,150,24,1)"}},"hoveron":"points","set":"SharedData86d1ea8a","name":"GO:CC","legendgroup":"GO:CC","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[16.9808508866633,12.3326961122057,15.7301738685569,4.55590069743852,23.3628102500845,9.31704498443527,28.3694145125974,21.7186180331347,16.9418889857877,26.7057413452098],"y":[9.3817813622263,7.2905034254808,6.81405416434321,4.59524101210622,4.29183338534094,3.84237655084453,2.81700072996479,2.32264965654631,1.67229126039032,1.5853478802512],"text":["GO:0016885 (7) <br> ligase activity, forming carbon-carbon bonds <br> 4.152e-10","GO:0009374 (5) <br> biotin binding <br> 5.123e-08","GO:0016421 (6) <br> CoA carboxylase activity <br> 1.534e-07","GO:0003989 (3) <br> acetyl-CoA carboxylase activity <br> 2.540e-05","GO:0033293 (80) <br> monocarboxylic acid binding <br> 5.107e-05","GO:0005515 (14833) <br> protein binding <br> 1.438e-04","GO:0043177 (142) <br> organic acid binding <br> 1.524e-03","GO:0031406 (173) <br> carboxylic acid binding <br> 4.757e-03","GO:0016874 (178) <br> ligase activity <br> 2.127e-02","GO:0035662 (4) <br> Toll-like receptor 4 binding <br> 2.598e-02"],"key":["GO:0016885","GO:0009374","GO:0016421","GO:0003989","GO:0033293","GO:0005515","GO:0043177","GO:0031406","GO:0016874","GO:0035662"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(220,57,18,1)","opacity":0.8,"size":[10.8649782781033,9.83920734309063,10.414746449235,7.81049946980777,15.9380348599676,22.6771653543307,16.8494914444983,17.1487912293309,17.1914255068478,9.0499540290216],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(220,57,18,1)"}},"hoveron":"points","set":"SharedData86d1ea8a","name":"GO:MF","legendgroup":"GO:MF","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[197.884597028376,198.274891475068,198.235862030399,199.258433480733],"y":[2.90858534149138,2.42632287615782,2.07941202746134,1.36774835813696],"text":["KEGG:00061 (18) <br> Fatty acid biosynthesis <br> 1.234e-03","KEGG:00640 (31) <br> Propanoate metabolism <br> 3.747e-03","KEGG:00620 (46) <br> Pyruvate metabolism <br> 8.329e-03","KEGG:04922 (104) <br> Glucagon signaling pathway <br> 4.288e-02"],"key":["KEGG:00061","KEGG:00640","KEGG:00620","KEGG:04922"],"type":"scatter","mode":"markers","marker":{"autocolorscale":false,"color":"rgba(221,68,119,1)","opacity":0.8,"size":[13.1631441028617,14.2598554887729,14.9890221084741,16.3629836313714],"symbol":"circle","line":{"width":1.88976377952756,"color":"rgba(221,68,119,1)"}},"hoveron":"points","set":"SharedData86d1ea8a","name":"KEGG","legendgroup":"KEGG","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","_isNestedKey":false,"frame":null},{"x":[2,50.0088542588985],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(220,57,18,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[53.9047281742518,71.3153887019656],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(16,150,24,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[75.2112626173189,193.957499557287],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(255,153,0,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[197.85337347264,200.003895873915],"y":[-1,-1],"text":"","type":"scatter","mode":"lines","line":{"width":11.3385826771654,"color":"rgba(221,68,119,1)","dash":"solid"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[180],"y":[16.2],"text":"values above this threshold are capped","hovertext":"","textfont":{"size":7.55905511811024,"color":"rgba(190,190,190,1)"},"type":"scatter","mode":"text","hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0,210],"y":[16,16],"text":"","type":"scatter","mode":"lines","line":{"width":0.755905511811024,"color":"rgba(190,190,190,1)","dash":"dash"},"hoveron":"points","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":29.2835201328352,"r":6.6417600664176,"b":51.8931100989696,"l":61.0377750103778},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0,210],"tickmode":"array","ticktext":["GO:MF","GO:CC","GO:BP","KEGG"],"tickvals":[26.0044271294493,62.6100584381087,134.584381087303,198.928634673278],"categoryorder":"array","categoryarray":["GO:MF","GO:CC","GO:BP","KEGG"],"nticks":null,"ticks":"","tickcolor":null,"ticklen":3.65296803652968,"tickwidth":0,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":10.6268161062682},"tickangle":-45,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.132835201328352,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[-1,18.5],"tickmode":"array","ticktext":["0","2","4","6","8","10","12","14",">16"],"tickvals":[0,2,4,6,8,10,12,14,16],"categoryorder":"array","categoryarray":["0","2","4","6","8","10","12","14",">16"],"nticks":null,"ticks":"outside","tickcolor":"rgba(190,190,190,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":true,"linecolor":"rgba(190,190,190,1)","linewidth":0.66417600664176,"showgrid":false,"gridcolor":null,"gridwidth":0,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"annotations":[{"text":"-log10(p-adj)","x":0,"y":0.5,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(0,0,0,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-90,"xanchor":"right","yanchor":"center","annotationType":"axis","xshift":-37.3931091739311},{"text":"query_1","x":0.5,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(169,169,169,1)","family":"","size":13.2835201328352},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"shapes":[{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":null,"line":{"color":null,"width":0,"linetype":[]},"yref":"paper","xref":"paper","x0":0,"x1":1,"y0":0,"y1":24.9730178497302,"yanchor":1,"ysizemode":"pixel"}],"showlegend":false,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895}},"hovermode":"closest","barmode":"relative","dragmode":"zoom"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"38b010f92d28":{"x":{},"y":{},"text":{},"colour":{},"size":{},"alpha":{},"type":"scatter"},"38b06b3fc2f":{"x":{},"xend":{},"y":{},"yend":{}},"38b010a44de5":{"x":{},"xend":{},"y":{},"yend":{}},"38b016029c4":{"x":{},"xend":{},"y":{},"yend":{}},"38b03a42175b":{"x":{},"xend":{},"y":{},"yend":{}},"38b070625125":{"x":{},"y":{}},"38b06bc7eec":{"yintercept":{}}},"cur_data":"38b010f92d28","visdat":{"38b010f92d28":["function (y) ","x"],"38b06b3fc2f":["function (y) ","x"],"38b010a44de5":["function (y) ","x"],"38b016029c4":["function (y) ","x"],"38b03a42175b":["function (y) ","x"],"38b070625125":["function (y) ","x"],"38b06bc7eec":["function (y) ","x"]},"highlight":{"on":"plotly_click","off":"plotly_doubleclick","persistent":false,"dynamic":false,"color":null,"selectize":false,"defaultValues":null,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0,"ctGroups":["SharedData86d1ea8a"]},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="table-results-1" class="section level4">
<h4>Table results</h4>
<p>The results can also be visualized with a table:</p>
<pre class="r"><code>if(is.null(GO)){
  print(&quot;NOT enough data.&quot;)
}else{
  GO$result %&gt;% 
    as.data.frame() %&gt;% 
    select(starts_with(c(&quot;source&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;term_size&quot;, &quot;intersection_size&quot;))) %&gt;% 
    arrange(p_value) %&gt;%  
    mutate(p_value = format(p_value, scientific = T, digits = 2)) %&gt;% 
    DT::datatable(options = list(
    columnDefs = list(list(className = &#39;dt-center&#39;, targets = 1:6)),
    pageLength = 10)) %&gt;%
    formatStyle(&#39;source&#39;, backgroundColor = styleEqual(c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), c(&#39;orange&#39;, &#39;tomato&#39;, &#39;lightgreen&#39;, &#39;violet&#39;)))
}</code></pre>
<div id="htmlwidget-c763ca1c2889feb598bc" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-c763ca1c2889feb598bc">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","21","22","23","24","25","26","27","28","29","30","31","32","33","34","35","36","37","38","39","40","41","42","43","44","45","46","47","48","49","50","51","52"],["GO:MF","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:MF","GO:MF","GO:CC","GO:CC","GO:CC","GO:CC","GO:MF","GO:CC","GO:CC","GO:CC","GO:BP","KEGG","GO:MF","GO:CC","GO:BP","GO:CC","KEGG","GO:MF","KEGG","GO:CC","GO:BP","GO:BP","GO:CC","GO:BP","GO:MF","GO:BP","GO:BP","GO:BP","GO:MF","GO:BP","GO:CC","GO:CC","GO:CC","GO:CC","GO:BP","GO:BP","KEGG","GO:CC","GO:BP","GO:CC","GO:BP","GO:BP","GO:BP","GO:BP"],[3846,2653,3525,469,2716,1444,2130,30459,657,5484,2739,4067,2715,2127,1879,384,2129,2125,30457,9,6769,1014,12432,2330,109,5062,99,1450,18892,2116,836,24459,3836,19609,10350,26378,6342,14500,549,311,277,1026,1168,15733,361,2761,24853,161,9637,9634,9623,12695],["ligase activity, forming carbon-carbon bonds","biotin binding","CoA carboxylase activity","cytosol","intracellular organelle lumen","membrane-enclosed lumen","organelle lumen","malonyl-CoA biosynthetic process","acetyl-CoA carboxylase activity","monocarboxylic acid binding","extracellular exosome","extracellular vesicle","extracellular membrane-bounded organelle","extracellular organelle","protein binding","cytoplasm","intracellular non-membrane-bounded organelle","non-membrane-bounded organelle","malonyl-CoA metabolic process","Fatty acid biosynthesis","organic acid binding","cytosolic ribosome","cellular amide metabolic process","ribosomal subunit","Propanoate metabolism","carboxylic acid binding","Pyruvate metabolism","nuclear lumen","neutrophil aggregation","acetyl-CoA metabolic process","actin cytoskeleton","carbohydrate derivative biosynthetic process","ligase activity","acyl-CoA biosynthetic process","thioester biosynthetic process","regulation of mRNA metabolic process","Toll-like receptor 4 binding","regulation of mRNA splicing, via spliceosome","focal adhesion","nucleoplasm","extracellular space","cell-substrate junction","leukocyte migration involved in inflammatory response","protein homotetramerization","Glucagon signaling pathway","anchoring junction","organonitrogen compound metabolic process","cornified envelope","purine nucleoside bisphosphate biosynthetic process","ribonucleoside bisphosphate biosynthetic process","nucleoside bisphosphate biosynthetic process","cellular lipid metabolic process"],["4.2e-10","5.1e-08","1.5e-07","1.2e-06","5.7e-06","5.7e-06","5.7e-06","2.5e-05","2.5e-05","5.1e-05","7.0e-05","8.5e-05","8.5e-05","8.5e-05","1.4e-04","1.5e-04","3.2e-04","3.2e-04","3.7e-04","1.2e-03","1.5e-03","1.6e-03","1.7e-03","1.9e-03","3.7e-03","4.8e-03","8.3e-03","8.7e-03","1.3e-02","1.3e-02","1.6e-02","1.9e-02","2.1e-02","2.5e-02","2.5e-02","2.5e-02","2.6e-02","2.9e-02","3.0e-02","3.2e-02","3.3e-02","3.5e-02","3.6e-02","4.0e-02","4.3e-02","4.3e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.4e-02","4.9e-02"],[7,5,6,5420,6625,6626,6626,2,3,80,2110,2133,2134,2134,14833,12241,6424,6425,6,18,142,104,1167,189,31,173,46,5500,2,33,502,612,178,45,45,326,4,105,423,4215,3364,432,20,57,104,848,6446,58,60,60,60,1009],[5,4,4,43,38,38,38,2,2,6,21,21,21,21,66,57,35,35,2,2,6,5,14,6,2,6,2,29,2,2,9,4,5,2,2,8,2,5,8,29,22,8,3,2,2,11,34,3,2,2,2,5]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>source<\/th>\n      <th>source_order<\/th>\n      <th>term_name<\/th>\n      <th>p_value<\/th>\n      <th>term_size<\/th>\n      <th>intersection_size<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-center","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0}],"pageLength":10,"order":[],"autoWidth":false,"orderClasses":false,"rowCallback":"function(row, data, displayNum, displayIndex, dataIndex) {\nvar value=data[1]; $(this.api().cell(row, 1).node()).css({'background-color':value == \"GO:BP\" ? \"orange\" : value == \"GO:MF\" ? \"tomato\" : value == \"GO:CC\" ? \"lightgreen\" : value == \"KEGG\" ? \"violet\" : null});\n}"}},"evals":["options.rowCallback"],"jsHooks":[]}</script>
</div>
<div id="treemap-1" class="section level4 tabset tabset-fade">
<h4 class="tabset tabset-fade">Treemap</h4>
<div id="bp-1" class="section level5">
<h5>BP</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:BP&quot;) &gt;= 5 &amp; !is.null(GO)){
  simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:BP&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;BP&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
BP &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  BP &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/BP-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mf-1" class="section level5">
<h5>MF</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:MF&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:MF&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;MF&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
MF &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  MF &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/MF-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="cc-1" class="section level5">
<h5>CC</h5>
<pre class="r"><code>if(sum(GO$result$source == &quot;GO:CC&quot;) &gt;= 5 &amp; !is.null(GO)){
simMatrix &lt;- GO$result %&gt;% 
  as.data.frame() %&gt;%
  filter(source == &quot;GO:CC&quot;) %&gt;% 
  pull(term_id) %&gt;% 
  calculateSimMatrix(orgdb=&quot;org.Hs.eg.db&quot;, ont=&quot;CC&quot;, method=&quot;Rel&quot;)

scores &lt;- setNames(-log10(GO$result$p_value), GO$result$term_id)

reducedTerms &lt;- reduceSimMatrix(simMatrix,
                                scores,
                                threshold=0.7,
                                orgdb=&quot;org.Hs.eg.db&quot;)

treemapPlot(reducedTerms)
CC &lt;- T
}else{
  print(&quot;NOT enough data.&quot;)
  CC &lt;- F
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/CC-cond2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="scatterplot-1" class="section level4">
<h4>ScatterPlot</h4>
<pre class="r"><code>if(BP | MF | CC){
scatterPlot(simMatrix, reducedTerms, size = &quot;score&quot;, labelSize = 5)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
<p><img src="figure/sample2_vs_sample1.Rmd/scatter-cond2-1.png" width="1440" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><strong>6. Network analysis</strong></h2>
<p>Overview from <strong>STRING Database</strong>: In STRING, each
protein-protein interaction is annotated with one or more ‘scores’.
Importantly, these scores do <strong>not</strong> indicate the strength
or the specificity of the interaction. Instead, they are indicators of
<strong>confidence</strong>, i.e. how likely STRING judges an
interaction to be true, given the available evidence. All scores rank
from 0 to 1, with 1 being the highest possible confidence. A score of
0.5 would indicate that roughly every second interaction might be
erroneous (i.e., a false positive).</p>
<pre class="r"><code># if(leng_dep != 0){
#   
#   string_db &lt;- STRINGdb$new(version=&quot;11.5&quot;, species=9606, score_threshold=400)
# 
#   string_mapped &lt;- string_db$map(string_res, &quot;Gene.name&quot;, removeUnmappedRows = TRUE, quiet = T)
# 
#   hits &lt;- string_mapped$STRING_id[1:leng_dep]
# 
#   string_db$plot_network(hits, add_link = T)
# }else{
#   print(&quot;NOT enough data.&quot;)
# }</code></pre>
<div id="cytoscape-network" class="section level3">
<h3><strong>6.1. Cytoscape Network</strong></h3>
<p>Cytoscape is an open source software platform for visualizing complex
networks and integrating these with any type of attribute data. In this
presentation all proteins showing at least one confidence level from the
STRING database are shown. The <strong>color</strong> code is relative
to <strong>fold change</strong> and the <strong>size</strong> is
directly proportional to the <strong>significance
(p-value)</strong>.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>if(n_edge != 0){
  knitr::include_graphics(&quot;assets/network_FC_pvalue.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
<div id="cytoscape-enrichmentmap" class="section level3">
<h3><strong>6.2. Cytoscape EnrichmentMap</strong></h3>
<p>An <strong>enrichment map</strong> is a different sort of network.
Instead of nodes representing genes, nodes represent pathways or
functions. Edges between these pathways or functions represent shared
genes or pathway crosstalk. An enrichment map is a way to visualize your
enrichment results to help reduce redundancy and uncover main
themes.</p>
<pre class="r"><code>print(&quot;Enable Cytoscape option to see this results.&quot;)</code></pre>
<pre><code>[1] &quot;Enable Cytoscape option to see this results.&quot;</code></pre>
<pre class="r"><code>Gname &lt;- results %&gt;% filter(significant) %&gt;% pull(name)

if(!params$Imputation){
  
  Uni &lt;- unique_pair(data_filt_unique, conditions = conditions, table = T) %&gt;% 
  pull(name) %&gt;% 
  unique()

  Gname &lt;- c(Gname, Uni)
}

if(length(Gname) == 0){
  
  print(&quot;NOT enough data.&quot;)
  
}else{
  
  GO &lt;- gost(query = Gname, organism = &quot;hsapiens&quot;, sources = c(&quot;GO:BP&quot;, &quot;GO:MF&quot;, &quot;GO:CC&quot;, &quot;KEGG&quot;), ordered_query = T, evcodes = TRUE)
  
  if(is.null(GO)){
    
    print(&quot;NOT enough data.&quot;)

    enrich_map = F
    
  }else{
    
    enrich_map = T
    
    gem &lt;- GO$result[,c(&quot;term_id&quot;, &quot;term_name&quot;, &quot;p_value&quot;, &quot;intersection&quot;)]
    colnames(gem) &lt;- c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;Genes&quot;)
    gem$FDR &lt;- gem$p.Val
    gem$Phenotype = &quot;+1&quot;
    gem &lt;- gem[,c(&quot;GO.ID&quot;, &quot;Description&quot;, &quot;p.Val&quot;, &quot;FDR&quot;, &quot;Phenotype&quot;, &quot;Genes&quot;)]
    head(gem)
  
    write.table(gem, here::here(&quot;output/enrichmentmap.txt&quot;),col.name=TRUE,sep=&quot;\t&quot;,row.names=FALSE,quote=FALSE)
    
  }
}</code></pre>
<pre class="r"><code>if(enrich_map){
  knitr::include_graphics(&quot;assets/network_enrichmentmap.png&quot;,  error = F)
}else{
  print(&quot;NOT enough data.&quot;)
}</code></pre>
</div>
</div>
<div id="section-3" class="section level2 unnumbered">
<h2 class="unnumbered"></h2>
<pre class="r"><code>saveWorkbook(excel, here::here(paste0(&quot;output/results_table_&quot;, test, &quot;.xlsx&quot;)), overwrite = T)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=Italian_Italy.1252  LC_CTYPE=Italian_Italy.1252   
[3] LC_MONETARY=Italian_Italy.1252 LC_NUMERIC=C                  
[5] LC_TIME=Italian_Italy.1252    

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] devtools_2.4.3              usethis_2.1.6              
 [3] forcats_0.5.1               stringr_1.4.0              
 [5] dplyr_1.0.8                 purrr_0.3.4                
 [7] readr_2.1.2                 tidyr_1.2.0                
 [9] tibble_3.1.6                tidyverse_1.3.1            
[11] openxlsx_4.2.5              here_1.0.1                 
[13] limma_3.46.0                RColorBrewer_1.1-3         
[15] RCy3_2.10.2                 STRINGdb_2.2.2             
[17] naniar_0.6.1                visdat_0.5.3               
[19] ggrepel_0.9.1               ggplot2_3.3.6              
[21] UpSetR_1.4.0                ggstatsplot_0.9.3          
[23] datapasta_3.1.0             patchwork_1.1.1            
[25] DT_0.23                     rrvgo_1.2.0                
[27] org.Hs.eg.db_3.12.0         AnnotationDbi_1.52.0       
[29] gprofiler2_0.2.1            enrichR_3.0                
[31] preprocessCore_1.52.1       SummarizedExperiment_1.20.0
[33] Biobase_2.50.0              GenomicRanges_1.42.0       
[35] GenomeInfoDb_1.26.7         IRanges_2.24.1             
[37] S4Vectors_0.28.1            BiocGenerics_0.36.1        
[39] MatrixGenerics_1.2.1        matrixStats_0.61.0         
[41] DEP_1.12.0                  MBQN_2.2.0                 
[43] workflowr_1.7.0            

loaded via a namespace (and not attached):
  [1] rappdirs_0.3.3         R.methodsS3_1.8.1      bit64_4.0.5           
  [4] knitr_1.39             DelayedArray_0.16.2    R.utils_2.11.0        
  [7] data.table_1.14.2      RCurl_1.98-1.6         doParallel_1.0.17     
 [10] generics_0.1.2         callr_3.7.0            RSQLite_2.2.10        
 [13] correlation_0.8.1      chron_2.3-56           bit_4.0.4             
 [16] tzdb_0.3.0             lubridate_1.8.0        xml2_1.3.3            
 [19] httpuv_1.6.5           assertthat_0.2.1       xfun_0.30             
 [22] hms_1.1.1              jquerylib_0.1.4        evaluate_0.15         
 [25] promises_1.2.0.1       fansi_1.0.3            readxl_1.4.0          
 [28] caTools_1.18.2         dbplyr_2.1.1           igraph_1.3.0          
 [31] DBI_1.1.2              htmlwidgets_1.5.4      paletteer_1.4.0       
 [34] hash_2.2.6.2           ellipsis_0.3.2         crosstalk_1.2.0       
 [37] backports_1.4.1        insight_0.17.1         gridBase_0.4-7        
 [40] vctrs_0.4.0            remotes_2.4.2          imputeLCMD_2.0        
 [43] cachem_1.0.6           withr_2.5.0            treemap_2.4-3         
 [46] prettyunits_1.1.1      cluster_2.1.0          lazyeval_0.2.2        
 [49] crayon_1.5.1           labeling_0.4.2         pkgconfig_2.0.3       
 [52] slam_0.1-50            pkgload_1.2.4          statsExpressions_1.3.2
 [55] ProtGenerics_1.22.0    wordcloud_2.6          rlang_1.0.2           
 [58] RJSONIO_1.3-1.6        lifecycle_1.0.1        sandwich_3.0-1        
 [61] affyio_1.60.0          BiocFileCache_1.14.0   modelr_0.1.8          
 [64] cellranger_1.1.0       rprojroot_2.0.3        graph_1.68.0          
 [67] datawizard_0.4.1       Matrix_1.2-18          zoo_1.8-10            
 [70] reprex_2.0.1           whisker_0.4            GlobalOptions_0.1.2   
 [73] processx_3.5.3         pheatmap_1.0.12        png_0.1-7             
 [76] viridisLite_0.4.0      rjson_0.2.21           mzR_2.24.1            
 [79] parameters_0.18.1      bitops_1.0-7           shinydashboard_0.7.2  
 [82] getPass_0.2-2          R.oo_1.24.0            KernSmooth_2.23-17    
 [85] blob_1.2.3             shape_1.4.6            tmvtnorm_1.5          
 [88] scales_1.2.0           memoise_2.0.1          magrittr_2.0.3        
 [91] plyr_1.8.7             gplots_3.1.3           zlibbioc_1.36.0       
 [94] compiler_4.0.3         plotrix_3.8-2          pcaMethods_1.82.0     
 [97] clue_0.3-60            cli_3.2.0              affy_1.68.0           
[100] XVector_0.30.0         ps_1.6.0               MASS_7.3-53           
[103] tidyselect_1.1.2       vsn_3.58.0             stringi_1.7.6         
[106] highr_0.9              yaml_2.3.5             GOSemSim_2.16.1       
[109] norm_1.0-10.0          MALDIquant_1.21        grid_4.0.3            
[112] sass_0.4.1             tools_4.0.3            circlize_0.4.15       
[115] rstudioapi_0.13        foreach_1.5.2          git2r_0.30.1          
[118] gridExtra_2.3          farver_2.1.0           mzID_1.28.0           
[121] digest_0.6.29          BiocManager_1.30.18    shiny_1.7.1           
[124] proto_1.0.0            Rcpp_1.0.8.3           broom_0.8.0           
[127] performance_0.9.0      later_1.3.0            ncdf4_1.19            
[130] httr_1.4.3             MSnbase_2.15.7         ComplexHeatmap_2.13.1 
[133] colorspace_2.0-3       brio_1.1.3             rvest_1.0.2           
[136] XML_3.99-0.9           fs_1.5.2               rematch2_2.1.2        
[139] gmm_1.6-6              sessioninfo_1.2.2      plotly_4.10.0         
[142] xtable_1.8-4           jsonlite_1.8.0         zeallot_0.1.0         
[145] NLP_0.2-1              testthat_3.1.3         R6_2.5.1              
[148] tm_0.7-8               gsubfn_0.7             pillar_1.7.0          
[151] htmltools_0.5.2        mime_0.12              glue_1.6.2            
[154] fastmap_1.1.0          BiocParallel_1.24.1    codetools_0.2-16      
[157] pkgbuild_1.3.1         mvtnorm_1.1-3          utf8_1.2.2            
[160] lattice_0.20-41        bslib_0.3.1            sqldf_0.4-11          
[163] curl_4.3.2             gtools_3.9.2           zip_2.2.0             
[166] GO.db_3.12.1           rmarkdown_2.14         desc_1.4.1            
[169] munsell_0.5.0          GetoptLong_1.0.5       GenomeInfoDbData_1.2.4
[172] iterators_1.0.14       impute_1.64.0          haven_2.5.0           
[175] gtable_0.3.0           bayestestR_0.12.1     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
