---
title: "LFQ_Analysis"
author: "Fabio Bedin | MS-Unit at IEO"
output: html_document
params:
  proteinGroups:
    label: "Input dataset:"
    value: proteinGroups_exp2.txt
    input: file
  expdesign:
    label: "Experimental design:"
    value: expdesign_exp2.txt
    input: file
  n_peptides: 
    label: "N° Peptides"
    value: 0
    input: select
    choices: [0, 1, 2, 3]
  Normalization:
    label: "Normalization:"
    value: Max-LFQ
    input: select
    choices: [Max-LFQ, VSN, MBQN]
  FDR:
    label: "FDR"
    value: 0.05
    input: select
    choices: [0.05, 0.01]
  Fold_change: 
    label: "Fold Change"
    value: 1
    input: slider
    min: 0
    max: 2.5
    step: 0.5
    sep: ""
  Imputation: TRUE
  Cytoscape: FALSE
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(fig.align = "center", warning=FALSE, message=FALSE, fig.height=8, fig.width=10)
```

```{css fontsize, echo=FALSE}
p{
  font-size: 16px;
}
```

```{r libreries}
## Proteomics
library("MBQN")
library("DEP")
library("SummarizedExperiment")
library("preprocessCore")
library("enrichR")
library("gprofiler2")
library("org.Hs.eg.db")
library("rrvgo")

## Plots
library("DT")
library("patchwork")
library("datapasta")
library("ggstatsplot")
library("UpSetR")
library("ggrepel")
library("visdat")
library("naniar")

## Networks
library("STRINGdb")
library("RCy3")
library("RColorBrewer")

## Other
library("limma")
library("here")
library("openxlsx")
library("tidyverse")
```

```{r custom-functions}
source(here::here("code/custom_functions.R"))
```

```{r load-PG}
data <- read.csv(here::here(paste0("data/", params$proteinGroups)), header = TRUE,stringsAsFactors = FALSE, sep = "\t")
## qui metterei anche Razor...unique.peptides al posto di Peptides, perchè così non conto i peptidi che sono attribuiti a più di 1 PG
data <- data[data$Reverse != "+" & data$Potential.contaminant != "+" & data$Only.identified.by.site != "+",]

data_unique <- make_unique(data, "Gene.names", "Protein.IDs", delim = ";")
```

```{r expdesig}
# cond_1 <- params$test_1
# 
# cond_2 <- params$test_2
# 
# conditions<-c(cond_1,cond_2)
# 
# test<- paste(cond_1,cond_2,sep="_vs_")

expdesign <- read.table(here::here(paste0("data/", params$expdesign)), header = T, stringsAsFactors = F)

# expdesign <- subset(expdesign, condition %in% conditions)

columns<-match(paste("LFQ.intensity.",expdesign$label,sep=""),colnames(data_unique))

data_se <- make_se(data_unique, columns, expdesign)

## define automaticaly the best statistical threshold to filter data besed on replicates:
# if(max(expdesign$replicate)<=3){
#        threshold<-0
#      } else if(max(expdesign$replicate)<6){
#        threshold<-1
#      } else if (max(expdesign$replicate)>=6){
#        threshold<-trunc(max(expdesign$replicate)*0.25) ## 0.25 serve per avere sempre il 75% di valid vales
#      }

data_filt <- filter_missval(data_se, thr = 1)
```


# All EGFR

***

# **Introduction**

First, we need to wrangling the original dataset. From the **proteinGroups** obtained by [MaxQuant](https://www.maxquant.org/) software, I remove proteins that are marked form the software as potential contaminant, only identify by site and reverse.   
We che also filter the initial datasets by excluding all proteins identified by less than `r params$n_peptides` peptides.   
Now, we generate some quality control plots to investigate the structure of the dataset and observe the effect of filters.    
  
#### **Normalization strategy:**   

```{r normalization, results = 'asis'}
if(params$Normalization == "VSN"){
  data_filt <- normalize_vsn(data_filt)
  print("VSN normalization is used")
} else if(params$Normalization == "MBQN"){
  data_filt <- normalize_MBQN(data_filt)
  print("MBQN normalization is used")
} else {
  print("Max-LFQ normalization is used")
}
```

```{r batch-effects}
# batch <- c("A", "A", "A", "A", "B", "B", "B", "B", "C", "C", "C", "D", "D", "D", "D")
# 
# assay(data_filt) <- limma::removeBatchEffect(assay(data_filt), batch)
```

## **1. Quality control plots** {.tabset .tabset-fade}

### Proteins per samples {.tabset}

This plot show the number of proteins identify in each samples *after and before User define* filters:

#### Filtered

```{r proteins-filt}
plot_numbers_lables(data_filt)
```

#### Not-Filtered

```{r proteins-NOfilt}
plot_numbers_lables(data_se)
```

### Distributions {.tabset}

This plot is useful for checking the distributions of the samples *after and before User define* filters:

#### Filtered

```{r normalization-filt}
plot_normalization(data_filt)
```

#### Not-Filtered

```{r normalization-NOfilt}
plot_normalization(data_se)
```

### Missing data {.tabset}

This plot we can explore the missing data pattern *after and before User define* filters.
Notice that the percentages of missingness are provided in the data. These are accurate to 1 decimal place.
Also the dataset is arranged by columns with most missingness.

#### Filtered

```{r missing-data-filt}
assay(data_filt) %>% 
  as.data.frame() %>% 
  vis_miss(., sort_miss = TRUE)
```

#### Not-Filtered

```{r missing-data-NOfilt}
assay(data_se) %>% 
  as.data.frame() %>% 
  vis_miss(., sort_miss = TRUE)
```

### Sample CVs {.tabset}

The coefficient of variation (CV) is a statistical measure of the dispersion of data points in a data series around the mean. The coefficient of variation represents the ratio of the standard deviation to the mean, and it is a useful statistic for comparing the degree of variation from one data series to another, even if the means are drastically different from one another.

#### Filtered

```{r CVs-filt}
plot_cvs(data_filt)
```

#### Not-Filtered

```{r CVs-NOfilt}
plot_cvs(data_se)
```

## **2. Upset plot** {.tabset .tabset-fade}

With this plot we can identify the numper of prteins in common within the 2 conditions or in common between all condition.


### Both conditions

```{r upset-both, fig.height=12, fig.width=15}
define_set <- assay(data_filt) %>% colnames()
n_sets <- length(define_set)
n_each <- n_sets / 4

assay(data_filt) %>% 
  as.data.frame() %>% 
  rownames_to_column() %>% 
  pivot_longer(!rowname, names_to = "samples", values_to = "intensity") %>% 
  mutate(intensity = if_else(is.na(intensity), 0, 1)) %>% 
  pivot_wider(names_from = samples, values_from = intensity) %>% 
  as.data.frame() %>% 
  upset(nsets = n_sets,
        sets = define_set,
        order.by = "freq",
        keep.order = T,
        text.scale = 2.5,
        point.size = 4,
        line.size = 0.5, 
        # sets.bar.color = rep(c("red3", "royalblue1", "red", "yellow"), each = n_each),
        main.bar.color  = "gray44")
  
```

## **3. `r if(params$Imputation){"Imputation"}else{"Unique proteins"}`**

`r if(params$Imputation){"An idealized version of a label-free discovery mass spectrometry proteomics experiment would provide absolute abundance measurements for a whole proteome, across varying conditions. Unfortunately, this ideal is not realized. Measurements are made on peptides requiring an inferential step to obtain protein level estimates. The inference is complicated by experimental factors that necessitate relative abundance estimation and result in widespread non-ignorable missing data. Relative abundance on the log scale takes the form of parameter contrasts. In a complete-case analysis, contrast estimates may be biased by missing data and a substantial amount of useful information will often go unused."}else{"This table show unique protein for both conditions"}`


```{r imputation, eval=params$Imputation, include=params$Imputation, results='hide'}
data_imputed <- DEP::impute(data_filt, fun = "man")

plot_imputation(data_filt, data_imputed)

data_filt <- data_imputed
```


## **4. Differential enrichment analysis** 

Protein-wise linear models combined with empirical Bayes statistics are used for the differential enrichment analysis (or differential expression analysis) and False discovery rate (FDR) is ajusted with *Benjamini-Hochberg* procedure.
Significant proteins are are define with **FDR = `r params$FDR`** and **Fold change = `r params$Fold_change`**

```{r DEP}
data_diff <- anova_test(data_filt)

dep <- add_rejections(data_diff, alpha = params$FDR, lfc = params$Fold_change)

results<-get_results(dep)
```

There are **`r results %>% filter(significant) %>% nrow()`** significant proteins

The results from the previous analysis can be easily visualized by a number of functions. These visualizations assist in the determination of the optimal cutoffs to be used, highlight the most interesting samples and contrasts, and pinpoint differentially enriched/expressed proteins.

### **4.1. Visualization of the results** {.tabset .tabset-fade}

#### PCA

The PCA plot can be used to get a high-level overview of the data. This can be very useful to observe batch effects, such as clear differences between replicates.

```{r PCA}
if(params$Imputation){
  if(nrow(dep) >= 500){
    n_value <- 500
  }else {
    n_value <- nrow(dep)
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = "condition", label = T, label_size = 3)
} else {
  var <- apply(assay(dep), 1, sd)
  df <- assay(dep)[order(var, decreasing = TRUE)[seq_len(nrow(dep))],]
  n_value <- df %>% as.data.frame() %>% drop_na() %>% nrow()
  if(n_value >= 500){
    n_value <- 500
  }
  plot_pca(dep, x = 1, y = 2, n = n_value, point_size = 4, indicate = "condition", label = T, label_size = 3)
}
```

#### Heatmap unsupervised

```{r complex-heatmap}
mat <- as.matrix(assay(data_filt))
mat_z <- scale(mat)
aaa <- results %>% filter(significant) %>% pull(name)
mat_final <- mat_z %>% 
  as.data.frame() %>% 
  rownames_to_column(var = "gene_names") %>% 
  filter(gene_names %in% aaa) %>% 
  column_to_rownames("gene_names") %>% 
  as.matrix()


  

ComplexHeatmap::Heatmap(mat_final,
                        name = "z-score",
                        # clustering_distance_rows = "pearson",
                        # clustering_method_rows = "average",
                        use_raster = FALSE,
                        show_row_names = FALSE)
```

#### Heatmap time course

```{r complex-heatmap-time-course}
col_split <- c("sample2", "sample2", "sample2", "sample2", "sample2",
               "sample3", "sample3", "sample3", "sample3", "sample3",
               "sample4", "sample4", "sample4", "sample4", "sample4",
               "sample5", "sample5", "sample5", "sample5", "sample5",
               "sample1", "sample1", "sample1", "sample1", "sample1")

ComplexHeatmap::Heatmap(mat_final,
                        name = "heatmap",
                        use_raster = FALSE,
                        show_row_names = FALSE,
                        # clustering_distance_rows = "single",
                        # cluster_rows = FALSE,
                        column_split = col_split,
                        cluster_column_slices = FALSE)
                        # row_km = 5)
```
